---
title: 'Arthropod genomics beyond fruit flies: bridging the gap between proximate and ultimate causation'
layout: publication
permalink: /publications/ultimate-and-proximate-evolution-in-insects/
---

<br />

<p class="info-text">Published: 17 November 2015</p>

# Arthropod genomics beyond fruit flies: bridging the gap between proximate and ultimate causation

<h5 class="info-text my-4">Yannick Wurm</h5>

[*Briefings in Functional Genomics*](https://doi.org/10.1093/bfgp/elv034), Volume 14, Issue 6, November 2015, Pages 381–383

<br />


<hr />

Many of us can still remember the publication of the *Drosophila melanogaster* genome sequence <span class="info-text">[1]</span>. This new resource enhanced the research toolkit for this species, which included laboratory protocols for rearing, mutagenesis, crossing, phenotyping, establishing stable lines, extracting nucleic acids, creating transgenic individuals and *in situ* imaging <span class="info-text">[2, 3]</span>. The combined toolkit brought extensive insight into the molecular, physiological and genetic bases of development and behavior in this species. Some of *D. melanogaster*’s genomic features set it apart from other model species whose genomes had also been sequenced, including *Saccharomyces cerevisiae*, *Caenorhabditis elegans*, *Danio rerio*, *Homo sapiens* and *Mus musculus*. For example, *D. melanogaster* lacks canonical de novo and maintenance DNA methyltransferase genes and shows only limited amounts of DNA methylation <span class="info-text">[4]</span>. Similarly, *D. melanogaster* chromosome ends are maintained by a pair of transposable elements rather than by a telomerase enzyme which synthesizes TTAGGG-like repeats in most eukaryotes <span class="info-text">[5, 6]</span>. Furthermore, the specific genes and interactions underlying circadian rhythms in *D. melanogaster* are different from those found in many other animals <span class="info-text">[7]</span>. Back when *D. melanogaster* was the only arthropod with a sequenced genome, many of the specific features of its genome were extrapolated as being representative of arthropods, the group of 6 million extant species <span class="info-text">[8]</span>, which includes insects, crustaceans, arachnids such as spiders, and myriapods such as millipedes and centipedes.

However, as additional arthropod genomes were sequenced, it became clear that even among arthropods, *D. melanogaster* and its dipteran relatives (other flies and mosquitoes) are the odd ones out. Indeed, *D. melanogaster* has more derived chromosomal structure (microsynteny) and more derived gene sequences than other insects <span class="info-text">[9]</span>. Similarly, its *Hox* gene cluster is organized in a more derived manner than in non-dipteran insects <span class="info-text">[10]</span>. *Drosophila melanogaster*’s genome being relatively derived may be linked to traits that make the species an attractive laboratory organism. Indeed, short generation times, ease of rearing and tolerance of relatively high population densities <span class="info-text">[2]</span> may contribute to *D. melanogaster* having a greater effective population size than many other insects <span class="info-text">[11]</span>. This characteristic is expected to increase the efficiency of natural selection <span class="info-text">[12]</span> and thus may accelerate the rate of change in dipteran genomes. Consistently with this idea, many characteristics of other arthropod genomes appear to be more vertebrate-like than *Drosophila*-like <span class="info-text">[4, 7, 13–17]</span>.

The bottom line is: *Drosophila* genomes are not representative of arthropod genomes. To ecologically and evolutionary minded researchers, this is unlikely to come as a surprise. Indeed, *Drosophila* behavior and morphology cannot be considered representative of a >500 million year old phylum <span class="info-text">[18]</span> that includes >6 million extant species <span class="info-text">[8]</span>. Life history traits of arthropods are hugely diverse, including ant species that perform slave raiding, others that perform agriculture and some that select a single reproductive queen by executing supernumerary queens <span class="info-text">[19–21]</span>, treehoppers with morphologies resembling abstract art <span class="info-text">[22]</span>, assassin bugs that carry corpses of their prey as camouflage <span class="info-text">[23]</span>, parasitoid wasps that lay eggs in larvae of other species <span class="info-text">[24]</span> or even produce normal reproductive larvae and sterile soldiers that will attack unrelated parasitoids <span class="info-text">[25]</span> and highly complex courtship and mating rituals <span class="info-text">[26]</span>.

Research in ecology and evolution has traditionally involved a diversity of study systems and focused on understanding how evolutionary processes including natural selection, sexual selection, host–parasite dynamics and drift are responsible for particular traits. Inferences are largely based on knowledge of interactions within and between species often derived from limited molecular information such as microsatellite or restriction fragment length polymorphism genotypes (e.g. <span class="info-text">[27]</span>). In such research, genes have mainly been considered at conceptual levels. This approach thus contrasts starkly with the mechanistic, laboratory-based approaches of *Drosophila* research, which aim to understand the proximate mechanisms through which individual genes contribute to shaping phenotypes. The recent drop in sequencing costs is enabling the two approaches to merge <span class="info-text">[28]</span>. Novel sequence information—including from the recent 1KITE thousand insect transcriptome project <span class="info-text">[18]</span> and from the ongoing i5k project to sequence 5000 arthropod genomes <span class="info-text">[29]</span>—is enabling ecology and evolution researchers to identify genes and genetic architectures underlying phenotypes across a range of study systems.

This issue of *Briefings* focuses on recent developments at the overlap between mechanistic and evolutionary research questions, in particular honing in on research from non-*Drosophila* arthropods.

We begin by an overview of opportunities, challenges and efforts to integrate evolutionary developmental biology insights on genotype–phenotype relationships with evolutionary ecology insights on phenotype–environment relationships. This review includes diverse examples of techniques and study systems, including the discussion of the role of the *Ultrabithorax* gene in shaping water-strider leg morphologies, which are under both natural and sexual selection <span class="info-text">[30]</span>. Similar approaches have also honed in on other groups of genes. Verhulst and Zande clarify how chromosomal sex-determining systems (ZZ/ZW, XX/XY, XX/XO and haplodiploidy) translate into sexual identity being expressed at the level of individual cells—and ultimately whole organisms. Sex-specific spliceforms of transformer and doublesex play central roles in this but the details vary between species <span class="info-text">[31]</span>. Viljakainen <span class="info-text">[32]</span> provides a comparative and evolutionary overview of the conserved and divergent components of insect innate immune defence, considering population genetics and molecular evolution results. Expanding outwards, we look at the molecules underlying interactions between herbivorous insects and their plant hosts. In particular, Simon et al. <span class="info-text">[33]</span> reviews recent research on the molecules underlying sensing, feeding, digesting and detoxifying in Lepidoptera (which include moths and butterflies) and Hemiptera (which includes aphids, cicadas, planthoppers and leafhoppers).

As highlighted by these reviews, linking candidate loci to phenotypes remains challenging: it can be difficult or even impossible to rear such non-traditional model organisms in the laboratory, and despite recent improvements, protocols for approaches such as transgenics or *in situ* remain uncommon for most species. Population genomics approaches may provide an intermediate solution, as they can indicate whether changes in a gene or genomic region are likely to have been neutral or selectively advantageous. Such approaches can thus shed light on historic events but also complement other approaches for identifying and prioritizing candidate genes. An overview of population genomics approaches and examples of the insights they have already provided on non-*Drosophila* arthropods are reviewed in depth by <span class="info-text">[34]</span>. The combination of population genomics, increased sequencing of species <span class="info-text">[29]</span> and increased accessibility of molecular tools such as transgenics <span class="info-text">[35]</span> and RNA interference <span class="info-text">[36]</span> will continue to help bridge the gap between our understanding of ultimate causes and proximate mechanisms.

<br />

## Funding
<hr />

Y.W. was supported by the Biotechnology and Biological Sciences Research Council [BB/K004204/1] and the Natural Environment Research Council [NE/L00626X/1, EOS Cloud].

<br />

## References
<hr />

1.	Adams MD Celniker SE Holt RAet al. . The genome sequence of Drosophila melanogaster. *Science* 2000;287: 2185–95.

2.	Ashburner M Golic KG Hawlwy RS. Drosophila. *A Laboratory Handbook*. Cold Spring Harbor Laboratory Press, 1989.

3.	Sambrook J Fritsch EF Maniatis Tet al. . *Molecular Cloning, Vol. 2*. New York: Cold Spring Harbor Laboratory Press, 1989.

4.	Glastad KM Hunt BG Yi SVet al. . DNA methylation in insects: on the brink of the epigenomic era. *Insect Mol Biol* 2011;20:553–65.

5.	Pardue ML Rashkova S Casacuberta Eet al. . Two retrotransposons maintain telomeres in Drosophila. *Chromosome Res* 2005;13:443–53.

6.	Gomes NM Shay JW Wright WE. Telomere biology in metazoa. *FEBS Lett* 2010;584:3741–51. Telomeres.

7.	Ingram KK Kutowoi A Wurm Yet al. . The molecular clockwork of the fire ant Solenopsis invicta. *PLoS One* 2012;7:e45715.

8.	Hamilton AJ Basset Y Benke KKet al. . Correction. *Am Nat* 2011;177:544–5.

9.	Kirkness EF Haas BJ Sun Wet al. . Genome sequences of the human body louse and its primary endosymbiont provide insights into the permanent parasitic lifestyle. *Proc Natl Acad Sci USA* 2010;107:12168–73.

10.	Heffer A Pick L. Conservation and variation in hox genes: how insect models pioneered the evo-devo field. *Annu Rev Entomol* 2013;58:161–79.

11.	Thomas JA Welch JJ Lanfear Ret al. . A generation time effect on the rate of molecular evolution in invertebrates. *Mol Biol Evol* 2010;27:1173–80.

12.	Charlesworth B. Fundamental concepts in genetics: effective population size and patterns of molecular evolution and variation. *Nat Rev Genet* 2009;10:195–205.

13.	Drosophila 12 Genomes Consortium. Evolution of genes and genomes on the Drosophila phylogeny. *Nature* 2007;450:203–18.

14.	Wurm Y Wang J Riba-Grognuz Oet al. . The genome of the fire ant Solenopsis invicta. *Proc Natl Acad Sci USA* 2011;108:5679–84.

15.	Corona M Libbrecht R Wurm Yet al. . Vitellogenin underwent subfunctionalization to acquire caste and behavioral specific expression in the harvester ant Pogonomyrmex barbatus. *PLoS Genet* 2013;9:e1003730.

16.	Simola DF Wissler L Donahue Get al. . Social insect genomes exhibit dramatic evolution in gene composition and regulation while preserving regulatory features linked to sociality. *Genome Res* 2013;23:1235–47.

17.	Mason J Randall T Capkova Frydrychova R. Telomerase lost? *Chromosoma* 2015;1–9.

18.	Misof B Liu S Meusemann Ket al. . Phylogenomics resolves the timing and pattern of insect evolution. *Science* 2014;346:763–7.

19.	Hölldobler B Wilson EO. The Ants. Cambridge, MA: The Belknap Press of Harvard University Press, 1990.

20.	Fletcher DJC Blum MS. Regulation of queen number by workers in colonies of social insects. *Science* 1983;219:312–4.

21.	Wurm Y Wang J Keller L. Changes in reproductive roles are associated with changes in gene expression in fire ant queens. *Mol Ecol* 2010;19:1200–11.

22.	Maderspacher F Stensmyr M. Myrmecomorphomania. *Curr Biol* 2011;21:R291–3.

23.	Brandt M Mahsberg D. Bugs with a backpack: the function of nymphal camouflage in the West African assassin bugs Paredocla and Acanthaspis spp. *Anim Behav* 2002;63:277–84.

24.	Wurm Y Keller L. Parasitoid wasps: From natural history to genomic studies. *Curr Biol* 2010;20:R242–4.

25.	Cruz Y. A sterile defender morph in a polyembryonic hymenopterous parasite. *Nature* 1981;294:446–7.

26.	Choe JC Crespi BJ. The evolution of mating systems in insects and arachnids. Cambridge: *Cambridge University Press*, 1997.

27.	Buechel SD Wurm Y Keller L. Social chromosome variants differentially affect queen determination and the survival of workers in the fire ant Solenopsis invicta. *Mol Ecol* 2014;23:5117–27.

28.	Nygaard S Wurm Y. Ant genomics: challenges to overcome and opportunities to seize. *Myrmecological News* 2015;21:59–72.

29.	i5K Consortium. The i5k initiative: advancing arthropod genomics for knowledge, human health, agriculture, and the environment. *J Hered* 2013;104:595–600.

30.	Santos ME Berger CS Refki PNet al. . Integrating evo-devo with ecology for a better understanding of phenotypic evolution. *Brief Funct Genomic* 2015;14:384–95.

31.	Verhulst EC van de Zande L. Double nexus—doublesex is the connecting element in sex determination. *Brief Funct Genomic* 2015;14:396–406.

32.	Viljakainen L. Evolutionary genetics of insect innate immunity. *Brief Funct Genomic* 2015;14:407–12.

33.	Simon JC d’Alençon E Guy Eet al. . Genomics of adaptation to host-plants in herbivorous insects. *Brief Funct Genomic* 2015;14:413–23.

34.	Hasselmann M Ferretti L Zayed A. Beyond fruit-flies: population genomic advances in non-Drosophila arthropods. *Brief Funct Genomic* 2015;14:424–31.

35.	Gilles AF Averof M. Functional genetics for all: engineered nucleases, CRISPR and the gene editing revolution. *Evodevo* 2014;5:43.

36.	Bellés X. Beyond Drosophila: RNAi in vivo and functional genomics in insects. *Annu Rev Entomol* 2010;55:111–28.

© The Author 2015. Published by Oxford University Press. All rights reserved. For permissions, please email: journals.permissions@oup.com

© The Author 2015. Published by Oxford University Press. All rights reserved. For permissions, please email: journals.permissions@oup.com

<br />
