<html><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript">
(function(){(function(){function d(a){this.t={};this.tick=function(a,b,c){this.t[a]=[void 0!=c?c:(new Date).getTime(),b]};this.tick("start",null,a)}var a=new d;window.jstiming={Timer:d,load:a};if(window.performance&&window.performance.timing){var a=window.performance.timing,b=window.jstiming.load,c=a.navigationStart,a=a.responseStart;0<c&&a>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_","_wtsrt",a),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),
b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(e){}})();})()
</script><script type="text/javascript">var KX_timer = new window.jstiming.Timer(); KX_timer.name = 'published';</script><title>DAY3_exercises_Oksana_WORKS_denovo.docx</title><style type="text/css">
      body {
        font-family: arial, sans, sans-serif;
        margin: 0;
      }

      iframe {
        border: 0;
        frameborder: 0;
        height: 100%;
        width: 100%;
      }

      #header, #footer {
        background: #f0f0f0;
        padding: 10px 10px;
      }

      #header {
        border-bottom: 1px #ccc solid;
      }

      #footer {
        border-top: 1px #ccc solid;
        border-bottom: 1px #ccc solid;
        font-size: 13;
      }

      #contents {
        margin: 6px;
      }

      .dash {
        padding: 0 6px;
      }
    </style></head><body><div id="header">DAY3_exercises_Oksana_WORKS_denovo.docx</div><div id="contents"><style type="text/css">@import url('https://themes.googleusercontent.com/fonts/css?kit=MSSLfUayeNh9PW3ng9UWrqo0P1CSBNc3gBWclSzSx0c');.lst-kix_list_1-3>li:before{content:"\0025cf  "}.lst-kix_list_1-1>li:before{content:"\0025cb  "}.lst-kix_list_1-6>li:before{content:"\0025cf  "}.lst-kix_list_1-2>li:before{content:"\0025a0  "}.lst-kix_list_1-5>li:before{content:"\0025a0  "}ul.lst-kix_list_1-0{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}.lst-kix_list_1-7>li:before{content:"\0025cb  "}.lst-kix_list_1-0>li:before{content:"\0025cf  "}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}.lst-kix_list_1-8>li:before{content:"\0025a0  "}ul.lst-kix_list_1-7{list-style-type:none}.lst-kix_list_1-4>li:before{content:"\0025cb  "}ol{margin:0;padding:0}.c5{line-height:1.15;padding-top:0pt;text-align:left;direction:ltr;margin-left:36pt;padding-bottom:0pt}.c13{line-height:1.15;padding-top:11pt;padding-bottom:2pt}.c11{font-size:11pt;font-style:italic;font-family:"Verdana"}.c27{max-width:487.3pt;background-color:#ffffff;padding:72pt 54pt 72pt 54pt}.c6{padding-top:2pt;text-align:left;direction:ltr}.c0{color:#dd1144;font-size:8pt;font-family:"Courier New"}.c1{font-size:8pt;font-family:"Courier New"}.c22{font-size:14pt;font-family:"Arial"}.c17{padding-top:2pt;margin-left:28.4pt}.c15{font-size:20pt;font-family:"Calibri"}.c12{color:#1155cc;text-decoration:underline}.c10{color:#999988;font-style:italic}.c2{font-size:11pt;font-family:"Courier New"}.c25{padding-top:2pt;margin-left:28.3pt}.c23{color:#0000ff;text-decoration:underline}.c20{margin:0;padding:0}.c19{padding-left:0pt;margin-left:36pt}.c14{color:inherit;text-decoration:inherit}.c18{font-style:italic}.c16{height:14pt}.c7{direction:ltr}.c4{color:#009999}.c9{height:12pt}.c8{font-size:8pt}.c28{text-align:center}.c3{font-weight:bold}.c26{font-family:"Verdana"}.c21{text-align:left}.c24{font-family:"Courier New"}.title{padding-top:12pt;line-height:1.0;text-align:center;color:#000000;font-size:16pt;font-family:"Arial";font-weight:bold;padding-bottom:3pt}.subtitle{padding-top:6pt;line-height:1.0;text-align:center;color:#000000;font-size:12pt;font-family:"Arial";padding-bottom:3pt}li{color:#000000;font-size:12pt;font-family:"Cambria"}p{color:#000000;font-size:12pt;margin:0;font-family:"Cambria"}h1{padding-top:24pt;line-height:1.0;text-align:justify;color:#345a8a;font-size:18pt;font-family:"Calibri";font-weight:bold;padding-bottom:0pt}h2{padding-top:12pt;line-height:1.0;text-align:justify;color:#000000;font-style:italic;font-size:14pt;font-family:"Arial";font-weight:bold;padding-bottom:3pt}h3{padding-top:12pt;line-height:1.0;text-align:justify;color:#000000;font-size:13pt;font-family:"Arial";font-weight:bold;padding-bottom:3pt}h4{padding-top:12pt;line-height:1.0;text-align:justify;color:#000000;font-size:14pt;font-family:"Cambria";font-weight:bold;padding-bottom:3pt}h5{padding-top:12pt;line-height:1.0;text-align:justify;color:#000000;font-style:italic;font-size:13pt;font-family:"Cambria";font-weight:bold;padding-bottom:3pt}h6{padding-top:12pt;line-height:1.0;text-align:justify;color:#000000;font-size:11pt;font-family:"Cambria";font-weight:bold;padding-bottom:3pt}</style><p class="c7"><span class="c15 c3">DAY </span><span class="c3 c15">3</span><span class="c15 c3">&nbsp;EXERCISES</span></p><h5 class="c7 c13"><a name="h.zgy66088lxz0"></a><span>Oksana Riba-Grognuz</span></h5><hr><h1 class="c7"><span>Differential Gene Expression</span></h1><p class="c7"><span>We
 will be using one of the most popular programs for differential 
expression analyses and transcript isoform assembly from spliced 
alignments of RNA-seq reads: </span><span class="c12"><a class="c14" href="http://tophat.cbcb.umd.edu/">TopHat</a></span><span class="c12">&nbsp;</span><span>and</span><span><a class="c14" href="http://cufflinks.cbcb.umd.edu/">&nbsp;</a></span><span class="c12"><a class="c14" href="http://cufflinks.cbcb.umd.edu/">Cufflinks</a></span><span>.
 These programs are actively maintained and updated. TopHat relies on 
Bowtie to carry out short-read alignments. Recently, following the 
release of Bowtie2, new TopHat and Cufflinkes versions were released. We
 will be using the latest release of TopHat and Cufflinks with Bowtie1. 
To do so, login to vital-it and source file 
“Tophat/latest_versions.bashrc” as shown below. Source command will 
modify your $PATH as specified in latest_versions.bashrc and update 
environment variables specifying versions of TopHat and Cufflinks. </span></p><p class="c6"><span class="c2">$ ssh -t </span><span class="c2 c3">username</span><span class="c2">@prd.vital-it.ch ssh dee-serv0</span><span class="c2 c3">X</span></p><p class="c6"><span class="c2">$ cd /scratch/cluster/weekly/</span><span class="c2 c3">username</span></p><p class="c6"><span class="c2">$ source Top</span><span class="c2">H</span><span class="c2">at/latest_versions.bashrc</span></p><p class="c7 c9"><span></span></p><h2 class="c7 c28"><a name="h.v3t2kjkgidhq"></a><span>YOU NEED R 2.15!!!!</span></h2><p class="c7"><span>If everything worked the installed software versions should be:</span></p><ul class="c20 lst-kix_list_1-0 start"><li class="c7 c19"><span>tophat-2.0.3.Linux_x86_64 </span></li><li class="c19 c7"><span>cufflinks-2.0.0.Linux_x86_64</span></li></ul><p class="c7 c9"><span></span></p><p class="c7"><span>Please
 check which versions of Cufflinks and TopHat you have. If these are 
different from the ones listed above, give us a sign and we will set up 
the correct ones.</span></p><p class="c6"><span class="c2">$ which tophat</span></p><p class="c6"><span class="c2">$ echo $TOPHAT_VERSION</span></p><p class="c6"><span class="c2">$ which cufflinks</span></p><p class="c6"><span class="c2">$ echo $CUFFLINKS_VERSION</span></p><h2 class="c7 c16"><a name="h.baj4m3nih6h4"></a></h2><h2 class="c7"><a name="h.baj4m3nih6h4"></a><span>Tophat: mapping reads to genome. </span></h2><p class="c7"><span>We will start by preparing all necessary input files to run TopHat. As it uses </span><span class="c12"><a class="c14" href="http://bowtie-bio.sourceforge.net/">Bowtie</a></span><span>&nbsp;to
 align short reads to a reference genome, we need to generate Bowie 
index for genome. For this go to GenomeSubset/Assembly/ folder and run 
the command bowtie-build on scaffold sequences contained in FASTA file </span><span class="c3">SINV_subset_1.fa</span><span>.
 Start by looking how to launch this command (HINT: we do not need any 
options). Note that it is convenient to use the same prefix for input 
fasta file and bowtie index.</span></p><p class="c6"><span class="c2">$ cd /scratch/cluster/weekly/</span><span class="c2 c3">username</span><span class="c2">/GenomeSubset/Assembly</span></p><p class="c6"><span class="c2">$ ls</span></p><p class="c6"><span class="c2">$ bowtie-build -h</span></p><p class="c6 c9"><span class="c2"></span></p><p class="c6"><span>Please launch the appropriate command.</span></p><p class="c7 c17"><span class="c18">Q: How many index files were generated?</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>We will use 75 bp single Illumina RNA-seq data from 3 conditions, each with 4 biological</span><span>&nbsp;</span><span>replicates:
 Queens, Workers and Males, prefixed with Q, W, M respectively. Each 
replicate represents a pool of individuals. A 3 digit number in file 
names indicates red fire ant colony from which sample was taken. Have a 
look in the folder RNA-seq/Raw</span></p><p class="c6"><span class="c2">$ ls /scratch/cluster/weekly/username/RNA-seq/Raw</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>Due
 to time constraints we will launch TopHat and Cufflinks on 3 files. For
 the remainder of the pipeline we will use pre-calculated TopHat and 
Cufflinks results with all files. Go to the directory TopHat.</span></p><p class="c6"><span class="c2">$ cd /scratch/cluster/weekly/username/TopHat</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>In
 this directory you will find a file called “file.list” that lists 3 
file names in column 1 and corresponding fragment sizes in column 2. 
Fragment sizes must be specified when running Cufflinks on single reads 
(note that for paired read analyses Cufflinks calculates fragment size 
distribution based on alignments). Have a look on TopHat parameters.</span></p><p class="c6"><span class="c2">$ tophat -h</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>A
 file launching TopHat is called runTopHat.sh, have a look on it. This 
script takes all files in “file.list” and processes 1 by 1. Output 
directories are named according to sample names (prefix of fastq files).
 </span></p><p class="c6"><span class="c2">$ less runTopHat.sh</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>In
 the beginning of this file there is a little trick to avoid manually 
changing username: the variable with username is set using command 
“whoami”. Thus if you used your username as working directory name you 
do </span><span class="c3">NOT</span><span>&nbsp;need to change links to
 input files (genome index and raw files location) in launch script. 
Optionally you can change TopHat parameters. Use nano or vi editor 
within Vital-IT, or scp the script to edit locally as we did on Day1. 
Once done, launch the script.</span></p><p class="c6"><span class="c2">$ ./runTopHat.sh</span></p><p class="c7 c9"><span></span></p><h2 class="c7 c21"><a name="h.izzxru72s74h"></a><span>Cufflinks: mapping-based (= reference-based) gene/transcript identification - separately for each sample </span></h2><p class="c7"><span>While
 TopHat is running we can get ready for the next part of the pipeline: 
transcript assembly. Cufflinks assembles putative transcripts based on 
the alignments generated by TopHat. Have a look on Cufflinks parameters 
and on respective launch script “runCufflinks.sh”. To do this open a new
 terminal window or tab. </span></p><p class="c6"><span class="c2">$ cufflinks -h</span></p><p class="c6"><span class="c2">$ less runCufflinks.sh</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>When
 TopHat is over launch Cufflinks and while it is running examine TopHat 
output files. Use samtools to look into bam files. Bam </span><span>is a</span><span>&nbsp;binary version of </span><span class="c12"><a class="c14" href="http://samtools.sourceforge.net/SAM1.pdf">SAM</a></span><span>&nbsp;(Sequence Alignment/Map) format </span><span>used</span><span>&nbsp;for storing large nucleotide sequence alignments. </span></p><p class="c6"><span class="c2">$ ./runCufflinks.sh</span></p><p class="c6"><span class="c2">$ samtools view W403/accepted_hits.bam | less</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>Column
 6 in bam file contains CIGAR strings. These strings specify the 
structure of alignment. For example if all 75 bp align it will state 
75M, meaning 75 Match. If there is a 315bp gap in read alignment it will
 state 61M315N14M, meaning that first 61bp Match, then there are 315 
Non-matching genomic bases, and finally 14bp Match (total 75bp of read 
matched). CIGAR strings with N, are thus reads likely containing splice 
junctions (span over several exons). &nbsp;</span></p><p class="c7 c9"><span class="c3"></span></p><p class="c7"><span class="c3">This transcript identification was performed once per sample. </span><span>When
 Cufflinks finished assembling transcripts for each lane have a look on 
its output files. Next step is to merge all assemblies and thus generate
 a reference set that can be used for differential expression part. 
&nbsp;For this part we will be using Tophat/Cufflinks output pre-run on 
all RNA-seq files. This output is in the folder TopHat_full.</span></p><p class="c6"><span class="c2">$ less M350</span><span class="c2">b</span><span class="c2">/Cufflinks/genes.fpkm_tracking</span></p><p class="c6"><span class="c2">$ less M350</span><span class="c2">b</span><span class="c2">/Cufflinks/transcripts.gtf</span></p><p class="c6"><span class="c2">$ cd /scratch/cluster/weekly/username/TopHat_full</span></p><p class="c7 c9"><span></span></p><p class="c7"><span class="c18 c3 c22">Cuffmerge: merging results from the 3*4=12 cufflinks analyses.</span></p><p class="c7"><span>See how to run cuffmerge.</span></p><p class="c6"><span class="c2">$ cuffmerge -h</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>As you can see the program requires an input of gtf file list. Generate one.</span></p><p class="c6"><span class="c2">$ ls -1 */Cufflinks/transcripts.gtf &gt; gtf.list</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>Run
 Cuffmerge to combine transcripts assembled for each replicate. We will 
use reference file generated by Cuffmerge for differential expression 
analysis with Cuffdiff.</span></p><p class="c6"><span class="c2">$ genomeRef=/scratch/cluster/weekly/</span><span class="c2 c3">username</span><span class="c2">/GenomeSubset/Assembly/SINV_subset_1.fa</span></p><p class="c6"><span class="c2">$ cuffmerge -s $genomeRef gtf.list </span></p><p class="c7 c9"><span></span></p><p class="c7"><span>Have a look on the assembled merged gtf.</span></p><p class="c6"><span class="c2">$ less merged_asm/merged.gtf</span></p><p class="c7 c9"><span></span></p><h2 class="c7"><a name="h.z6oxvk9hsmtk"></a><span>Renaming our favorite candidate genes (so we can find them easily)</span></h2><p class="c7"><span>We will want to be able to easily identify our favorite genes after analysis. Thus</span><span>&nbsp;identify
 4 Vitellogenin proteins (Vg1, Vg2, Vg3, Vg4) and 2 Transformer proteins
 in the assembled “merged.gtf” we need to extract fasta sequences 
corresponding to coordinates in gtf file. After that we will run Blast 
of these sequences against file “Proteins.fasta” containing the proteins
 of interest. We will format file with “Proteins.fasta” as Blast 
database and run translated transcript query against it.</span></p><p class="c7"><span class="c2">$ gffread merged_asm/merged.gtf -g $genomeRef -w merged_asm/merged.fa</span></p><p class="c6"><span class="c2">$ </span><span class="c2">formatdb -p T -i Proteins.fasta</span></p><p class="c6"><span class="c2">$ blastall -p blastx -a 1 -m 8 -i merged_asm/merged.fa -d Proteins.fasta -v 1 -b 1 -e 1.0e-5 </span><span class="c2">&nbsp;&gt; merged_asm/merged.blastx.all</span></p><p class="c6 c9"><span class="c2"></span></p><p class="c7"><span>One
 of the Vg4 hits &nbsp;Additionally we will retain only hits with 
minimum alignment length of 150 for Transformer and over 1500 for 
Vitellogenins (column 4).</span></p><p class="c6 c9"><span class="c2"></span></p><p class="c6"><span class="c18">Q: one of the </span><span class="c18">fs</span></p><p class="c6"><span class="c2">cat merged_asm/merged.blastx.all </span><span class="c2">| awk '{if($2~/Tra/ &amp;&amp; $4&gt;=150){print $0} else if($4&gt;=1500){print $0}}' &gt; merged_asm/merged.blastx</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>Blast
 identifies matching transcript sequences prefixed “TCONS_” by default. 
As we will be working on gene level, we need to identify corresponding 
genes prefixed “XLOC_”. Use script “get.xloc.sh” to do this. Redirect 
the output of this script using “&gt;” to a file.</span></p><p class="c6"><span class="c2">$ cat merged_asm/merged.blastx</span></p><p class="c6"><span class="c2">$ ./get.xloc.sh &gt; myLoci.txt</span></p><p class="c6"><span class="c2">$ cat myLoci.txt</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>Finally
 we would like to update our gtf reference file with names listed in 
column 2 of myLoci.txt. For this we will make a backup copy of original 
file and then use a script “update_reference.sh” to do changes. </span></p><p class="c6"><span class="c2">$ cp merged_asm/merged.gtf merged_asm/merged_original.gtf</span></p><p class="c6"><span class="c2">$ ./update_reference.sh</span></p><p class="c6"><span class="c2">$ ./update_reference.sh myLoci.txt</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>Next
 we will run Cuffdiff that will calculate differential expression levels
 for genes and isoforms, as well as differential splicing and promoter 
use. We will need to estimate our average fragment length (Remember: 
this is only necessary when analyzing single Illumina reads). Use R (on 
Vital-IT) to read script file.list and calculate average fragment length
 (specified for each file in column 2). Launch Cuffdiff or use provided 
launch script called “runCuffdiff.sh”. If you choose to use the launch 
script then edit input file locations and average fragment length. </span></p><p class="c6"><span class="c2">$ cuffdiff -h</span></p><p class="c6"><span class="c2">$ ./runCuffdiff.sh</span></p><p class="c7 c9"><span></span></p><p class="c7"><span>Have a look at output files of Cuffdiff.</span></p><p class="c6"><span class="c2">$ ls Diff_FDR0.01</span></p><p class="c6"><span class="c2">$ less Diff_FDR0.01/gene_exp.diff</span></p><p class="c6"><span class="c2">$ less Diff_FDR0.01/genes.</span><span class="c2">count_tracking</span></p><p class="c6"><span class="c2">$ less Diff_FDR0.01/genes.read_group_tracking</span></p><p class="c6"><span class="c2">$ grep Vg Diff_FDR0.01/gene_exp.diff</span></p><h2 class="c7"><a name="h.1r05ens3puqp"></a><span>CummeRbund: Insanely great plots</span></h2><p class="c7"><span>We will now use R package </span><span class="c23"><a class="c14" href="http://compbio.mit.edu/cummeRbund/">cummeRbund</a></span><span>&nbsp;developed
 for the analysis of Cuffdiff output. Make sure you are using the latest
 R version (2.15). Start by downloading all files contained in Cuffdiff 
output directory to your computer locally.</span><span>&nbsp;It will be faster if you “zip -r Diff_FDR0.01.zip Diff_FDR0.01” first.</span><span class="c8">&nbsp;[</span><span class="c12 c8"><a class="c14" href="http://antgenomes.org/%7Eyannickwurm/Diff_FDR0.01.zip">failproof backup</a></span><span class="c8">]</span></p><p class="c6"><span class="c2">$ mkdir Diff_FDR0.01</span></p><p class="c6"><span class="c2">$ scp </span><span class="c2 c3">username</span><span class="c2">@prd.vital-it.ch:/scratch/cluster/weekly/</span><span class="c2 c3">username</span><span class="c2">/TopHat_full/Diff_FDR0.01/* Diff_FDR0.01/</span></p><p class="c6"><span class="c2">$ scp </span><span class="c2 c3">username</span><span class="c2">@prd.vital-it.ch:/scratch/cluster/weekly/</span><span class="c2 c3">username</span><span class="c2">/TopHat_full/myLoci.txt .</span></p><p class="c6"><span class="c2">$ R</span></p><p class="c5"><span class="c1 c10"># install package</span></p><p class="c5"><span class="c1 c10"># source("http://bioconductor.org/biocLite.R")</span></p><p class="c5"><span class="c1 c10"># biocLite(c("cummeRbund", "Hmisc", "gplots"))</span></p><p class="c5 c9"><span class="c1 c10"></span></p><p class="c5"><span class="c1">library(cummeRbund)</span></p><p class="c5"><span class="c1">options(width</span><span class="c1 c3">=</span><span class="c1 c4">65</span><span class="c1">)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># import data</span></p><p class="c5"><span class="c1">cuff </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;readCufflinks(dir </span><span class="c1 c3">=</span><span class="c1">&nbsp;</span><span class="c0">"Diff_FDR0.01"</span><span class="c1">, rebuild</span><span class="c1 c3">=</span><span class="c1">T)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># look at object structure</span></p><p class="c5"><span class="c1">str(cuff)</span></p><p class="c5"><span class="c1">genes </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;genes(cuff)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># plot gene expression density</span></p><p class="c5"><span class="c1">dens</span><span class="c1 c3">&lt;-</span><span class="c1">csDensity(genes)</span></p><p class="c5"><span class="c1">dens</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># boxplots per sample</span></p><p class="c5"><span class="c1">b</span><span class="c1 c3">&lt;-</span><span class="c1">csBoxplot(genes)</span></p><p class="c5"><span class="c1">b</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># assess general similarity between samples</span></p><p class="c5"><span class="c1 c10"># using a scatter plot comparing gene FPKM values for each comparison</span></p><p class="c5"><span class="c1">(s</span><span class="c1 c3">&lt;-</span><span class="c1">csScatter(genes,</span><span class="c0">"Q"</span><span class="c1">, </span><span class="c0">"W"</span><span class="c1">, smooth</span><span class="c1 c3">=</span><span class="c1">T))</span></p><p class="c5"><span class="c1">(s</span><span class="c1 c3">&lt;-</span><span class="c1">csScatter(genes,</span><span class="c0">"Q"</span><span class="c1">, </span><span class="c0">"M"</span><span class="c1">, smooth</span><span class="c1 c3">=</span><span class="c1">T))</span></p><p class="c5"><span class="c1">(s</span><span class="c1 c3">&lt;-</span><span class="c1">csScatter(genes,</span><span class="c0">"M"</span><span class="c1">, </span><span class="c0">"W"</span><span class="c1">, smooth</span><span class="c1 c3">=</span><span class="c1">T))</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># inspect differentially expressed genes</span></p><p class="c5"><span class="c1">(v</span><span class="c1 c3">&lt;-</span><span class="c1">csVolcano(genes,</span><span class="c0">"Q"</span><span class="c1">,</span><span class="c0">"W"</span><span class="c1">))</span></p><p class="c5"><span class="c1">(v</span><span class="c1 c3">&lt;-</span><span class="c1">csVolcano(genes,</span><span class="c0">"Q"</span><span class="c1">,</span><span class="c0">"M"</span><span class="c1">))</span></p><p class="c5"><span class="c1">(v</span><span class="c1 c3">&lt;-</span><span class="c1">csVolcano(genes,</span><span class="c0">"M"</span><span class="c1">,</span><span class="c0">"W"</span><span class="c1">))</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># &nbsp;gene fpkm values</span></p><p class="c5"><span class="c1">gene.fpkm</span><span class="c1 c3">&lt;-</span><span class="c1">fpkm(genes)</span></p><p class="c5"><span class="c1">head(gene.fpkm)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># access gene fpkm as a matrix</span></p><p class="c5"><span class="c1">gene.matrix</span><span class="c1 c3">&lt;-</span><span class="c1">fpkmMatrix(genes)</span></p><p class="c5"><span class="c1">head(gene.matrix)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># isoform fpkm</span></p><p class="c5"><span class="c1">isoform.fpkm</span><span class="c1 c3">&lt;-</span><span class="c1">fpkm(isoforms(cuff))</span></p><p class="c5"><span class="c1">head(isoform.fpkm)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># gene expression differences</span></p><p class="c5"><span class="c1">gene.diff</span><span class="c1 c3">&lt;-</span><span class="c1">diffData(genes)</span></p><p class="c5"><span class="c1">head(gene.diff)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># extract data for significant differences</span></p><p class="c5"><span class="c1">sig_gene_data </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;subset(gene.diff, (significant </span><span class="c1 c3">==</span><span class="c1">&nbsp;</span><span class="c0">"yes"</span><span class="c1">))</span></p><p class="c5 c9"><span class="c8 c26"></span></p><p class="c5"><span class="c11">#Q: why significant gene names are not unique?</span></p><p class="c5 c9"><span class="c11"></span></p><p class="c5"><span class="c1">sigGenes </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;unique(sig_gene_data$gene_id)</span></p><p class="c5"><span class="c1">length(sigGenes)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># cummeRbund is currently implementing the support for replicates</span></p><p class="c5"><span class="c1 c10"># we extract replicate values by reading file "genes.read_group_tracking"</span></p><p class="c5"><span class="c1">repData &nbsp;</span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;read.table(</span><span class="c0">"Diff_FDR0.01/genes.read_group_tracking"</span><span class="c1">,</span></p><p class="c5"><span class="c1">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;header</span><span class="c1 c3">=</span><span class="c1">T, sep</span><span class="c1 c3">=</span><span class="c0">"\t"</span><span class="c1">)</span></p><p class="c5"><span class="c1">sampleId </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;paste(repData$condition, repData$replicate, sep</span><span class="c1 c3">=</span><span class="c0">""</span><span class="c1">)</span></p><p class="c5"><span class="c1">repData &nbsp;</span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;data.frame(repData, </span><span class="c0">"sampleId"</span><span class="c1 c3">=</span><span class="c1">sampleId)</span></p><p class="c5"><span class="c1">repFPKM &nbsp;</span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;reshape(repData, &nbsp;idvar</span><span class="c1 c3">=</span><span class="c0">"tracking_id"</span><span class="c1">, </span></p><p class="c5"><span class="c1">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; timevar</span><span class="c1 c3">=</span><span class="c0">"sampleId"</span><span class="c1">, direction</span><span class="c1 c3">=</span><span class="c0">"wide"</span><span class="c1">,</span></p><p class="c5"><span class="c1">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; drop</span><span class="c1 c3">=</span><span class="c1">c(names(repData)[</span><span class="c1 c4">2</span><span class="c1">:</span><span class="c1 c4">6</span><span class="c1">], names(repData)[</span><span class="c1 c4">8</span><span class="c1">:</span><span class="c1 c4">9</span><span class="c1">]))</span></p><p class="c5"><span class="c1">rowNames </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;repFPKM[,</span><span class="c1 c4">1</span><span class="c1">]</span></p><p class="c5"><span class="c1">repFPKM &nbsp;</span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;repFPKM[</span><span class="c1 c4">-1</span><span class="c1">]</span></p><p class="c5"><span class="c1">row.names(repFPKM) </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;rowNames</span></p><p class="c5"><span class="c1">colnames(repFPKM) &nbsp;</span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;substr(colnames(repFPKM), </span><span class="c1 c4">6</span><span class="c1">,</span><span class="c1 c4">7</span><span class="c1">)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10">#heatmap for significant gene values in replicates</span></p><p class="c5"><span class="c1">library(gplots) &nbsp;</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1">FPKM.sig &nbsp; &nbsp; </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;repFPKM[sigGenes,]</span></p><p class="c5"><span class="c1">FPKM.sig &nbsp; &nbsp; </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;log2(FPKM.sig</span><span class="c1 c3">+</span><span class="c1 c4">0.5</span><span class="c1">)</span></p><p class="c5"><span class="c1">genes.dend &nbsp; </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;as.dendrogram (hclust (as.dist (</span><span class="c1 c4">1</span><span class="c1 c3">-</span><span class="c1">cor (t(FPKM.sig)))))</span></p><p class="c5"><span class="c1">samples.dend </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;as.dendrogram (hclust (as.dist (</span><span class="c1 c4">1</span><span class="c1 c3">-</span><span class="c1">cor (FPKM.sig))))</span></p><p class="c5"><span class="c1">heatmap.2(as.matrix(FPKM.sig), scale</span><span class="c1 c3">=</span><span class="c0">"row"</span><span class="c1">,</span></p><p class="c5"><span class="c1">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; Rowv</span><span class="c1 c3">=</span><span class="c1">genes.dend, Colv</span><span class="c1 c3">=</span><span class="c1">samples.dend, trace</span><span class="c1 c3">=</span><span class="c0">"none"</span><span class="c1">,</span></p><p class="c5"><span class="c1">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; col</span><span class="c1 c3">=</span><span class="c1">colorRampPalette(c(</span><span class="c0">"white"</span><span class="c1">, </span><span class="c0">"yellow"</span><span class="c1">, </span><span class="c0">"red"</span><span class="c1">))(</span><span class="c1 c4">100</span><span class="c1">))</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1">sample.names</span><span class="c1 c3">&lt;-</span><span class="c1">samples(genes)</span></p><p class="c5"><span class="c1">head(sample.names)</span></p><p class="c5"><span class="c1">gene.featurenames </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;featureNames(genes)</span></p><p class="c5"><span class="c1">head(gene.featurenames)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># let's look at Vitellogenins</span></p><p class="c5"><span class="c1">myGene </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;getGenes(cuff,</span><span class="c0">"Vg1"</span><span class="c1">)</span></p><p class="c5"><span class="c1">myGene</span></p><p class="c5"><span class="c1">expressionPlot(myGene)</span></p><p class="c5"><span class="c1">fpkm(myGene)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1">myGene </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;getGenes(cuff,</span><span class="c0">"Vg2"</span><span class="c1">)</span></p><p class="c5"><span class="c1">expressionPlot(myGene)</span></p><p class="c5"><span class="c1">fpkm(myGene)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1">myGene </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;getGenes(cuff,</span><span class="c0">"Vg3"</span><span class="c1">)</span></p><p class="c5"><span class="c1">expressionPlot(myGene)</span></p><p class="c5"><span class="c1">fpkm(myGene)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1">myGene </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;getGenes(cuff,</span><span class="c0">"Vg4"</span><span class="c1">)</span></p><p class="c5"><span class="c1">expressionPlot(myGene)</span></p><p class="c5"><span class="c1">fpkm(myGene)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c11"># Q: What do you conclude about expression of different Vitellogenins?</span></p><p class="c5"><span class="c1 c10"># let's take a Vg overexpressed in Queens and find 20 similar gene profiles</span></p><p class="c5"><span class="c1">mySimilar &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;findSimilar(cuff,</span><span class="c0">"Vg2"</span><span class="c1">,n</span><span class="c1 c3">=</span><span class="c1 c4">20</span><span class="c1">)</span></p><p class="c5"><span class="c1">mySimilar.expression </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;expressionPlot(mySimilar,logMode</span><span class="c1 c3">=</span><span class="c1">T,showErrorbars</span><span class="c1 c3">=</span><span class="c1">F)</span></p><p class="c5"><span class="c1">print(mySimilar.expression)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># we can also define a profile to search with numeric values</span></p><p class="c5"><span class="c1 c10"># let's find genes overexpressed in both female castes: queens and workers</span></p><p class="c5"><span class="c1">sample.names</span></p><p class="c5"><span class="c1">myProfile &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;c(</span><span class="c1 c4">0</span><span class="c1">,</span><span class="c1 c4">1000</span><span class="c1">,</span><span class="c1 c4">1000</span><span class="c1">)</span></p><p class="c5"><span class="c1">mySimilar2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;findSimilar(cuff,myProfile,n</span><span class="c1 c3">=</span><span class="c1 c4">10</span><span class="c1">)</span></p><p class="c5"><span class="c1">mySimilar2.expression </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;expressionPlot(mySimilar2,logMode</span><span class="c1 c3">=</span><span class="c1">T,showErrorbars</span><span class="c1 c3">=</span><span class="c1">F)</span></p><p class="c5"><span class="c1">print(mySimilar2.expression)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># let's find genes overexpressed in both sexual castes: queens and males</span></p><p class="c5"><span class="c1">myProfile &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;c(</span><span class="c1 c4">1000</span><span class="c1">,</span><span class="c1 c4">1000</span><span class="c1">,</span><span class="c1 c4">0</span><span class="c1">)</span></p><p class="c5"><span class="c1">mySimilar2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;findSimilar(cuff,myProfile,n</span><span class="c1 c3">=</span><span class="c1 c4">10</span><span class="c1">)</span></p><p class="c5"><span class="c1">mySimilar2.expression </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;expressionPlot(mySimilar2,logMode</span><span class="c1 c3">=</span><span class="c1">T,showErrorbars</span><span class="c1 c3">=</span><span class="c1">F)</span></p><p class="c5"><span class="c1">print(mySimilar2.expression)</span></p><p class="c5 c9"><span class="c1"></span></p><p class="c5"><span class="c1 c10"># k-means clustering</span></p><p class="c5"><span class="c1">myGenes</span><span class="c1 c3">&lt;-</span><span class="c1">getGenes(cuff,sig_gene_data$gene_id)</span></p><p class="c5"><span class="c1">ic </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;csCluster(myGenes, k </span><span class="c1 c3">=</span><span class="c1">&nbsp;</span><span class="c1 c4">4</span><span class="c1">)</span></p><p class="c5"><span class="c1">head(ic$cluster)</span></p><p class="c5"><span class="c1">icp </span><span class="c1 c3">&lt;-</span><span class="c1">&nbsp;csClusterPlot(ic)</span></p><p class="c5"><span class="c1">icp</span></p><p class="c5 c9"><span class="c2"></span></p><p class="c7 c9"><span class="c3 c24"></span></p><p class="c7"><span class="c3">Optional: Alternative Splicing</span></p><p class="c7"><span>We</span><span>&nbsp;know
 that Transformer genes (Tra1 and Tra2) have sex-specific alternative 
splicing. Prepare a gff file to examine possible isoforms in Apollo. 
Fasta sequence for scaffold containing these genes is in the file 
“SIgn00005.fa”. First we get the coordinates of Tra1 and Tra2 genes to 
Tra.gtf. Then we convert gtf to gff. Finally gff and fasta files must be
 uploaded to local computer to visualize with Apollo.</span></p><p class="c6"><span class="c2">$ grep Tra merged_asm/merged.gtf &gt; Tra.gtf</span></p><p class="c6"><span class="c2">$ gffread Tra.gtf -o Tra.gff</span></p><p class="c6"><span class="c2">$ scp </span><span class="c2 c3">username</span><span class="c2">@prd.vital-it.ch:/scratch/cluster/weekly/</span><span class="c2 c3">username</span><span class="c2">/TopHat_full/Tra.gtf .</span></p><p class="c6"><span class="c2">$ scp </span><span class="c2 c3">username</span><span class="c2">@prd.vital-it.ch:/scratch/cluster/weekly/</span><span class="c2 c3">username</span><span class="c2">/TopHat_full/SIgn00005.fa .</span></p><p class="c7 c25"><span class="c18">Q: Can you identify Tra1 and Tra2 isoforms that are likely to be artefacts?</span></p><p class="c25 c7"><span class="c18">Q: Which ones look plausible? </span></p><p class="c5 c9"><span class="c2"></span></p></div><div id="footer"><span>Published by <a target="_blank" title="Learn more about Google Drive" href="https://docs.google.com/">Google Drive</a></span><span class="dash">–</span><a href="https://docs.google.com/abuse?id=1UB8I67AqW1nPbIo1JN4td76Bbyu1b1hGxnr9Yoe2t-k">Report Abuse</a><span class="dash">–</span><span>Updated automatically every 5 minutes</span></div><script type="text/javascript">
(function(){if(window.jstiming){window.jstiming.a={};window.jstiming.b=1;var e=function(b,a,d){var c=b.t[a],g=b.t.start;if(c&&(g||d))return c=b.t[a][0],void 0!=d?g=d:g=g[0],c-g},n=function(b,a,d){var c="";window.jstiming.pt&&(c+="&srt="+window.jstiming.pt,delete window.jstiming.pt);try{window.external&&window.external.tran?c+="&tran="+window.external.tran:window.gtbExternal&&window.gtbExternal.tran?c+="&tran="+window.gtbExternal.tran():window.chrome&&window.chrome.csi&&(c+="&tran="+window.chrome.csi().tran)}catch(g){}var f=
window.chrome;if(f&&(f=f.loadTimes)){f().wasFetchedViaSpdy&&(c+="&p=s");if(f().wasNpnNegotiated){var c=c+"&npn=1",k=f().npnNegotiatedProtocol;k&&(c+="&npnv="+(encodeURIComponent||escape)(k))}f().wasAlternateProtocolAvailable&&(c+="&apa=1")}var l=b.t,s=l.start,f=[],k=[],h;for(h in l)if("start"!=h&&0!=h.indexOf("_")){var m=l[h][1];m?l[m]&&k.push(h+"."+e(b,h,l[m][0])):s&&f.push(h+"."+e(b,h))}delete l.start;if(a)for(var p in a)c+="&"+p+"="+a[p];(a=d)||(a="https:"==document.location.protocol?"https://csi.gstatic.com/csi":
"http://csi.gstatic.com/csi");return[a,"?v=3","&s="+(window.jstiming.sn||"_s")+"&action=",b.name,k.length?"&it="+k.join(","):"",c,"&rt=",f.join(",")].join("")};window.jstiming.getReportUri=n;var q=function(b,a,d){b=n(b,a,d);if(!b)return"";a=new Image;var c=window.jstiming.b++;window.jstiming.a[c]=a;a.onload=a.onerror=function(){window.jstiming&&delete window.jstiming.a[c]};a.src=b;a=null;return b};window.jstiming.report=function(b,a,d){if("prerender"==document.webkitVisibilityState){var c=!1,g=function(){if(!c){a?
a.prerender="1":a={prerender:"1"};var f;"prerender"==document.webkitVisibilityState?f=!1:(q(b,a,d),f=!0);f&&(c=!0,document.removeEventListener("webkitvisibilitychange",g,!1))}};document.addEventListener("webkitvisibilitychange",g,!1);return""}return q(b,a,d)};window.jstiming.reportDone=function(b){if(window.jstiming.b<=(b||1))return!1;for(var a in window.jstiming.a)return!1;return!0};var r=function(b,a,d,c){return 0<d?(c?b.tick(a,c,d):b.tick(a,"",d),!0):!1};window.jstiming.getNavTiming=function(b){if(window.performance&&
window.performance.timing){var a=window.performance.timing;r(b,"_dns",a.domainLookupStart)&&r(b,"dns_",a.domainLookupEnd,"_dns");r(b,"_con",a.connectStart)&&r(b,"con_",a.connectEnd,"_con");r(b,"_req",a.requestStart)&&r(b,"req_",a.responseStart,"_req");r(b,"_rcv",a.responseStart)&&r(b,"rcv_",a.responseEnd,"_rcv");if(r(b,"_ns",a.navigationStart)){r(b,"ntsrt_",a.responseStart,"_ns");r(b,"nsfs_",a.fetchStart,"_ns");var d=window.external&&window.external.startE;!d&&(window.chrome&&window.chrome.csi)&&
(d=Math.floor(window.chrome.csi().startE));d&&(r(b,"_se",d),r(b,"sens_",a.navigationStart,"_se"));r(b,"ntplt0_",a.loadEventStart,"_ns");r(b,"ntplt1_",a.loadEventEnd,"_ns")}}}};})()
</script><script type="text/javascript">KX_timer.tick('tl'); if (document.location.protocol == 'https:') {window.jstiming.report(KX_timer, undefined , 'https://gg.google.com/csi');} else {window.jstiming.report(KX_timer);}</script></body></html>