---
layout: page
title: Teaching
---

We teach on several courses, including:

 * [BIO113 Evolution (first year)](http://qmplus.qmul.ac.uk/course/view.php?id=3972)
 * [SBC322 Ecological and Evolutionary Genomics (3rd year)](http://www.sbcs.qmul.ac.uk/undergraduate/undergraduate/UG_modules/189312.html)
 * [SBC361 Research Methods and Communication 2 (3rd year)](http://www.sbcs.qmul.ac.uk/undergraduate/undergraduate/UG_modules/189319.html)
 * BIO721P Genome Bioinformatics (Masters)
 * BIO781P Statistics and Bioinformatics (Masters)

Much of the material we use is QMUL's QMPlus content management system. Some of the material we use is on GitHub or provided below.

<div class="col-sm-12 col-md-12">
    <h2 id="genome-bioinformatics">Genome Bioinformatics (Masters)</h2>
    <p>This module from the <a href="https://www.qmul.ac.uk/postgraduate/taught/coursefinder/courses/121410.html?gclid=EAIaIQobChMIgfrbmr7T2QIVT7ftCh3HeAo1EAAYASAAEgLQcPD_BwE">QMUL's Bioinformatics MSc</a> covers the essential aspects of next generation sequence  analysis, including genome assembly, variant calling and population genetics. Also covers essential computer skills needed for bioinformatics, such as Linux and using our high performance computing cluster.</p>
    <ul>
      <li><a href="https://wurmlab.github.io/genomicscourse/2017/practicals/reference_genome/read-cleaning">Practical 1:</a> Read cleaning.</li>
      <li><a href="https://wurmlab.github.io/genomicscourse/2017/practicals/reference_genome/assembly">Practical 2:</a> Genome assembly.</li>
      <li><a href="https://wurmlab.github.io/genomicscourse/2017/practicals/population_genetics/map_call">Practical 3:</a> Read mapping and variant calling.</li>
      <li><a href="http://wurmlab.github.io/genomicscourse/2017/practicals/population_genetics/popgen">Practical 4:</a> Population genetics in R.</li>
    </ul>
    <h2 id="bioinformatics--population-genomics">Bioinformatics &amp; Population Genomics</h2>
    <p><a href="//wurmlab.github.io/genomicscourse/2016-SIB/practicals/">Practicals</a> prepared with Rodrigo Pracana, Julien Roux, Rob Waterhouse and Stefan Schiffels for a 1 week <a href="https://www.sib.swiss/training/course/2016-05-joint-spring-school-bioinformatics-and-population-genomics">Bioinformatics &amp; Population Genomics </a> course. <a href="//www.slideshare.net/yannickwurm/2016-0529introsibspringschoolleuker-bad">[Overview slides]</a>. We use similar material for <a href="https://www.qmul.ac.uk/postgraduate/taught/coursefinder/courses/121410.html?gclid=EAIaIQobChMIgfrbmr7T2QIVT7ftCh3HeAo1EAAYASAAEgLQcPD_BwE">QMUL’s Bioinformatics MSc</a> and our <a href="https://www.qmul.ac.uk/postgraduate/taught/coursefinder/courses/121430.html">Ecological and Evolutionary Genomics MSc</a>:</p>
    <ul>
        <li><a href="//wurmlab.github.io/genomicscourse/2016-SIB/practicals/reference_genome/assembly">Reads to genome to gene predictions</a>:
            <br> Short read cleaning, genome assembly, quality control, gene prediction, quality control. <a href="//www.slideshare.net/yannickwurm/2016-0530mondayassembly">[Slides]</a></li>
        <li>Population sequencing to genotypes to population genetics statistics:
            <ul>
                <li><a href="//wurmlab.github.io/genomicscourse/2016-SIB/practicals/population_genetics/map_call">Mapping reads, calling variants, visualizing variant calls.</a></li>
                <li><a href="//wurmlab.github.io/genomicscourse/2016-SIB/practicals/population_genetics/popgen">Analysing variants: PCA, measuring Differentiation &amp; Diversity.</a></li>
            </ul>
        </li>
        <li><a href="//wurmlab.github.io/genomicscourse/2016-SIB/practicals/msmc/msmc-tutorial/guide">Inferring population sizes and gene flow.</a></li>
        <li>Gene expression:
            <ul>
                <li><a href="//wurmlab.github.io/genomicscourse/2016-SIB/practicals/rnaseq/TP1">From raw sequencing data to transcript expression levels.</a></li>
                <li><a href="//wurmlab.github.io/genomicscourse/2016-SIB/practicals/rnaseq/TP2">Gene-level clustering and differential expression analysis.</a></li>
            </ul>
        </li>
    </ul>
</div>

<div class="col-sm-10 col-md-10" id="programming_in_r">
    <h2>Learning how to program in R</h2> As biology is becoming a data science its increasingly important to be able to wrangle large datasets. We use variations of the following for QMUL's SBC361 (3rd year Research Methods) and Bio781p (MSc Statistics &amp; Bioinformatics):
    <ul>
        <li><a href="//wurmlab.github.io/SBC361-programming-in-R/practical1.html">Practical 1: R refresher and regular expressions</a></li>
        <li><a href="//wurmlab.github.io/SBC361-programming-in-R/practical2.html">Practical 2: Functions and for loops</a></li>
        <li><a href="//wurmlab.github.io/SBC361-programming-in-R/practical3.html">Practical 3: Practice to pull it all together</a></li>
    </ul>
</div>
<div class="col-sm-2 col-md-2">
    <div style="left: 0px; width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
        <iframe src="//www.slideshare.net/slideshow/embed_code/key/jCQoRUBEZOsam0" frameborder="0" allowfullscreen="true" webkitallowfullscreen="true" mozallowfullscreen="true" style="top: 0px; left: 0px; width: 100%; height: 100%; position: absolute;"></iframe>
    </div>
</div>

<div class="col-sm-12 col-md-12">
    <h2 id="ecological-and-evolutionary-genomics">Ecological and Evolutionary Genomics</h2>
    <p>Our students did some great blogging.</p>
    <p>2016: <a href="https://ecoevo2016.wordpress.com">Ecoevo @ Wordpress</a></p>
    <ul>
      <li><a href="https://ecoevo2016.wordpress.com/2016/12/05/origin-and-functional-evolution-of-y-chromosomes-across-mammals/">Origin and functional evolution of Y chromosomes across mammals</a></li>
      <li><a href="https://ecoevo2016.wordpress.com/2016/12/02/fight-or-flight-monarch-butterflies-and-their-migratory-journey/">Fight or flight? Monarch butterflies and their migratory journey.</a></li>
      <li><a href="https://ecoevo2016.wordpress.com/2016/11/28/speciation-of-darwins-finches-one-gene-that-rules-it-all/">Speciation of Darwin&rsquo;s Finches: One gene that rules it all</a></li>
      <li><a href="https://ecoevo2016.wordpress.com/2016/12/02/genomic-history-of-australo-papuans/">Genomic history of Australo-Papuans</a></li>
      <li><a href="https://ecoevo2016.wordpress.com/2016/11/28/wait-where-are-their-penises-asexual-reproduction-in-bdelloid-rotifers/">Wait where are their penises? Asexual reproduction in Bdelloid rotifers</a></li>
    </ul>
    <p>2015: <a href="https://thewinnower.com/keywords/sbc322">SBC322 @ Winnower</a></p>
    <ul>
      <li><a href="https://thewinnower.com/papers/3256-snips-and-snails-puppy-dog-tails-x-and-y-what-really-makes-a-man">Snips and Snails, Puppy-dog Tails, X and Y&hellip; What really makes a Man?</a></li>
      <li><a href="https://thewinnower.com/papers/3252-what-predicts-genetic-diversity">What Predicts Genetic Diversity?</a></li>
      <li><a href="https://thewinnower.com/papers/3072-picky-eaters-do-mosquitoes-prefer-the-taste-of-people-who-smell-better">Picky Eaters: Do mosquitoes prefer the taste of people who smell 'better'?</a></li>
      <li><a href="https://thewinnower.com/papers/3356-mountain-gorillas-on-verge-of-extinction">Mountain Gorillas On Verge Of Extinction</a></li>
      <li><a href="https://thewinnower.com/papers/3255-why-do-whales-not-have-gills-genome-sequencing-and-adaptive-history-of-the-minke-whale">Why Do Whales Not Have Gills? Genome Sequencing And Adaptive History Of The Minke Whale</a></li>
      <li><a href="https://thewinnower.com/papers/3021-is-sex-really-necessary">Is sex really necessary?</a></li>
      <li><a href="https://thewinnower.com/papers/3254-did-elephant-sharks-forget-evolution">Did Elephant sharks forget evolution?</a></li>
      <li><a href="https://thewinnower.com/papers/3253-the-future-of-carrion-and-hooded-crows-to-speciate-or-not-to-speciate">The Future of Carrion and Hooded Crows; To Speciate, or Not to Speciate?</a></li>
    </ul>
    <p>2014: <a href="https://qmecoevogenomics.wordpress.com">QMEcoEvoGenomics @ Wordpress</a> </p>
    <ul>
      <li><a href="https://qmecoevogenomics.wordpress.com/2015/01/25/sex-does-everyone-need-it-apparently-not-the-bdelliod-rotifer-adineta-vaga-doesnt/">Sex, does everyone need it? Apparently not, the bdelliod rotifer Adineta vaga doesn&rsquo;t</a></li>
      <li><a href="https://qmecoevogenomics.wordpress.com/2014/12/16/origins-and-functional-evolution-of-y-chromosomes-across-mammals/">Origins and functional evolution of Y chromosomes across mammals</a></li>
      <li><a href="https://qmecoevogenomics.wordpress.com/2014/12/16/the-origin-of-the-honeybee-a-worldwide-survey-of-honeybee-genomes-reveals-their-evolutionary-history/">The Origin of the Honeybee &ndash; A Worldwide Survey of Honeybee Genomes Reveals their Evolutionary History</a></li>
      <li><a href="https://qmecoevogenomics.wordpress.com/2014/12/16/a-worldwide-survey-of-genome-sequence-variation-provides-insight-into-the-evolutionary-history-of-the-honeybee-apis-mellifera/">A worldwide survey of genome sequence variation provides insight into the evolutionary history of the honeybee Apis mellifera</a></li>
      <li><a href="https://qmecoevogenomics.wordpress.com/2014/12/05/genome-wide-signatures-of-convergent-evolution-in-echo-locating-mammals/">Genome-wide signatures of convergent evolution in Echo locating mammals</a></li>
      <li><a href="https://qmecoevogenomics.wordpress.com/2014/12/02/the-origin-of-the-tasmanian-devil-clonally-transmissible-cancer/">The Origin of the Tasmanian Devil Clonally Transmissible Cancer</a></li>
      <li><a href="https://qmecoevogenomics.wordpress.com/2014/11/25/comparative-population-genomics-in-animals-uncovers-the-determinants-of-genetic-diversity/">Comparative population genomics in animals uncovers the determinants of genetic diversity</a></li>
      <li><a href="https://qmecoevogenomics.wordpress.com/2014/11/24/genome-chronicles-the-giant-pandas-history/">Genome chronicles &ndash; The Giant Panda&rsquo;s (hi)story</a></li>
    </ul>

</div>

<div class="col-sm-12 col-md-12">
    <h2>Evolution</h2> 1st year introduction (part of QMUL's BIO113):
</div>
<div class="col-sm-2 col-md-2" style="padding-bottom:38px;">
    <!-- 1 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
        <iframe src="//www.slideshare.net/slideshow/embed_code/key/DxhOYKI8g76CcU" frameborder="0" allowfullscreen="" style="width: 100%; height: 100%; position: absolute;"></iframe>
    </div>
</div>
<div class="col-sm-2 col-md-2">
    <!-- 2 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
        <iframe src="//www.slideshare.net/slideshow/embed_code/key/6n2KuXrF13QOay" frameborder="0" allowfullscreen="" style="width: 100%; height: 100%; position: absolute;"> </iframe>
    </div>
</div>
<div class="col-sm-2 col-md-2">
    <!-- 3 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
        <iframe src="//www.slideshare.net/slideshow/embed_code/key/EbEcpxCEUx66x1" frameborder="0" allowfullscreen="" style="width: 100%; height: 100%; position: absolute;"> </iframe>
    </div>
</div>
<div class="col-sm-2 col-md-2">
    <!-- 4 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
        <iframe src="//www.slideshare.net/slideshow/embed_code/key/Y9nUUktwu7G2u" frameborder="0" allowfullscreen="" style="width: 100%; height: 100%; position: absolute;"> </iframe>
    </div>
</div>
<div class="col-sm-2 col-md-2">
    <!-- 5 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
        <iframe src="//www.slideshare.net/slideshow/embed_code/key/LOuf0tvzOMHvwr" frameborder="0" allowfullscreen="" style="width: 100%; height: 100%; position: absolute;"> </iframe>
    </div>
</div>
<div class="col-sm-12 col-md-12">
    <h2>Older stuff</h2>
    <h3>2012 Swiss Bioinformatics Institute Summerschool</h3> From 2012 Swiss Bioinformatics Institute <a href="//www.isb-sib.ch/education/sib-phd-training-network/events/past-events/phd-summer-school-2012.html">summerschool in Bioinformatics and Population Genomics</a> (with Oksana Riba-Grognuz, Kate Ridout and Eyal Privman). This is largely superceded by the newer version (see above):
    <ul>
        <li><a href="//www.slideshare.net/yannickwurm/20120622-fridayadelboden" title="20120622 fridayadelboden" target="_blank">Overview and summary slides</a></li>
        <li><a href="DAY1_exercises.docx.html">UNIX introduction, <em>de novo</em> genome assembly &amp; <em>de novo</em> transcriptome assembly, and cross-validation using Cytoscape of the genome and transcriptome assemblies</a> [<a href="//docs.google.com/document/pub?id=1lx8cGQlyBZV8RXm7gSLqFafcOUfPq6_GjadqBxqiAzQ">Google Docs link</a>]</li>
        <li><a href="MWAS%20practical.docx.html">Automated gene model prediction and manual refinement</a> [<a href="//docs.google.com/document/pub?id=10P7oSJ3M_Jg6EzSPbsKx1HV6WQprOURcpvIBxa5Yk6g">Google Docs link</a>]</li>
        <li><a href="DAY3_exercises_Oksana_WORKS_denovo.docx.html">Differential gene expression with tophat/cufflinks/cummerbund</a> [<a href="//docs.google.com/document/pub?id=1UB8I67AqW1nPbIo1JN4td76Bbyu1b1hGxnr9Yoe2t-k">Google Docs link</a>]</li>
        <li><a href="//www.slideshare.net/yannickwurm/20120622-fridayadelboden">Some random tips</a> and <a href="Experimental%20design%20for%20genomics.html">discussion topics</a> [<a href="//docs.google.com/document/pub?id=1WcHIJPzkOXXjnViwJt_ZVERlJz2YUqhM6yXQ1OLQTxQ">Google Docs link</a>]</li>
    </ul>
</div>
<div class="col-sm-12 col-md-12">
    <h3>2006-2010 stuff from Uni Lausanne:</h3>
    <ul>
        <li><a href="bachelor3-tp4Microarray">Introduction to microarray analysis</a>: Three hour practical for 3rd year Biology students. Download normalized affimterix data, blindly use Limma to find significant genes, cluster them with TMEV, conduct GO analysis of one cluster… conclude about confounding factors in experimental design.</li>
        <li><a href="bachelor2-tp2Blast">Introduction to BLAST</a>: Two hour practical for 2nd year biology students. Calculate score &amp; evalue of an alignment by hand… compare with true Blast results obtained on NCBI, see that blastp and blastn have different sensitivities. Check some new NCBI blast features.</li>
    </ul>
    <p>Semester programming projects:</p>
    <ul>
        <li><strong>Programming project</strong>: for introduction to python/biopython class. <a href="bachelor2-introToProg/ProgrammingForBiologists_Wurm_project.pdf">Determining Codon Adaptation Index in ants</a>. Required datafiles: <a href="bachelor2-introToProg/SI_putativeTranscripts.fasta">sequence fasta file</a> and <a href="bachelor2-introToProg/SI_All_bx">blastx report</a>.</li>
        <li><strong>Programming project</strong>: for introduction to python/biopython class. <a href="bachelor2-introToProg/ProteinCorrectionProject.pdf">Improved protein sequences for non-model organisms</a>.</li>
    </ul>
</div>
