<html><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript">
(function(){(function(){function d(a){this.t={};this.tick=function(a,b,c){this.t[a]=[void 0!=c?c:(new Date).getTime(),b]};this.tick("start",null,a)}var a=new d;window.jstiming={Timer:d,load:a};if(window.performance&&window.performance.timing){var a=window.performance.timing,b=window.jstiming.load,c=a.navigationStart,a=a.responseStart;0<c&&a>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_","_wtsrt",a),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),
b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(e){}})();})()
</script><script type="text/javascript">var KX_timer = new window.jstiming.Timer(); KX_timer.name = 'published';</script><title>MWAS practical.docx</title><style type="text/css">
      body {
        font-family: arial, sans, sans-serif;
        margin: 0;
      }

      iframe {
        border: 0;
        frameborder: 0;
        height: 100%;
        width: 100%;
      }

      #header, #footer {
        background: #f0f0f0;
        padding: 10px 10px;
      }

      #header {
        border-bottom: 1px #ccc solid;
      }

      #footer {
        border-top: 1px #ccc solid;
        border-bottom: 1px #ccc solid;
        font-size: 13;
      }

      #contents {
        margin: 6px;
      }

      .dash {
        padding: 0 6px;
      }
    </style></head><body><div id="header">MWAS practical.docx</div><div id="contents"><style type="text/css">@import url('https://themes.googleusercontent.com/fonts/css?kit=m0tazYRimFnV1hoGKbgtnw');ol{margin:0;padding:0}.c5{padding-top:5pt;direction:ltr;padding-bottom:5pt}.c4{max-width:415pt;background-color:#ffffff;padding:72pt 90pt 72pt 90pt}.c9{color:#0000ff;text-decoration:underline}.c10{color:#548dd4;font-size:18pt}.c8{color:inherit;text-decoration:inherit}.c1{text-align:justify;direction:ltr}.c6{font-size:11pt;font-family:"Courier New"}.c7{color:#1155cc;text-decoration:underline}.c3{height:12pt}.c11{vertical-align:super}.c2{font-style:italic}.c12{direction:ltr}.c0{font-weight:bold}.title{padding-top:12pt;line-height:1.0;text-align:center;color:#000000;font-size:16pt;font-family:"Arial";font-weight:bold;padding-bottom:3pt}.subtitle{padding-top:0pt;line-height:1.0;text-align:center;color:#000000;font-size:12pt;font-family:"Arial";padding-bottom:3pt}li{color:#000000;font-size:12pt;font-family:"Cambria"}p{color:#000000;font-size:12pt;margin:0;font-family:"Cambria"}h1{padding-top:12pt;line-height:1.0;text-align:left;color:#000000;font-size:16pt;font-family:"Arial";font-weight:bold;padding-bottom:3pt}h2{padding-top:12pt;line-height:1.0;text-align:left;color:#000000;font-style:italic;font-size:14pt;font-family:"Arial";font-weight:bold;padding-bottom:3pt}h3{padding-top:12pt;line-height:1.0;text-align:left;color:#000000;font-size:13pt;font-family:"Arial";font-weight:bold;padding-bottom:3pt}h4{padding-top:12pt;line-height:1.0;text-align:left;color:#000000;font-size:14pt;font-family:"Cambria";font-weight:bold;padding-bottom:3pt}h5{padding-top:12pt;line-height:1.0;text-align:left;color:#000000;font-style:italic;font-size:13pt;font-family:"Cambria";font-weight:bold;padding-bottom:3pt}h6{padding-top:5pt;line-height:1.0;text-align:left;color:#000000;font-size:7pt;font-family:"Times New Roman";font-weight:bold;padding-bottom:5pt}</style><p class="c1"><span class="c10 c0">MWAS practical </span></p><p class="c1 c3"><span class="c10 c0"></span></p><p class="c1"><span>The
 data for todays practical can be downloaded from the Vital-It server
using the command scp. It will need to be on your local machine:</span></p><p class="c1 c3"><span></span></p><p class="c12"><span class="c6">$ scp </span><span class="c6 c7 c0"><a class="c8" href="mailto:username@prd.vital-it.ch">username</a></span><span class="c6 c7"><a class="c8" href="mailto:username@prd.vital-it.ch">@prd.vital-it.ch</a></span><span class="c6">:/scratch/cluster/monthly/kridout/summer2012_Day2.zip ./</span><span>&nbsp;</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>To
 annotate our genome we will need an automated pipeline of repeat
masking, homology searching and gene prediction. To do this we will be
using the program MAKER on the webserver MWAS.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>To avoid overloading the MWAS server, </span><span class="c0">PLEASE WORK IN PAIRS AND SEND ONLY A SINGLE MAKER REQUEST</span><span>.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>Depending
 on the type of analysis to be performed we could try to annotate all
scaffolds, or we might identify scaffolds of interest. Here we will pick
 a number of interesting scaffolds out of yesterdays assemblies</span><span>. We will do this using BLAST.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>First, we will need to login to Vital-It.</span></p><p class="c1"><span class="c6">$ ssh </span><span class="c6"><a class="c8" href="mailto:username@prd.vital-it.ch">username</a></span><span class="c6"><a class="c8" href="mailto:username@prd.vital-it.ch">@prd.vital-it.ch</a></span></p><p class="c1"><span class="c6">$ ssh dee-serv02.vital-it.ch</span></p><p class="c3 c5"><span class="c6"></span></p><p class="c1"><span>Make the directories that we will work in:</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c6">$ mkdir /scratch/cluster/weekly/</span><span class="c6 c0">username/MWAS</span></p><p class="c1"><span class="c6">$ mkdir /scratch/cluster/weekly/</span><span class="c6 c0">username/MWAS/BLAST</span></p><p class="c1"><span class="c6">$ mkdir /scratch/cluster/weekly/</span><span class="c6 c0">username/MWAS/BLAST/db</span></p><p class="c1 c3"><span class="c6 c0"></span></p><p class="c1"><span>Change into the </span><span>MWAS </span><span>folder and collect a list of interesting p</span><span>roteins</span><span>.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c6">$ cd /scratch/cluster/weekly/</span><span class="c6 c0">username/MWA</span><span class="c6 c0">S</span></p><p class="c12"><span class="c6">$ unzip /scratch/cluster/monthly/kridout/Interesting_genes.fa.zip</span></p><p class="c1 c3"><span class="c6 c0"></span></p><p class="c1"><span>We need to temporarily add the BLAST executables to your $PATH (</span><span>do not place this permanently into your .bashrc as you will need to run an older version of BLAST for another practical</span><span>):</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c6">$ source /mnt/common/Blast/Blast.bashrc</span></p><p class="c1 c3"><span class="c6"></span></p><p class="c1"><span>Next
 we need to turn the scaffolds into a BLAST database for searching.
First change into the BLAST directory and make the database:</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c6">$ cd /scratch/cluster/weekly/</span><span class="c6 c0">username/MWAS/BLAST</span></p><p class="c1"><span class="c6">$ makeblastdb -in [path</span><span class="c6">_</span><span class="c6">to</span><span class="c6">_</span><span class="c6">your</span><span class="c6">_</span><span class="c6">best</span><span class="c6">_</span><span class="c6">scaffold</span><span class="c6">s</span><span class="c6">] -dbtype nucl -out [db/db</span><span class="c6">_</span><span class="c6">name]</span></p><p class="c1 c3"><span class="c6"></span></p><p class="c1"><span>Now we blast our interesting proteins against the scaffolds assembled yester</span><span>day</span><span>&nbsp;to find those of interest.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c6">$ tblastn -db [db/db name] -query [path_to_Interesting_genes.fa] -evalue 2e-8 -out [output name] -outfmt 6 </span></p><p class="c1 c3"><span class="c6"></span></p><p class="c1"><span>Finally, we will run a script to make a fasta file of all the matching scaffolds.</span></p><p class="c1 c3"><span></span></p><p class="c12"><a href="#" name="id.3b0d7101ab1d"></a><span class="c6">$ perl /scratch/cluster/monthly/kridout/scaffoldsFromBlast.pl [path</span><span class="c6">_</span><span class="c6">to</span><span class="c6">_</span><span class="c6">your</span><span class="c6">_best_</span><span class="c6">scaffolds] [blast</span><span class="c6">_</span><span class="c6">output] &gt; [</span><span class="c6">scaffold_subset</span><span class="c6">]</span></p><p class="c1 c3"><span class="c6"></span></p><p class="c1"><span>Use </span><span class="c6">abyss-fac</span><span class="c6 c0">&nbsp;</span><span>to see how many scaffolds you have collected and the range of scaffold lengths.</span></p><p class="c1"><span>&nbsp; &nbsp; </span></p><p class="c1"><span>Exit Vital-It and Collect the file using scp.</span></p><p class="c1 c3"><span></span></p><p class="c12"><span class="c6">$ scp </span><span class="c6 c0 c7"><a class="c8" href="mailto:username@prd.vital-it.ch">username</a></span><span class="c6 c7"><a class="c8" href="mailto:username@prd.vital-it.ch">@prd.vital-it.ch</a></span><span>:</span><span class="c6">/scratch/cluster/weekly/</span><span class="c6 c0">username/BLAST/[</span><span class="c6 c0">scaffold_subset</span><span class="c6 c0">]</span><span class="c6">&nbsp;./</span><span>&nbsp;</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">The remainder of this practical will be done on your local machine.</span></p><p class="c1 c3"><span class="c0"></span></p><p class="c1"><span>&gt;Navigate to the MAKER website:</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c9"></span></p><p class="c1 c3"><span class="c9"></span></p><p class="c1"><span>&gt;Mak</span><span>e a new</span><span>&nbsp;</span><span>login</span><span>&nbsp;</span><span>using your username (i.e. student4)</span><span>&nbsp;and select the </span><span class="c2">New Job</span><span>&nbsp;tab</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>The </span><span class="c2">Denovo Annotation</span><span>&nbsp;section should contain any sequences to be annotated. In our case, this means the scaffold</span><span>&nbsp;subset that we have just collected</span><span>. &nbsp;</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Upload your scaffold file</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>EST evidence from our study species or a relative can be added. &nbsp;</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Upload the transcript file </span><span class="c2">Sinvicta_tr.fa </span><span>from the day 2 files</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>Searching
 the full swissprot database and the full repeat database will take too
much time, so a subset of proteins and repeats has been prepared for
you.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Upload the protein file </span><span class="c2">Swissprot_subset.prot</span></p><p class="c1 c3"><span class="c2"></span></p><p class="c1"><span>&gt;Upload the repeats file </span><span class="c2">Sinvicta_repeats.fa</span></p><p class="c1 c3"><span class="c2"></span></p><p class="c1"><span>&gt;Under </span><span class="c2">Denovo Annotation </span><span>select your scaffold file</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Under </span><span class="c2">EST Evidence </span><span>select the transcript file</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Under </span><span class="c2">Protein Homology Evidence </span><span>select the swissprot protein file</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Under the </span><span class="c2">Configure Repeat Masking </span><span>section select the repeats file</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Select the course training files for SNAP (we will not be using AUGUSTUS and GeneMark, although this is </span><span>recommended</span><span>)</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>Very
 short scaffolds might still contain whole genes, however there is a
high likelihood of broken genes in these regions. In the interest of
time and accuracy we will skip them.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Under </span><span class="c2">Annotation Properties </span><span>set the minimum contig length to 1000</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Select </span><span class="c2">Add Job to Queue </span><span>to launch MAKER</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>This process will take several </span><span>hours</span><span>. You can monitor the status of your job from the </span><span class="c2">Running Jobs </span><span>tab.</span></p><p class="c1 c3"><span></span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">MWAS Questions</span></p><p class="c1 c3"><span class="c0"></span></p><p class="c1"><span>You
 should have now started the process of genome annotation. Generating a
set of reliable genes is not a trivial task. It is very important to
understand the potential problems with genes predicted with automated
methods. Consider the data that you have produced this far and answer
the following questions:</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q1.</span><span>&nbsp;Genome/transcript
 assembly does not yet produce scaffolds that are guaranteed to be
accurate. What effects could misassemblies in the data have on genome
annotation? &nbsp;</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q2. </span><span>Errors
 in the EST or protein annotation databases will be perpetuated through
this annotation process. &nbsp;This is a problem for all homology based
annotation methods. How might you try to deal with this uncertainty?</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q3. </span><span>What are the pros and cons of a homology based gene finding approach?</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q4. </span><span>What are the pros and cons of a denovo prediction approach (e.g. AUGUSTUS) </span><span>for</span><span>&nbsp;gene finding?</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q5. </span><span>What are the potential biases generated by predicting genes in this way?</span></p><p class="c1 c3"><span></span></p><p class="c1 c3"><span class="c0"></span></p><p class="c1"><span class="c0">WHILE MAKER IS RUNNING CHECK THAT YOU HAVE APPOLO INSTALLED AND FUNCTIONING</span></p><p class="c1 c3"><span class="c0"></span></p><p class="c1"><span>If Apollo is not installed, you should download and</span><span>&nbsp;install it before we continue with the Apollo practical. </span><span>Downloading it from the local server will be fastest.</span></p><p class="c1 c3"><span></span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c10 c0">Apollo manual gene editing</span></p><p class="c1 c3"><span class="c10 c0"></span></p><p class="c1"><span>Sometimes
 we might want to view or manually edit specific genes. To perform this
task, programs such as IGV or Apollo can be used. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Download all data from MWAS (under </span><span class="c2">View Results</span><span>)</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Open Apollo and load the gff file (this is gff3 format) of all scaffolds downloaded from MAKER. Untick the box labeled </span><span class="c2">Embedded sequence </span><span>and select the original scaffold file</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>The
 top 2 panes represent the forwards strand and the bottom 2 are the
reverse. Results from the different prediction methods are displayed in
the black panes and the final annotations in the blue.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Click on the squares (predictions) in the black panes to see which programs have produced the different results</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;To color the different prediction methods, right click on the prediction (in the black panel) and select &nbsp;</span><span class="c2">Change color of this feature type</span></p><p class="c1 c3"><span class="c2"></span></p><p class="c1"><span>Scaffold and gff3 files have been prepared for you to examine and compare with your own annotations. Open the fasta file </span><span class="c2">Sinvicta_good_scaffolds.fa </span><span>and the gff3 file </span><span class="c2">Sinvicta_good_predictions.gff3 </span><span>in Apollo.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q6. </span><span>What are the main differences that you see between the files (do not try to investigate individual genes)?</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q7. </span><span>Based
 on these results, what can you say about which programs over predict
and which under predict? What do you think of the reliability of these
programs?</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>The
 gff3 file given to you contains examples of correctly and incorrectly
annotated genes. In some places genes have been forced together, or
separated at introns. Some genes also have incorrect splice sites. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q8. </span><span>Using Apollo, identify likely genes that might have been badly annotated. Why did you choose these genes?</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>Using
 tools such as Apollo we can perform manual editing of gene predictions.
 Splice sites occur at the intron/exon boundary and are highly
conserved. Apollo highlights the ends of exons that do not have the
necessary splicing sites using an orange triangle:</span></p><p class="c1 c3"><span></span></p><p class="c1"><img src="MWAS%20practical.docx_files/pubimage.png" height="277" width="486"><span>&nbsp;</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>Acceptable splice sites can be colored as in the above example. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Right click on the nucleotide sequence to select </span><span class="c2">color by splice site potential &nbsp;</span><span>(you will not see the nucleotides until you have zoomed in sufficiently far)</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>Simply choosing the closest splice site is not a reliable method for fixing this prediction. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q9. </span><span>How might you determine the true splice positions of a predicted gene?</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>The gene </span><span class="c0">maker-Sign00006-snap-gene-0.10-mRNA-1 </span><span>contains a number of incorrect splicing sites identified by Apollo.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Select this gene under the </span><span class="c2">Annotation</span><span>&nbsp;tab for a better view and zoom in to examine the sequence</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q10. </span><span>Do you think that this gene represents a real protein? Explain your answer. What else might this gene be? </span></p><p class="c1 c3"><span></span></p><p class="c1"><span>A protein domain scan on the whole genome determined that t</span><span>he fire ant genome contains multiple Vitellogenins. This is potentially interesting because </span><span>the single Vitellogenin of the honey bee plays roles in determining lifespan and division of labor (Amdam et al 2012). </span><span>Protein2genome sugge</span><span>st there are three vitellogenins in our</span><span>&nbsp;scaffold:</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">maker-Sign00006-snap-gene-1.41-mRNA-1</span></p><p class="c1"><span class="c0">maker-Sign00006-snap-gene-1.42-mRNA-1</span></p><p class="c1"><span class="c0">maker-Sign00006-snap-gene-1.43-mRNA-1</span></p><p class="c1 c3"><span class="c0"></span></p><p class="c1"><span class="c0">Q11. </span><span>What evidence supports these predictions?</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>Gene </span><span class="c0">maker-Sign00006-snap-gene-1.40-mRNA-1 </span><span>is considerably shorter, but has also matched this protein. Experimental evidence suggests that there is indeed a 4</span><span class="c11">th</span><span>&nbsp;Vitellogenin gene, despite the missing EST evidence. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span>A closer examination of the BLAST evidence suggests that this gene has been truncated by the prediction software.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q12. </span><span>Why do you think this is? </span></p><p class="c1 c3"><span></span></p><p class="c1"><span>We
 will attempt to extend this gene. Given more time we would examine the
BLAST results against a number of Vitellogenin homologues to determine
the correct splice, start and stop sites. Here, we will try to combine
the BLASTX prediction and EST results. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span>First, we should duplicate the gene(s) that we intend to work on in case we make a mistake.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Double click on each of these genes to select it as a whole. Right click on the selected gene and choose </span><span class="c2">Duplicate transcript</span></p><p class="c1 c3"><span class="c2"></span></p><p class="c1"><span>The
 first new BLAST exon (from right to left) does not begin at a splice
site, however this can be extended using EST evidence which does.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Right click in the blue panel underneath the nucleotide that your new exon should start at. Select </span><span class="c2">Create new annotation -&gt; gene</span></p><p class="c1 c3"><span class="c2"></span></p><p class="c1"><span>&gt;Set the new gene length to a single base</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Find the end of your exon and repeat this process</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Hold the </span><span class="c2">shift </span><span>key and click on the two new genes to select them both. &nbsp;Right click on one of these and select </span><span class="c2">Merge transcripts</span></p><p class="c1 c3"><span class="c2"></span></p><p class="c1"><span>&gt;Double click on the new gene to select the whole thing. Right click on this gene and select </span><span class="c2">Merge exons</span></p><p class="c1 c3"><span class="c2"></span></p><p class="c1"><span>You
 have built your first new exon. Apollo will place an orange triangle at
 the end of the gene if the splice site was not correct. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Check your gene for correct splicing and frame. If the frame is incorrect there will be stop codons. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;If
 your exon is correctly spliced, you should try to join it to the rest
of the gene. Select the gene and the new exon, right click and </span><span class="c2">Merge transcripts</span></p><p class="c1 c3"><span class="c2"></span></p><p class="c1"><span>&gt;You
 may now find that the splicing does not work. This is because the
original gene was built to end at a stop codon. Use the EST and BLAST
evidence to find the correct splice site. Make sure that the correct
reading frame is maintained</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>To move the end of the exon by small amounts, right click on the transcript and select </span><span class="c2">Exon detail editor. </span><span>Navigate
 to the necessary part of the sequence (remember that the sequence will
be displayed from left to right, where as these genes on the reverse
strand are viewed in the pale blue panel from right to left).</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>&gt;Repeat this process until you reach the end of the gene. Remember to stop the gene at a stop codon.</span></p><p class="c1 c3"><span></span></p><p class="c1"><span>Predicted genes:</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">maker-Sign00006-snap-gene-6.41-mRNA-1</span></p><p class="c1"><span class="c0">maker-Sign00006-snap-gene-6.43-mRNA-1</span></p><p class="c1 c3"><span class="c0"></span></p><p class="c1"><span>Both
 map in parts to the same EST. It is possible that these genes have been
 fragmented by Apollo and belong to the same transcript. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0">Q13. </span><span>How might you try to determine whether these are the same gene?</span></p><p class="c1 c3"><span></span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c0 c10">Extension</span></p><p class="c1 c3"><span class="c10 c0"></span></p><p class="c1"><span>Despite
 possible errors in the databases, genes can be annotated using
automated BLAST searching. A commonly used tool for automated annotation
 is BLAST2GO, which allows the user to annotate using a gene name,
domain, function, ontology and pathway. &nbsp;This can be very useful
for downstream processes. </span></p><p class="c1 c3"><span></span></p><p class="c1"><span>If you have finished editing your genes, choose a subset (maximum 10) and explore the functionality of BLAST2GO:</span></p><p class="c1 c3"><span></span></p><p class="c1"><span class="c9"><a class="c8" href="http://www.blast2go.com/b2glaunch">http://www.blast2go.com/b2glaunch</a></span></p></div><div id="footer"><span>Published by <a target="_blank" title="Learn more about Google Drive" href="https://docs.google.com/">Google Drive</a></span><span class="dash">–</span><a href="https://docs.google.com/abuse?id=10P7oSJ3M_Jg6EzSPbsKx1HV6WQprOURcpvIBxa5Yk6g">Report Abuse</a><span class="dash">–</span><span>Updated automatically every 5 minutes</span></div><script type="text/javascript">
(function(){if(window.jstiming){window.jstiming.a={};window.jstiming.b=1;var e=function(b,a,d){var c=b.t[a],g=b.t.start;if(c&&(g||d))return c=b.t[a][0],void 0!=d?g=d:g=g[0],c-g},n=function(b,a,d){var c="";window.jstiming.pt&&(c+="&srt="+window.jstiming.pt,delete window.jstiming.pt);try{window.external&&window.external.tran?c+="&tran="+window.external.tran:window.gtbExternal&&window.gtbExternal.tran?c+="&tran="+window.gtbExternal.tran():window.chrome&&window.chrome.csi&&(c+="&tran="+window.chrome.csi().tran)}catch(g){}var f=
window.chrome;if(f&&(f=f.loadTimes)){f().wasFetchedViaSpdy&&(c+="&p=s");if(f().wasNpnNegotiated){var c=c+"&npn=1",k=f().npnNegotiatedProtocol;k&&(c+="&npnv="+(encodeURIComponent||escape)(k))}f().wasAlternateProtocolAvailable&&(c+="&apa=1")}var l=b.t,s=l.start,f=[],k=[],h;for(h in l)if("start"!=h&&0!=h.indexOf("_")){var m=l[h][1];m?l[m]&&k.push(h+"."+e(b,h,l[m][0])):s&&f.push(h+"."+e(b,h))}delete l.start;if(a)for(var p in a)c+="&"+p+"="+a[p];(a=d)||(a="https:"==document.location.protocol?"https://csi.gstatic.com/csi":
"http://csi.gstatic.com/csi");return[a,"?v=3","&s="+(window.jstiming.sn||"_s")+"&action=",b.name,k.length?"&it="+k.join(","):"",c,"&rt=",f.join(",")].join("")};window.jstiming.getReportUri=n;var q=function(b,a,d){b=n(b,a,d);if(!b)return"";a=new Image;var c=window.jstiming.b++;window.jstiming.a[c]=a;a.onload=a.onerror=function(){window.jstiming&&delete window.jstiming.a[c]};a.src=b;a=null;return b};window.jstiming.report=function(b,a,d){if("prerender"==document.webkitVisibilityState){var c=!1,g=function(){if(!c){a?
a.prerender="1":a={prerender:"1"};var f;"prerender"==document.webkitVisibilityState?f=!1:(q(b,a,d),f=!0);f&&(c=!0,document.removeEventListener("webkitvisibilitychange",g,!1))}};document.addEventListener("webkitvisibilitychange",g,!1);return""}return q(b,a,d)};window.jstiming.reportDone=function(b){if(window.jstiming.b<=(b||1))return!1;for(var a in window.jstiming.a)return!1;return!0};var r=function(b,a,d,c){return 0<d?(c?b.tick(a,c,d):b.tick(a,"",d),!0):!1};window.jstiming.getNavTiming=function(b){if(window.performance&&
window.performance.timing){var a=window.performance.timing;r(b,"_dns",a.domainLookupStart)&&r(b,"dns_",a.domainLookupEnd,"_dns");r(b,"_con",a.connectStart)&&r(b,"con_",a.connectEnd,"_con");r(b,"_req",a.requestStart)&&r(b,"req_",a.responseStart,"_req");r(b,"_rcv",a.responseStart)&&r(b,"rcv_",a.responseEnd,"_rcv");if(r(b,"_ns",a.navigationStart)){r(b,"ntsrt_",a.responseStart,"_ns");r(b,"nsfs_",a.fetchStart,"_ns");var d=window.external&&window.external.startE;!d&&(window.chrome&&window.chrome.csi)&&
(d=Math.floor(window.chrome.csi().startE));d&&(r(b,"_se",d),r(b,"sens_",a.navigationStart,"_se"));r(b,"ntplt0_",a.loadEventStart,"_ns");r(b,"ntplt1_",a.loadEventEnd,"_ns")}}}};})()
</script><script type="text/javascript">KX_timer.tick('tl'); if (document.location.protocol == 'https:') {window.jstiming.report(KX_timer, undefined , 'https://gg.google.com/csi');} else {window.jstiming.report(KX_timer);}</script></body></html>
