---
title: 'Fourmidable: a database for ant genomics'
layout: publication
permalink: /publications/ant-genome-database/
---

<br />

<p class="info-text">Published: 06 January 2009</p>

# Fourmidable: a database for ant genomics

<h5 class="info-text my-4">Yannick Wurm, Paolo Uva, Frédéric Ricci, John Wang, Stephanie Jemielity, Christian Iseli, Laurent Falquet & Laurent Keller</h5>

[*BMC Genomics*](https://dx.doi.org/10.1186/1471-2164-10-5), 2009, 10:5

<br />

## Abstract
<hr />

#### Background

Fourmidable is an infrastructure to curate and share the emerging genetic, molecular, and functional genomic data and protocols for ants.

#### Description

The Fourmidable assembly pipeline groups nucleotide sequences into clusters before independently assembling each cluster. Subsequently, assembled sequences are annotated via Interproscan and BLAST against general and insect-specific databases. Gene-specific information can be retrieved using gene identifiers, searching for similar sequences or browsing through inferred Gene Ontology annotations. The database will readily scale as ultra-high throughput sequence data and sequences from additional species become available.

#### Conclusion

Fourmidable currently houses EST data from two ant species and microarray gene expression data for one of these. Fourmidable is publicly available at [http://fourmidable.unil.ch](//fourmidable.unil.ch){:target="_blank"}

<br />

## Background
<hr />

Ants are important model species for sociobiology and behavioral ecology <span class="info-text">[1]</span>. Life in an ant colony is marked by cooperation, but it also entails conflicts. Both aspects have been studied extensively to understand the prerequisites for social behavior and to test the kin selection theory (e.g. <span class="info-text">[2, 3]</span>). New molecular and genomic techniques are making it possible to identify the genes underlying social behavior in ants and other social insects <span class="info-text">[4–11]</span> as well as other fascinating aspects of social life including self-organization, life-history evolution, division of labor, and developmental plasticity <span class="info-text">[12–15]</span>. The extraordinary complexity and vast information content generated by modern genomic techniques can be overwhelming. To take full advantage of such techniques requires appropriate bioinformatics tools.

To provide a central repository for the emerging ant genomic data, we developed Fourmidable, a web-accessible, user-friendly tool. Fourmidable currently provides detailed assembly and annotation of nucleic acid sequences from ants, a repository for ant microarray experiments and a platform to share ant-specific molecular biology protocols.

<br />

## Construction and content
<hr />

Formidable contains publicly available sequence and gene expression data for ants and analyses of these data (summarized in <span class="info-text">Table 1</span>).

<div class="publication-image mb-3">
  <h5><em>Table 1 – <strong>Data Content in Fourmidable (October 2008).</strong></em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/table-1-data-content-in-fourmidable.webp" />
    <img src="/img/publications/table-1-data-content-in-fourmidable.jpg" width="1092" height="592" alt="Data Content in Fourmidable" />
  </picture>
</div>

#### Computation and Database Design

Fourmidable analyses are carried out via custom Perl scripts and publicly available software. Annotation and assembly information is stored in a MySQL database while sequences and BLAST <span class="info-text">[16]</span> results are kept in indexed text files for rapid retrieval while limiting database size. Data are stored separately for different species [see <span class="info-text">Additional file 1</span>]. Computationally intensive tasks are parallelized on the Vital-IT high-performance computing cluster [<span class="info-text">[17]</span>, <span class="info-text">Additional file 2</span>]. Fourmidable should thus readily handle large amounts of additional data. Data is accessible to web users via a PHP/Apache-based interface hosted by the Swiss Institute of Bioinformatics.

#### Nucleotide Sequence Data Preparation, Assembly, and Annotation

Fourmidable currently processes nucleotide sequences for the red fire ant *Solenopsis invicta* and the black garden ant *Lasius niger*. When available, raw .ab1 or .scf trace files are converted via Phred <span class="info-text">[18]</span> to FASTA nucleotide and quality score files. Additional sequences for which trace files cannot be obtained are downloaded from Genbank; quality score files are generated for these sequences with an arbitrary Phred quality score of 25. All input sequences are cleaned using Lucy <span class="info-text">[19]</span>, DUST [Tatusov and Lipman, unpublished], RepeatMasker <span class="info-text">[20]</span> and CrossMatch <span class="info-text">[21]</span> to respectively remove low-quality regions and sequences, low-complexity regions, interspersed repeats, and sequences from bacteria, organelles or cloning vectors. Cleaned sequences are then compared via reciprocal BLAST <span class="info-text">[16]</span>, and subsequently similar sequences are grouped into clusters. Within each cluster, sequences are independently assembled via CAP3 <span class="info-text">[22]</span>. This circumvents memory constraints that CAP3 would face if attempting a global assembly with large numbers of sequences. The output from clustering and CAP3 assemblies are contigs (each is the consensus of several assembled sequences) and singlets (sequences that did not assemble with others). All sequences are subsequently annotated as follows.

All sequences are compared to the non-redundant protein database <span class="info-text">[23]</span> via BLASTX as well as to several insect-specific databases via TBLASTX, BLASTX and BLASTN. Gene Ontology (GO) annotation of the strongest of the top five BLASTX hits to the non-redundant protein database is carried over to ant sequences.

To determine possible peptide sequences, we compute all six possible translations of transcriptome sequences with the potential to encode sequences longer than 30 amino acids. We do not use an *ab initio* gene prediction program such as ESTScan <span class="info-text">[24]</span> because the sensitivity of such programs is limited by the absence of solid training data for ants. Instead, all potential open reading frames are annotated via Interproscan <span class="info-text">[25]</span>. Some Interproscan hits directly provide GO annotation of ant sequences, complementing the BLASTX-inferred GO annotation mentioned above.

The BLAST, Interproscan and GO annotations are updated every two months or when new sequence data is added to the assembly pipeline.

At several steps during this assembly and annotation pipeline, bioinformatics software was run with parameters that differed from default parameters [see <span class="info-text">Additional file 3</span>], as determined by using reduced test datasets.

#### Gene Expression

Fourmidable is linked to the GEDAI gene expression database [Robin Liechti, unpublished]. Storage and simple analysis using Bioconductor packages <span class="info-text">[26]</span> is possible for data from single-color and two-color spotted microarrays, as well as for Affymetrix and Illumina microarrays.

<br />

## Utility and discussion
<hr />

As of October 2008, Fourmidable contains nucleotide sequence data for the fire ant *S. invicta* and the black garden ant *L. niger* as well as gene expression data for *S. invicta*. Currently accessible data are summarized in <span class="info-text">Table 1</span>. Sequencing, gene expression profiling, and genotyping data are rapidly expanding and will be added as they become publicly available. Fourmidable's home page <span class="info-text">[27]</span> centralizes links and search facilities to access Fourmidable's data and tools.

#### Sequence Search

There are several manners of accessing sequence information in Fourmidable. First, single sequences can be searched by species as well as by partial identifiers for input sequences or assembled contigs. Second, lists of identifiers can be used for searching. Third, user-supplied sequence data can be used for BLAST similarity searches against sequences in Fourmidable. Finally, users can navigate inferred Gene Ontology annotations for biological processes, cellular components, and molecular functions using the AmiGO browser <span class="info-text">[28]</span>. The first two search manners result in tables as described below. BLAST searching and GO browsing produce lists of sequence identifiers that can be used as inputs for the first two search manners.

####  Sequence Information

Sequence searches result in tables with one line per sequence for easy access to sequence annotation (see <span class="info-text">Figure 1</span>). In particular:

<div class="publication-image mb-3">
  <h5><em>Figure 1</em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/table-of-sequence-search-results.webp" />
    <img src="/img/publications/table-of-sequence-search-results.jpg" width="1092" height="771" alt="Table of sequence search results" />
  </picture>
  <p class="mt-2">
    <b>Table of sequence search results</b>. For each result, the following are shown from left to right if applicable: sequence identifiers for raw and assembled sequence (these respectively link to the raw datafiles and assembly information); links to raw and assembled sequence in Fasta format; links to the results of BLAST against different databases (red buttons if E-value ≤ 10<sup>-5</sup>; blue buttons if 0.01 > E-value > 10<sup>-5</sup>; bx, tbx and bn respectively indicate BLASTX, TBLASTX and BLASTN algorithms); a description as inferred from BLASTX against the non-redundant protein database; and a link to Interproscan annotation.
  </p>
</div>

- Clicking on an identifier in the "Raw Sequence" column provides information on how that sequence was obtained and allows users to download the raw sequence. Tracefiles can be viewed with the Baylor College of Medicine Trace Viewer <span class="info-text">[29]</span> or downloaded.

- Cleaned FASTA-format sequence can be downloaded for individual singlets and contigs.

- If the sequence is part of an assembled contig, the "Assembled Contig" identifier links to a display of the consensus sequence and the relevant input sequences as well as their quality scores. Additionally, a multiple sequence alignment highlights nucleotide polymorphisms within the consensus sequence.

- BLAST results between the sequence of interest and sequences from the non-redundant protein database and several insect nucleotide and protein databases are summarized by blue and red buttons, indicating weak (0.01 > E-value > 10<sup>-5</sup>) and stronger similarity (E-value ≤ 10<sup>-5</sup>) respectively. Clicking on a button displays the complete BLAST report.

- An additional link to Interproscan results and six-frame protein translations is displayed if Interproscan annotation is available.

#### Additional Features

A convenient repository is available for ant molecular biology protocols (commonly in .doc or .pdf formats). New or revised protocols can be added via an upload form. Fourmidable also supports upload of result files from microarray gene expression experiments. The GEDAI platform allows straightforward sharing of microarray results and performing simple microarray analyses (including preprocessing, direct and indirect two-sample comparisons, 2 × 2 factorial and gene set enrichment analyses). GEDAI also provides summaries of the expression levels of specific microarray probe identifiers across multiple microarray experiments. Finally, Fourmidable provides download links to individual files containing all raw or assembled sequences, as well as sequence annotation in text format [see also <span class="info-text">Additional file 4</span>].

#### Past Applications

The sequence assembly and annotation information provided by Fourmidable has already proved useful in several published studies <span class="info-text">[14, 30]</span>. Most recently, Fourmidable's data helped J. Wang and colleagues to characterize genes that are differently expressed between workers from two alternative social forms of fire ants <span class="info-text">[6]</span>.

#### Outlook

Fourmidable was initially developed as a private database in Lausanne. Recently it has been updated and made publicly accessible because of increasing interest in ant molecular research. To further develop Fourmidable, several primary investigators in the USA have submitted grant applications. This should lead to improved integration of gene expression data with sequence annotation, as well as support for genetic mapping and linkage data. When large amounts of genomic sequence become available for ants, the current approach for assembly and annotation may become computationally unrealistic. An alternative may be to adapt existing genome assembly, annotation and browsing tools.

<br />

## Conclusion
<hr />

Fourmidable is a web-based database centralizing genomic resources for ants. As of October 2008, it contains raw sequence, assembled sequence, expression and annotation data for the fire ant *S. invicta* and the black garden ant *L. niger*, as well as ant-specific molecular biology protocols. Fourmidable will readily expand to accommodate additional data from these and additional species.

<br />

## Availability and requirements
<hr />

Fourmidable is publicly available <span class="info-text">[27]</span>. It has been tested with Firefox 2 and 3, Safari 3 and Internet Explorer 7. The web interface is valid HTML 4.01 Transitional and CSS 2.1.

<br />

## References
<hr />

1. Hölldobler B, Wilson EO: The Ants. 1990

2. Sundstrom L, Chapuisat M, Keller L: Conditional manipulation of sex ratios by ant workers: a test of kin selection theory. *Science*. 1996, **274**: 993-995. 10.1126/science.274.5289.993.

3. Bourke AFG, Franks NR: Social Evolution in Ants. 1995

4. Robinson GE, Grozinger CM, Whitfield CW: Sociogenomics: social life in molecular terms. *Nat Rev Genet.* 2005, **6** (4): 257-270. 10.1038/nrg1575.

5. Robinson GE: Integrative animal behavior and sociogenomics. *TREE*. 1999, **14**: 202-205.

6. Wang J, Ross KG, Keller L: Genome-wide expression patterns and the genetic architecture of a fundamental social trait. *PLoS Genet.* 2008, **4**: e1000127-10.1371/journal.pgen.1000127.

7. Keller L, Ross KG: Selfish genes: a green beard in the red fire ant. *Nature*. 1998, **394**: 573-575. 10.1038/29064.

8. Whitfield CW, Cziko A, Robinson GE: Gene expression profiles in the brain predict behavior in individual honey bees. *Science*. 2003, **302**: 296-299. 10.1126/science.1086807.

9. Grozinger CM, Sharabash NM, Whitfield CW, Robinson GE: Pheromone-mediated gene expression in the honey bee brain. *Proc Nat Acad Sci USA*. 2003, **100** (Suppl 2): 14519-14525. 10.1073/pnas.2335884100.

10. Nelson CM, Ihle KE, Fondrk MK, Page RE, Amdam GV: The gene vitellogenin has multiple coordinating effects on social organization. *PLoS Biol.* 2007, **5**: e62-10.1371/journal.pbio.0050062.

11. Wurm Y, Wang J, Keller L: Behavioral genomics: A, bee, C, G, T. *Curr Biol.* 2007, **17**: R51-R53. 10.1016/j.cub.2006.12.019.

12. Abouheif E, Wray GA: Evolution of the gene network underlying wing polyphenism in ants. *Science*. 2002, **297**: 249-252. 10.1126/science.1071468.

13. Jemielity S, Kimura M, Parker KM, Parker JD, Cao X, Aviv A, Keller L: Short telomeres in short-lived males: what are the molecular and evolutionary causes?. *Aging Cell*. 2007, **6**: 225-233. 10.1111/j.1474-9726.2007.00279.x.

14. Graff J, Jemielity S, Parker JD, Parker KM, Keller L: Differential gene expression between adult queens and workers in the ant Lasius niger. *Mol Ecol.* 2007, **16**: 675-683. 10.1111/j.1365-294X.2007.03162.x.

15. Valles SM, Strong CA, Hunter WB, Dang PM, Pereira RM, Oi DH, Williams DF: Expressed sequence tags from the red imported fire ant, Solenopsis invicta: annotation and utilization for discovery of viruses. *J of Inv Pathol.* 2008, **99**: 74-81. 10.1016/j.jip.2008.01.004.

16. Altschul S, Madden T, Schaffer A, Zhang J, Zhang Z, Miller W, Lipman D: Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. *Nucleic Acids Res.* 1997, **25**: 3389-3402. 10.1093/nar/25.17.3389.

17. Swiss Institute of Bioinformatics: Vital-IT center for high-performance computing. [https://www.vital-it.ch/](https://www.vital-it.ch/){:target="_blank"}

18. Ewing B, Hillier L, Wendl MC, Green P: Base-calling of automated sequencer traces using phred. I. Accuracy assessment. *Genome Res.* 1998, **8**: 175-185.

19. Chou H, Holmes MH: DNA sequence quality trimming and vector removal. *Bioinformatics*. 2001, **17**: 1093-1104. 10.1093/bioinformatics/17.12.1093.

20. Smit A, Hubley R, Green P: RepeatMasker Open-3.2.2. [http://www.repeatmasker.org](//www.repeatmasker.org){:target="_blank"}

21. Cross_Match. [http://www.phrap.org/](//www.phrap.org/){:target="_blank"}

22. Huang X, Madan A: Cap3: a DNA sequence assembly program. *Genome Res.* 1999, **9**: 868-877. 10.1101/gr.9.9.868.

23. Pruitt KD, Tatusova T, Maglott DR: NCBI reference sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. *Nucleic Acids Res.* 2007, **35** (Database issue): D61-D65. 10.1093/nar/gkl842.

24. Iseli C, Jongeneel CV, Bucher P: ESTScan: a program for detecting, evaluating, and reconstructing potential coding regions in EST sequences. *Proc Int Conf Intell Syst Mol Biol.* 1999, 138-148.

25. Mulder NJ, Apweiler R, Attwood TK, Bairoch A, Bateman A, Binns D, Bork P, Buillard V, Cerutti L, Copley R, Courcelle E, Das U, Daugherty L, Dibley M, Finn R, Fleischmann W, Gough J, Haft D, Hulo N, Hunter S, Kahn D, Kanapin A, Kejariwal A, Labarga A, Langendijk-Genevaux PS, Lonsdale D, Lopez R, Letunic I, Madera M, Maslen J, McAnulla C, McDowall J, Mistry J, Mitchell A, Nikolskaya AN, Orchard S, Orengo C, Petryszak R, Selengut JD, Sigrist CJA, Thomas PD, Valentin F, Wilson D, Wu CH, Yeats C: New developments in the Interpro database. *Nucleic Acids Res.* 2007, **35** (Suppl 1): D224-228. 10.1093/nar/gkl841.

26. Gentleman RC, Carey VJ, Bates DM, Bolstad B, Dettling M, Dudoit S, Ellis B, Gautier L, Ge Y, Gentry J, Hornik K, Hothorn T, Huber W, Iacus S, Irizarry R, Leisch F, Li C, Maechler M, Rossini AJ, Sawitzki G, Smith C, Smyth G, Tierney L, Yang JYH, Zhang J: Bioconductor: open software development for computational biology and bioinformatics. *Genome Biol.* 2004, **5**: R80-10.1186/gb-2004-5-10-r80.

27. Fourmidable. [http://fourmidable.unil.ch](//fourmidable.unil.ch){:target="_blank"}

28. The GO Consortium: The Gene Ontology (GO) project in 2006. *Nucleic Acids Res.* 2006, **34** (Suppl 1): D322-326. 10.1093/nar/gkj021.

29. Durbin KJ, Baylor College of Medicine Human Genome Sequencing Center: Baylor College of Medicine Trace Viewer. [http://www.hgsc.bcm.tmc.edu/downloads/software/trace_viewer/](//www.hgsc.bcm.tmc.edu/downloads/software/trace_viewer/){:target="_blank"}

30. Wang J, Jemielity S, Uva P, Wurm Y, Graff J, Keller L: An annotated cDNA library and microarray for large-scale gene-expression studies in the ant Solenopsis invicta. *Genome Biol.* 2007, **8**: R9-10.1186/gb-2007-8-1-r9.

<br />

## Acknowledgements
<hr />

We thank Marc Robinson-Rechavi and two anonymous reviewers for critical reading of the manuscript; Robin Liechti, Volker Flegel, Peter Sperisen, Lorenzo Cerruti and Vital-IT <span class="info-text">[17]</span> at the Swiss Institute of Bioinformatics [https://www.sib.swiss/](https://www.sib.swiss/){:target="_blank"} for advice and access to bioinformatics resources; Cedric Wurm for designing the icons used on Fourmidable's homepage.

This work was supported by the AR and J Leenards Foundation; the Swiss National Science Foundation; the Herbette Foundation; and the Rectorate of the University of Lausanne. LF is supported by the Swiss State Secretariat for Education and Research.

<br />

## Author information
<hr />

#### Authors and Affiliations

**Department of Ecology and Evolution, Biophore, University of Lausanne, CH-1015, Lausanne, Switzerland**

Yannick Wurm, Frédéric Ricci, John Wang & Laurent Keller

**Istituto di Ricerche di Biologia Molecolare, Merck Research Laboratories, 00040, Pomezia, Rome, Italy**

Paolo Uva

**Institut for Infectious Diseases, University of Bern, CH-3010, Bern, Switzerland**

Stephanie Jemielity

**Ludwig Institute for Cancer Research, CH-1015, Lausanne, Switzerland**

Christian Iseli

**Swiss Institute of Bioinformatics, CH-1015, Lausanne, Switzerland**

Christian Iseli & Laurent Falquet

#### Corresponding author

Correspondence to [Yannick Wurm](mailto:yannick.wurm@unil.ch).

<br />

## Additional information
<hr />

#### Authors' contributions

LF, PU, YW, FR, SJ and JW designed the database. YW, PU, FR and CI developed the database. LK supported the work. YW drafted the manuscript. LK, LF, JW and SJ revised the manuscript. All authors read and approved the final manuscript.

<br />

## Electronic supplementary material
<hr />

- [12864_2008_1889_MOESM1_ESM.rtf](/publications/2009-ant-genome-database/12864_2008_1889_MOESM1_ESM.rtf) <br />
**Additional file 1**: Notes on implementation. We provide several details about decisions made relative to the implementation of the database, and the assembly pipeline. (.RTF, 2 KB)

- [12864_2008_1889_MOESM2_ESM.rtf](/publications/2009-ant-genome-database/12864_2008_1889_MOESM2_ESM.rtf) <br />
**Additional file 2**: List of tasks parallelized on the Swiss Institute of Bioinformatics Vital-IT computing cluster. Some tasks were parallelized for increased execution speed. (.RTF, 1 KB)

- [12864_2008_1889_MOESM3_ESM.rtf](/publications/2009-ant-genome-database/12864_2008_1889_MOESM3_ESM.rtf) <br />
**Additional file 3**: List of software parameters that differ from default. For the assembly and annotation pipelines, default parameters were sometimes unsatisfactory. This table summarizes the parameters used when they differed from default. (.RTF, 6 KB)

- [12864_2008_1889_MOESM4_ESM.rtf](/publications/2009-ant-genome-database/12864_2008_1889_MOESM4_ESM.rtf) <br />
**Additional file 4**: List of data available in text format. Some of the data in Fourmidable can be downloaded in text format. (.RTF, 1 KB)

<br />

## Rights and permissions
<hr />

This article is published under license to BioMed Central Ltd. This is an Open Access article distributed under the terms of the Creative Commons Attribution License ([https://creativecommons.org/licenses/by/2.0/](https://creativecommons.org/licenses/by/2.0/){:target="_blank"}), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.

[Reprints and Permissions](https://s100.copyright.com/AppDispatchServlet?title=Fourmidable%3A%20a%20database%20for%20ant%20genomics&author=Yannick%20Wurm%20et%20al&contentID=10.1186%2F1471-2164-10-5&copyright=Wurm%20et%20al%3B%20licensee%20BioMed%20Central%20Ltd.&publication=1471-2164&publicationDate=2009-01-06&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY){:target="_blank"}

<br />
