---
title: 'The Molecular Clockwork of the Fire Ant Solenopsis invicta'
layout: publication
permalink: /publications/fire-ant-clock-genes-evolution-period-cycle-cryptochrome-vrille/
---

<br />

<p class="info-text">Published: 13 November 2012</p>

# The Molecular Clockwork of the Fire Ant *Solenopsis invicta*

<h5 class="info-text my-4">Krista K. Ingram, Alexander Kutowoi, Yannick Wurm, DeWayne Shoemaker, Rudolf Meier, Guy Bloch</h5>

[*PloS One*](https://doi.org/10.1371/journal.pone.0045715), 2012, 7:e45715

<br />

## Abstract
<hr />

The circadian clock is a core molecular mechanism that allows organisms to anticipate daily environmental changes and adapt the timing of behaviors to maximize efficiency. In social insects, the ability to maintain the appropriate temporal order is thought to improve colony efficiency and fitness. We used the newly sequenced fire ant (*Solenopsis invicta*) genome to characterize the first ant circadian clock. Our results reveal that the fire ant clock is similar to the clock of the honeybee, a social insect with an independent evolutionary origin of sociality. Gene trees for the eight core clock genes, *period*, *cycle*, *clock*, *cryptochrome-m*, *timeout*, *vrille*, *par domain protein 1* & *clockwork orange*, show ant species grouping closely with honeybees and *Nasonia* wasps as an outgroup to the social Hymenoptera. Expression patterns for these genes suggest that the ant clock functions similar to the honeybee clock, with period and cry-m mRNA levels increasing during the night and *cycle* and *clockwork orange* mRNAs cycling approximately anti-phase to *period*. Gene models for five of these genes also parallel honeybee models. In particular, the single ant *cryptochrome* is an ortholog of the mammalian-type (cry-m), rather than *Drosophila*-like protein (cry-d). Additionally, we find a conserved VPIFAL C-tail region in *clockwork orange* shared by insects but absent in vertebrates. Overall, our characterization of the ant clock demonstrates that two social insect lineages, ants and bees, share a similar, mammalian-like circadian clock. This study represents the first characterization of clock genes in an ant and is a key step towards understanding socially-regulated plasticity in circadian rhythms by facilitating comparative studies on the organization of circadian clockwork.

<br />

## Introduction
<hr />

The circadian clock is a core molecular mechanism that allows organisms to anticipate daily environmental changes and adapt the timing of behaviors to maximize their efficiency. Although the components of the clock are largely conserved across a broad range of species, there is appreciable diversity in clock structure and function, particularly in insects <span class="info-text">[1]–[4]</span>. The basic model of the molecular clock in *Drosophila* consists of positive and negative feedback loops involving a principle suite of canonical “clock genes” inside pacemaker cells <span class="info-text">[5]</span>. The protein products of *clock (Clk)* and *cycle (Cyc)* genes interact and form a complex that bind to E-box elements in regulatory sequences of the *period (Per)* and *timeless (Tim)* gene promoter regions to activate transcription. *Per* and *Tim* mRNA accumulate in the cytoplasm during the night and the protein products enter the nucleus and bind to the CLK/CYC complex, inhibiting further transcription. On exposure to light, *Drosophila*-type *cryptochrome* (CRY-d) promotes rapid degradation of TIM that renders PER unstable. PER is eventually degraded, releasing the inhibition of transcription <span class="info-text">[6]–[9]</span>. The activity of CRY-d allows the period and phase of the clock to adjust to changes in photoperiod <span class="info-text">[10]</span>. Input pathways of the circadian clock respond to both environmental stimuli (including light and temperature) as well as social stimuli <span class="info-text">[8]</span>. In fact, recent work has shown that the context of the social environment plays a major role in circadian rhythmicity in social insects <span class="info-text">[11]–[13]</span>. Mapping the diversity in insect clock structure on an evolutionary backdrop of organisms that vary in key life history characteristics including social structure may reveal how evolution has shaped the various functions of the clock components <span class="info-text">[1]</span>, <span class="info-text">[3]</span>, <span class="info-text">[4]</span>, <span class="info-text">[11]</span>, <span class="info-text">[12]</span>, <span class="info-text">[14]</span>.

In social insects, the ability to maintain the correct temporal order is not only important for individual success, but also plays a significant role in the success of the colony. Circadian timing is important for foraging activities, sun-compass navigation, timing of mating flights, and synchronization of individuals and the organization of colony tasks, such as nest maintenance <span class="info-text">[11]</span>, <span class="info-text">[14]</span>, <span class="info-text">[15]</span>. Previous work has demonstrated that the circadian clock is associated with the division of labor in honeybees, bumble bees and ants, suggesting that the circadian clock is important for the social organization of insect societies <span class="info-text">[12]</span>, <span class="info-text">[15]–[19]</span>. The evolution of task-related plasticity in social insect colonies is thought to enhance task specialization and to improve colony efficiency; brood care workers have no circadian rhythms and are active around the clock whereas foragers have strong circadian rhythms with peaks of activity accurately timed to the external environments that they encounter <span class="info-text">[14]</span>, <span class="info-text">[15]</span>, <span class="info-text">[18]</span>, <span class="info-text">[19]</span>. Chronobiological plasticity has been documented in species that have evolved sociality independently (ants and bees) and in species that differ in organization of social structure (age-related task specialization in honey bees and size-related task specialization in bumble bees), suggesting that the evolution of clock-related plasticity in social insects is functionally significant <span class="info-text">[14]</span>, <span class="info-text">[17]</span>. Recent evidence that social factors can also affect the ontogeny of circadian rhythms and social synchronization in honeybees further suggests that the evolution of social insect societies has influenced the circadian clock in these species <span class="info-text">[11]</span>, <span class="info-text">[12]</span>, <span class="info-text">[14]</span>. Given this evidence for a significant interplay between the circadian clock and social organization is limited to bees, it is not clear whether these findings represent a general genuine influence of social evolution on the circadian system or a unique set of traits that are limited to the honey bee. To understand the link between circadian clocks and sociality, we need to study additional social species, including hemimetabolous and holometabous insects, that differ in life history traits.

Ants make an attractive model system for studying sociochronobiology because they display a remarkable diversity in ecology, morphology, behavior and life history characteristics and represent an independent evolution of sociality within the Hymenoptera <span class="info-text">[20]</span>. All ants are social and most show a highly derived form of sociality with a sterile worker caste characterized by behavioral specialization in different morphological [castes] or behavioral phenotypes [tasks] <span class="info-text">[21]</span>, <span class="info-text">[22]</span>. Colonies have no central control and interactions between workers in the form of antennal contacts and chemical detection determine decision-making and adjustments in worker behavior and task allocation <span class="info-text">[23]</span>, <span class="info-text">[25]</span>. In addition, the maturation of workers through different tasks occurs on the order of weeks to months in many ant species <span class="info-text">[20]</span>, rather than days in honey bees. Thus, the ontogeny of circadian rhythms may be slower in these ant species. Differences in the underlying social framework of ants and honey bees provide an ideal comparative framework for understanding the evolution of social clocks and the link between molecular mechanisms of circadian rhythm and social behavior.

The molecular dissection of the honeybee social clock revealed a clock mechanism that according to at least five lines of evidence is more similar to the mammalian model rather than to that of *Drosophila* <span class="info-text">[2]</span>, <span class="info-text">[26]</span>. First, the honeybee genome encodes a mammalian-type cryptochrome (*Cry-m*) rather than *Cry-d*, the ortholog found in *Drosophila*. *Cry-m* is not sensitive to light in-vitro and thus it is thought that it does not function in the photic input pathway to the clock, an essential function of *Cry-d* in *Drosophila* <span class="info-text">[5]</span>. Second, the honeybee also lacks an ortholog to timeless, an essential gene for clock function in *Drosophila*. Third, the CYC protein contains a transactivation domain in honeybees that is found in *Drosophila* CLK <span class="info-text">[2]</span>, <span class="info-text">[14]</span>. Fourth, in foraging bees that show strong circadian rhythms, brain mRNA levels of *Cry-m* and *Per* oscillate with a similar phase in both LD and DD conditions and levels of Cyc mRNA oscillate anti-phase to *Per* <span class="info-text">[2]</span>, <span class="info-text">[27]</span>. In contrast to *Drosophila*, the honey bee CLK protein does not contain a transactivation domain and the *Clk* gene does not oscillate. Finally, expression patterns of *amCyc*, *amCry*, *amClk* and *amTim2* (the honeybee ortholog to the *Drosophila* gene, timeout) in honeybee brains are more similar to mammals than *Drosophila*, suggesting that the honeybee clock, in many ways, functions similar to mammalian clocks <span class="info-text">[2]</span>, <span class="info-text">[14]</span>, <span class="info-text">[26]</span>.

Here, we describe for the first time the molecular clockwork for an ant species, the fire ant Soleopsis invicta, a social insect pest of great ecological and economic importance <span class="info-text">[28]</span>, <span class="info-text">[29]</span>. Using the recent fire ant genome <span class="info-text">[30]</span>, we characterize eight putative principle clock genes, *period (SiPer)*, *cycle (SiCyc)*, *clock (SiClk)*, *cryptochrome-m (SiCry)*, *timeout (SiTim)*, *vrille (SiVri)*, *par domain protein 1 (SiPdp1)* and the recently discovered *clockwork orange* gene (SiCwo) <span class="info-text">[31]–[33]</span>. These genes were selected because they have well-documented functions in insect and/or mammalian clocks <span class="info-text">[5]</span>, <span class="info-text">[6]</span>. We use phylogenetic analyses to establish orthology/paralogy of the eight genes across insects and use qPCR analysis to determine how the clock mechanism is functioning at the molecular level. We develop gene models for five genes in order to determine the degree of conservation for domains present in the ant clock and shared with mammalian and Drosophila models. In addition, we describe a novel domain shared across insect orthologs of *clockwork orange* that is absent in vertebrates.

<br />

## Results
<hr />

Phylogenetic analyses of the principle clock genes show concordant patterns and confirm orthology. For each gene, orthologs from all ant species either form a monophyletic group or cluster in a polytomy with other Aculeata (bees, ants and wasps). In most analyses (with the exception of *cycle*), *Nasonia*, the non-social parasitic wasp, is the sister group to the Aculeata (<span class="info-text">Figure S1</span>). On the tree for *cryptochrome*, the bumblebees (*Bombus impatiens*) and honeybees (*Apis mellifera*) form a clade that is nested within the Formicidae (<span class="info-text">Figure 1</span>). The *Cry* orthologs found in the Hymenoptera clearly group with *Cry2/Cry-m* (mammalian-like *Cry*) with the Drosophila-like *Cry1/Cry-d* being only distantly related. The first phylogeny for *clockwork orange* in insects shows similar relationships as the other core clock genes and also shows the relationship between insect CWO and mammalian orthologs DEC1 and DEC2 (<span class="info-text">Figure 2</span>).

<div class="publication-image mb-3">
  <h5><em>Figure 1 – <strong>Parsimony tree for cryptochrome gene family.</strong></em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/parsimony-tree-for-cryptochrome.webp" />
    <img src="/img/publications/parsimony-tree-for-cryptochrome.png" width="1092" height="1134" alt="Parsimony tree for cryptochrome gene family" />
  </picture>
  <p class="mt-2">
    Shown is a consensus tree with bootstrap values from 250 replicates. Outgroups include orthologs of photolyase and DASH. Cryptochromes segregate into two clades, <em>Drosophila</em>-like (CRY-d, also known as "insect CRY1") proteins and mammalian-like (CRY-m, also known as "insect CRY2") proteins. CRY-d proteins are light sensitive with no transcriptive repressive activity whereas CRY-m proteins are light insensitive and may function as the main transcriptional repressors of the core clockwork <span class="info-text">[1]</span>. All of the Hymenopterans (bees, ants, and wasps) have only the <em>Cry-m</em> genes.
  </p>
</div>

<br />

<div class="publication-image mb-3">
  <h5><em>Figure 2 – <strong>Parsimony tree for clockwork orange orthologs.</strong></em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/parsimony-tree-for-clockwork-orange.webp" />
    <img src="/img/publications/parsimony-tree-for-clockwork-orange.png" width="1092" height="453" alt="Parsimony tree for clockwork orange orthologs" />
  </picture>
  <p class="mt-2">
    Shown is a consensus tree with bootstrap values from 250 replicates. Insect CWO orthologs clearly separate from mammalian orthologs <em>Dec1</em> & <em>Dec2</em>.
  </p>
</div>

The gene models for *clock*, *cryptochrome*, *cycle* and *period* for *Solenopsis* contain all of the conserved domains and binding regions found in insect species with high amino acid similarity to *Apis* in conserved regions (<span class="info-text">Figure S2</span>). In the *cryptochrome*-m model, the fire ant ortholog, *SiCry-m*, clearly contains the NLS (nuclear localization signal) region found in mammalian-like cryptochromes that are absent in *Drosophila*-like cryptochromes (<span class="info-text">Figure 3</span>). In addition, the amino acid sequence surrounding this region has low similarity to *Drosophila*-like cryptochromes relative to mammalian-like cryptochromes.

<div class="publication-image mb-3">
  <h5><em>Figure 3 – <strong>Schematic models for CRY proteins from various animals.</strong></em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/schematic-models-for-CRY-proteins.webp" />
    <img src="/img/publications/schematic-models-for-CRY-proteins.png" width="1092" height="713" alt="Schematic models for CRY proteins" />
  </picture>
  <p class="mt-2">
    The protein models depicted are from the fire ant <em>Solenopsis invicta</em> (siCRY), the Western honey bee <em>Apis mellifera</em> (amCRY-m), the fruit fly <em>Drosophila melanogaster</em> (dmCRY), the Monarch butterfly <em>Danaus plexippus</em> (dpCRY1) the jewel wasp <em>Nasonia vitripennis</em> (nvCRY) and the domestic mouse <em>Mus musculus</em> (mmCRY1). Highlighted areas on the diagrams represent putative functional domains and motifs. The numbers below the domains indicate percents of identity/similarity to corresponding sequences on the protein of the fire ant. The numbers at the end of each diagram indicate the predicted protein size (number of amino acid residues). The protein domains on the CRY sequence are:. • FAD binding. Proteins containing this domain are photolyases (DNA repair enzymes) or function as blue light photoreceptors (Pfam domain accession number: PF03441). • DNA photolyase. This domain is an evolutionary conserved protein domain from bacteria to mammals. It binds to UV-damaged DNA containing pyrimidine dimers and, upon absorbing a near-UV photon 300 to 500 nm, breaks the cyclobutane ring joining the two pyrimidines of the dimer (Pfam domain accession number: PF00875). • ICAT - Inhibition CLOCK-ARNTL Transcription. A domain required for the inhibition of CLOCK-ARNTL-mediated transcription (Swiss-Prot record of mmCRY1 accession number: P97784). • RD-2b – A domain defined by <span class="info-text">[58]</span> based on studies with the clock proteins of the zebrafish. The domain is necessary for nuclear localization and the repression of CLOCK: BMAL-mediated transcription. • NLS - Nuclear localization signal in the RD-2b region, following <span class="info-text">[58]</span>. • EST - Expressed sequence tags.
  </p>
</div>

In the model for *clockwork orange*, the protein domains on CWO sequences that are shared between vertebrates and insects include bHLH, basic-helix-loop-helix, Hairy Orange and an NLS. Additional protein domains found on the vertebrate DEC2 sequence including an Ala/Gly -rich domain and an HDAC interacting domain are not found on the insect orthologs. EST reads and directed PCR amplifications of the putative C-terminal region revealed stop codons in different positions, suggesting that siCWO may have several alternative splice variants. Our comparative analysis of the 3′ end of the gene across insects revealed an additional conserved domain not yet described for this gene, called *clockwork orange* C-tail Domain, CWOCD (<span class="info-text">Figure 4</span>). The domain contains a conserved VPIFALH C-tail region that is present in all the putative orthologous insect sequences that were analyzed in this study. The 57–62 amino acids in this domain are largely conserved among insects (eg. ants and bees are similar at 52/58 sites in this region) and have little similarity to the Ala/Gly -rich domain in C-tail of vertebrate DEC2, the verterbrate orthdolog of CWO <span class="info-text">[33]</span>, <span class="info-text">[34]</span>. Analysis of the CWOCD C-tail by Blastp algorithm (NCBI) showed that similar sequences are also present in Hairy and class b basic helix-loop-helix proteins (data not shown).

<div class="publication-image mb-3">
  <h5><em>Figure 4 – <strong>Schematic models for Clockwork Orange (CWO) proteins from various insects.</strong></em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/schematic-models-for-clockwork-orange.webp" />
    <img src="/img/publications/schematic-models-for-clockwork-orange.png" width="1092" height="1400" alt="Schematic models for Clockwork Orange" />
  </picture>
  <p class="mt-2">
    The depicted models are for the fire ant <em>Solenopsis invicta</em> (siCWO), the honey bee <em>Apis mellifera</em> (amCWO), the fruit fly <em>Drosophila melanogaster</em> (dmCWO), Red Flour Beetle <em>Tribolium castaneum</em> (tcCWO). Also shown are related proteins from the house mouse <em>Mus musculus</em> (mmDEC2) and zebrafish <em>Danio rerio</em> (drDEC2). Highlighted areas on diagrams represent putative functional domains and motifs. For more details see legend to Fig. 3. Inset shows a CLUSTALW multiple sequence alignment of a new conserved domain discovered on the CWO protein sequence that we termed 'Clockwork Orange C-tail Domain' (CWOCD) the CLUSTALW alignment includes several additional CWO proteins from drosophilid and ant species (see <span class="info-text">Table S3</span>). Asterisks in the bottom of alignment indicate amino acids conserved between insects and vertebrates. Alignments were generated with CLUSTALW and colored with JalView according to the default CLUSTALX convention. • bHLH - Basic-helix-loop-helix. Proteins containing this domain are typically dimeric transcription factors, each with one helix containing basic amino acid residues that facilitate DNA binding to an E-box. (Pfam domain accession number PF00010.). • CWOCD - Clockwork orange C-tail domain. • Hairy Orange - The Orange domain is found in the <em>Drosophila</em> proteins Hesr-1, Hairy, and Enhancer of Split. The Orange domain is proposed to mediate specific protein-protein interaction that confers specificity among members of the Hairy/E(SPL) family. • PCRs - regions of targeted PCR for confirmation of mRNA sequence.
  </p>
</div>

The expression patterns of the eight principle clock genes suggest that the circadian clock of *S. invicta* functions similar to the honey bee clock (<span class="info-text">Figures 5</span>, <span class="info-text">6</span>; <span class="info-text">Table 1</span>). *SiPer* and *SiCry-m* oscillate with a similar phase–mRNA levels increase in the evening and peak during the dark phase (R<sup>2</sup>adj = 0.79, p = 0.003 and R<sup>2</sup>adj = 0.49, p = 0.015; respectively). *SiCyc* and *SiCwo* oscillate anti-phase to *SiPer* (R<sup>2</sup>adj = 0.54, p = 0.02 and R<sup>2</sup>adj = 0.60, p = 0.012 respectively). Weak oscillations for *SiVri* are similar in phase to *SiCyc* and *SiCwo* (R<sup>2</sup>adj = 0.63, p<0.0001). *SiTim2* has significant, weak oscillations (R<sup>2</sup>adj = 0.55, p = 0.019) but patterns are not consistent within individual colonies. *SiPdp1* and *SiClk* do not show significant, consistent oscillations across colonies (R<sup>2</sup>adj = −0.2, p = 0.32 and R<sup>2</sup>adj = 0.19, p = 0.01 respectively). There are no significant differences over time for the control gene, EF1α (ANOVA F = 1.40, p = 0.29).

<div class="publication-image mb-3">
  <h5><em>Figure 5 – <strong>Relative gene expression patterns of 8 principle clock genes across colonies.</strong></em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/relative-gene-expression-patterns.webp" />
    <img src="/img/publications/relative-gene-expression-patterns.png" width="1092" height="1476" alt="Relative gene expression patterns" />
  </picture>
  <p class="mt-2">
    Expression values for each data point (±SE) are plotted as the average relative expression level (ΔΔCt) across colonies (n = 3 colonies). Relative expression is calculated as the number of standard deviations above and below the mean value for all data points (across time). Standard error bars are calculated from variation across three colonies. The open stripe in the horizontal bar at base of the plot represents the daylight phase (12 hrs) and the solid stripe represents the dark phase (12 hrs) during the night. See <span class="info-text">Table 1</span> for additional details on clock gene expression.
  </p>
</div>

<br />

<div class="publication-image mb-3">
  <h5><em>Figure 6 – <strong>Summary representation of oscillation in mRNA for five core clock genes in fire ant brains under LD illumination.</strong></em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/summary-representation-of-oscillation.webp" />
    <img src="/img/publications/summary-representation-of-oscillation.png" width="1092" height="556" alt="Summary representation of oscillation in mRNA" />
  </picture>
  <p class="mt-2">
    Shown are four genes (<em>SiPer</em>, <em>SiCry</em>, <em>SiCwo</em>, and <em>SiCyc</em>) that show significant oscillations and have significant correlations to the cosine model with R<sup>2</sup>adj≥0.5, and one gene, <em>SiClk</em>, which does not oscillate. Lines represent schematic cartoons of the actual oscillations. <em>SiPer</em> and <em>SiCry</em> cycle in the same phase and peak during the night and <em>SiCwo</em> and <em>SiCyc</em> cycle antiphase to <em>SiPer</em> and peak during the day. Not shown on the figure are the two transcription factors, <em>SiVri</em> and <em>SiPdp1</em>; <em>SiTim</em> was excluded because the expression of this gene was not consistent across nests.
  </p>
</div>

<br />

<div class="publication-image mb-3">
  <h5><em>Table 1 – <strong>Summary of gene expression data.</strong></em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/table-1-summary-of-gene-expression-data.webp" />
    <img src="/img/publications/table-1-summary-of-gene-expression-data.png" width="1092" height="397" alt="Summary of gene expression data" />
  </picture>
</div>

<br />

## Discussion
<hr />

#### The Molecular Clockwork in the Ant is Similar to that in the Honey Bee

The molecular dissection of the fire ant circadian clock reveals that ants have a mammalian-like clock with high similarity to the honeybee clock. Our study represents the most complete molecular characterization of a hymenopteran clock to date. Sequences of the eight principle clock genes are more closely related to the honeybee orthologs than to those of *Nasonia* and expression patterns of these genes suggest the molecular clockwork of ants utilizes similar mechanisms as the bee clock. In addition, the gene models from the fire ant show that all genes contain the necessary domains for the predicted function of the genes according to a mammalian-like clockwork and lack key components that characterize *Drosophila*-like clocks.

The fact that fire ant clock orthologs are more similar to honeybee clock orthologs than *Nasonia* in phylogenetic analyses is consistent with the monophyly of the Aculeata; i.e., the hypothesis that ants are more closely related to bees than to chalcidoid wasps <span class="info-text">[35]</span>. Genome-wide BLASTp searches of fire ant proteins against protein databases indicate that 47% of *Solenopsis invicta* genes have the strongest similarity to apoid sequences and an additional 22% similar to *Nasonia* <span class="info-text">[30]</span>. It is well documented that the bee clock is intricately connected to the social biology of this species <span class="info-text">[14]</span>, <span class="info-text">[18]</span>, <span class="info-text">[19]</span>, <span class="info-text">[36]</span>, <span class="info-text">[37]</span>. Because ants and bees evolved sociality independently, it was not clear whether both clades evolved similar molecular mechanisms for regulating the circadian clock. Our results suggest that the clocks of ants and bees are regulated by the same mechanisms.

One diagnostic feature of the similarity in function of ant and bee clocks is found in the *cryptochrome* gene model. Cryptochromes belong to the photolyase gene family. Photolyases have a high affinity for complementary DNA strands and break certain types of pyrimidine dimers that arise when a pair of thymine or cytosine bases on the same strand of DNA become covalently linked. Proteins containing this domain also function as blue light photoreceptors that mediate blue light-induced gene expression and modulation of circadian rhythms. In *Drosophila*, the *cryptochrome* ortholog (*Cry-d*) functions as a photoreceptor in brain cells <span class="info-text">[1]</span>. The light-dependent function of the gene requires a highly variable C-terminal domain that permits the interaction of the gene with TIM1 [timeless, a gene not found in the ant or bee genome] <span class="info-text">[38]</span>. In mammalian-type cryptochromes (*Cry-m*), the C-terminal region is highly divergent from *Cry-d* orthologs. In contrast, *Cry-m* orthologs share three RD domains that are not found in *Cry-d* genes. These domains are responsible for nuclear localization and for the repression of CLK/BMAL transcription <span class="info-text">[2]</span>. Our results show ant genomes encode the *Cry-m*-type *cryptochrome* with high sequence similarity and comparable expression profiles to the honeybee ortholog. Two lines of evidence suggest that the *Cry* gene is a functional component of the circadian mechanism in ants, like it is in bees and mammals. First, Cry mRNAs continue to oscillate in complete darkness [DD] in another ant species, *P. occidentalis* (unpublished data). Second, sequence similarity between ant and bee models of this gene is high [98%] which suggests that the function of this gene is conserved <span class="info-text">[1]</span>.

Another diagnostic feature is whether the CYC protein contains a transactivation domain. CYC (typically called BMAL1 in mammals) proteins are transcriptional factors with PAS-bHLH domains. In honeybees, *amCYC* is phylogenetically related to *Drosophila* CYC, but contains a highly conserved C-terminal transactivation domain that is found in mammalian BMAL proteins. In addition, honeybees lack the C-terminal region on *amCLK* that is responsible for transcriptional activity in dmCLK. Our results show that ant orthologs to CYC and CLK parallel the structural features of honeybee clocks. Thus, we would predict similarity in the temporal pattern of gene expression in ants and bees. In bees, brain transcript levels of *amCyc* oscillate nearly anti-phase to *amPer* while *amClk* does not oscillate <span class="info-text">[2]</span>, <span class="info-text">[27]</span>, <span class="info-text">[39]</span>. In ants, two out of three nests showed strong oscillations in *siCyc* that were anti-phase to *siPer* oscillations. However, we also found a weak, non-significant trend in siClk under LD conditions. More data on the gene expression of *clock* in ants under DD conditions is necessary to determine whether the expression of this gene is under circadian influence in ants.

Our data provide the first report of qPCR gene expression patterns for *Vrille* and *Pdp1*, two conserved basic zipper transcription factors, in Hymenopterans. *SiVri* shows a possible daily oscillation antiphase to *SiPer* oscillations. *SiPdp1* does not appear to oscillate in the fire ant. The expression pattern of *SiVri* is similar to that found for aphids <span class="info-text">[40]</span> and is not consistent with that seen in *Drosophila*, where transcripts for the gene are controlled by CLK/CYC complex. A whole brain microarray analyses suggest that *amVri* mRNA levels do not vary during the day in both nurse and forager honey bees sampled in DD, but this finding has not been yet validated with qPCR <span class="info-text">[27]</span>. Additional studies are needed for determining if the pattern (and function) of *Vri* expression differ in bees and ants.

Overall, there is a potential difference in the expression of *Vri* but the remarkable similarity of ant and bee clocks supports the hypothesis that the Hymenoptera clock diverged from the basal insect clocks that contained both mammalian and *Drosophila* types of *Cry* and *Tim* <span class="info-text">[2]</span>. Although the clocks of social hemimetabolous insects, i.e. termites, have not yet been studied in detail, the holometabolous social insects appear to utilize a molecular clockwork that generally functions similar to mammalian clocks and have lost the set of clock genes that were retained in *Drosophila* <span class="info-text">[2]</span>. Our results will facilitate comparative sociogenomic analyses of circadian rhythms in the Aculeates. Understanding the similarities and differences in the regulation of the ant and bee clocks may give new insight into the role of circadian rhythms in regulating colony behavior and will help determine whether mechanisms of chronobiological plasticity are shared across social insects <span class="info-text">[11]</span>, <span class="info-text">[14]</span>, <span class="info-text">[17]</span>.

#### A Novel Clockwork Orange Domain in Insects

Recent studies indicate the *clockwork orange* is an integral part of the circadian clockwork in *Drosophila* <span class="info-text">[31]–[33]</span>. *Cwo* encodes a transcriptional repressor that is thought to compete with the CLK/CYC complex to bind to the E-box proteins <span class="info-text">[31]</span>. The CWO protein also regulates itself by forming its own negative feedback loop and repressing the transcription of the *Cwo* gene. In *Drosophila*, CWO appears to function in regulating the amplitude of circadian oscillations in other core clock genes. The mammalian ortholog to *Cwo*, *Dec2* is involved in sleep length shifts <span class="info-text">[41]</span>. As yet, we do not understand the role of CWO in hymenopteran insects.

To our knowledge, our results present the first gene-tree and model comparing *clockwork orange* across insects. Our analysis highlighted the presence of a novel domain in this gene that is present in insect CWO proteins but not the mammalian orthologs DEC1 and DEC2. The discovery of this conserved insect-specific domain in CWO contrasts with the overall similarity of social insect and mammalian orthologs of PER, CRY-m, CLK and CYC. This suggests that hymenopteran and *Drosophila* clocks may operate differently for some of the core feedback loops in the molecular clockwork but may share a mechanism for regulating the feedback loop of *clockwork orange*, the particular function of which is yet unknown for insects. Interestingly, the expression pattern of *SiCwo* in this study parallels patterns seen in the mouse, with high levels of *SiCwo* transcript during the day (peak at CT6) and lower levels during the night. This is similar to the oscillation seen in honey bees (peak at CT7; <span class="info-text">[27]</span>) and in contrast to the pattern seen in *Drosophila* where *Cwo* transcripts peak later at ∼CT12 <span class="info-text">[31]–[33]</span>. Thus, the structure of the *clockwork orange* gene in ants matches the structure found in insect *Cwo* genes and the expression pattern suggests a possible involvement in the positive loop of the clock.

*Clockwork orange* encodes a transcriptional repressor that inhibits CLK-mediated activation via interactions with PER <span class="info-text">[33]</span>. In *Drosophila*, evidence suggests that CWO acts primarily in the late night to terminate CLK/CYC-mediated transcription of target genes. CWO creates its own negative feedback system, as it is one of the target genes regulated by the CLK/CYC complex through canonical E-box sequences <span class="info-text">[31]</span>. The mammalian homologues (DEC1 and DEC2) appear to operate with a similar feedback mechanism. The feedback loop involving CWO is essential for the development of circadian rhythms in flies because CWO-deficient strains of flies show disruptions in oscillations of core clock genes and arrhythmic behavior <span class="info-text">[33]</span>. In addition, the direct suppression of core clock genes through the CWO negative feedback loop helps generate and sustain high-amplitude oscillations <span class="info-text">[33]</span>.

The discovery that *clockwork orange* activity regulates the amplitude of the other core genes in the *Drosophila* clock underscores the complexity of the integrated positive and negative feedback loops that comprise the insect clock. Given the extensive studies of the E-box mediated negative feedback system <span class="info-text">[5]</span>, one of the key components of the circadian clock, the finding that the *clockwork orange* gene can regulate circadian expression via the E-box demonstrates that the molecular mechanism for circadian rhythms likely has redundancy among its components. The high conservation of the CWOCD sequence in the C-tail of the insects' CWO proteins suggest that this domain has a conserved but yet unknown function. The difference between the C-tail of insects and vertebrates provides yet another clue as to how evolution has shaped the diversity of circadian clocks and their regulation.

<br />

## Materials and Methods
<hr />

#### Development of Gene Models and EST Analyses

We developed gene models for five clock proteins in the fire ant *Solenopsis invicta*: *Cry*, *Cyc*, *Per*, *Clk* and *Cwo*. Selected genes contained domains known to differ across species and/or had adequate information known from multiple species. To determine gene models, we first ran TBLASTN using *Drosophila* *Cry*, *Cyc*, *Per*, *Clk* or *Cwo* proteins. Subsequently, ruby/bioruby scripts [[https://bioinformatics.oxfordjournals.org/content/26/20/2617.short](https://academic.oup.com/bioinformatics/article/26/20/2617/194139){:target="_blank"}; [https://www.biomedcentral.com/1471-2105/10/221/abstract](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-221){:target="_blank"}] were used to extract relevant subsets of the fire ant genome. Automated gene models were generated using MAKER2 [[http://www.biomedcentral.com/1471-2105/12/491](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-491){:target="_blank"}] and subsequently manually refined using Apollo [[genomebiology.com/2002/3/12/research/0082](//genomebiology.com/2002/3/12/research/0082){:target="_blank"}; [https://genome.cshlp.org/content/18/1/188.full](https://genome.cshlp.org/content/18/1/188.full){:target="_blank"}]. Libraries of orthologous sequences were created by searching protein databases with the Blastp algorithm [[https://nar.oxfordjournals.org/content/36/suppl_2/W5.short](https://academic.oup.com/nar/article/36/suppl_2/W5/2505810){:target="_blank"}] and using siCRY, siCYC, siPER or siCLK protein sequences as a query. From the Blastp results, putative orthologs of CRY, CYC, PER or CLK were selected from species that are relatively well characterized (e.g. *Drosophila melanogaster* or the mouse) or of special interest (e.g. *Nasonia vitripennis*).

Swiss-Prot records for the mouse clock protein orthologs that have good annotation (marked regions) of the major protein domains were used to identify the protein domains on CRY, CYC, PER and CLK insect orthologs (mmCRY1 accession number: P97784, mmBmal1 accession number: Q9WTL8, mmPER1 accession number: O35973 and mmCLK accession number: O08785). Putative domains on the insect orthologous were identified using the EBI multiple sequence alignment CLUSTALX/CLUSTALW algorithm <span class="info-text">[42]</span>. For each focal clock protein (CRY, CYC, PER or CLK) the protein models were aligned with the various species and the sequences corresponding to the protein domains on the mice proteins were marked on the insect orthologs. The sequences corresponding to each protein domain on the various orthologs were confirmed with the NCBI <span class="info-text">[43]</span>, SMART <span class="info-text">[44]</span>, <span class="info-text">[45]</span>, Pfam <span class="info-text">[46]</span>, ProSite <span class="info-text">[47]</span> and InterPro <span class="info-text">[48]</span> databases. Additional domains and motifs were delineated based on sequences reported in the literature, and for which the appropriate citation is provided. The EBI Global Alignment program <span class="info-text">[49]</span> was used to determine the degree of amino acid residue identity/similarity between domains on the *S. invicta* protein models and corresponding amino acid sequences on orthologs from other animals. Domains were defined as conserved only if the amino acid sequence identity was ≥40% compared with the domain sequence on the mouse orthologs proteins <span class="info-text">[50]</span>.

Available EST sequences provided confirmation for most regions of the protein models. The “Translate tool” from the ExPASy proteomics server was used to predict the amino acid sequence encoded by each EST nucleotide sequence. Sequences that encode predicted polypeptides without stop codons in the middle were selected. The CLUSTALX/CLUSTALW algorithm was used to align the predicted EST amino acid sequences to the corresponding MAKER/Apollo protein model, and the protein models were corrected according to the amino acids predicted based on the EST sequences.

For CWO, a library of CWO ortholog sequences was compiled using the annotated CWOs found in <span class="info-text">[51]</span>. The original library was selected using sequences >20% identity to the corresponding siCWO sequence. The protein domain sequences and conserved regions were identified as above. The Stockholm Bioinformatics Center's NucPred [[https://www.sbc.su.se/~maccallr/nucpred/cgi-bin/single.cgi](https://www.sbc.su.se/~maccallr/nucpred/cgi-bin/single.cgi){:target="_blank"}] was used to search for putative nuclear localization signals [NLS]. The NLS sequence was predicted in all CWO proteins present in the model. It is located inside bHLH domain with 100% identity in ortholog CWO sequences analyzed in this study. Additional domains and motifs were delineated based on relevant literature in which their biochemical function was defined <span class="info-text">[32]</span>, <span class="info-text">[34]</span>, <span class="info-text">[41]</span>.

Five EST sequences clustered together with the siCWO model. One of these clusters contained gaps inside the alignment caused by regions in an EST sequence that were not present in the Apollo model for siCWO. Translation of directed PCR in this region confirmed the Apollo model, suggesting that the EST with the extra regions in its sequence may represent an unprocessed mRNA of siCWO.

#### Phylogenetic Analysis for the Clock Gene Proteins

Amino acid sequences from the open reading frames of the eight principle clock genes were aligned to orthologs found in Genbank and the ant genome database (now FOURMIDABLE <span class="info-text">[52]</span> and references in <span class="info-text">[30]</span>, <span class="info-text">[53]–[57]</span> ) using CLUSTALW in MEGA 5.0 (<span class="info-text">Table S1</span>). Sequences of genes with highly variable C-tail regions were trimmed. Parsimony trees were constructed from the alignments using TNT phylogenetic software after exporting the data from SequenceMatrix. All analyses used 100 random addition runs with bootstrap values based on 250 replicates. For comparison, phylogenies were also constructed using maximum likelihood methods based on the JTT matrix-based model in MEGA 5.0. The robustness of the unrooted tree was assessed using bootstraps (1000 replicates).

#### Gene Expression Analyses

Three *Solenopsis invicta* colonies were sampled from laboratory colonies housed at the USDA-ARS Center for Medical, Agricultural, and Veterinary Entomology in Gainesville, FL. Colonies were allowed to acclimate for 2 days in LD conditions (12 hours L/12 hours D) and on the third day, 20 foragers were collected from each colony at 7 timepoints (every 4 hours). Individuals were placed on dry ice immediately and remained frozen at −70°C until extraction. On the day of collection, samples were shipped overnight on dry ice. Total RNA was extracted from whole heads and pooled across 12 individuals for each source colony (Qiagen RNAez Micro Kit). Because ants were sampled in LD conditions, it is not possible to distinguish whether changes in gene expression over the day are a result of endogenous circadian rhythms, are influenced by gene expression responses to exogenous light patterns, or both.

Fire ant-specific primers were designed from exon-coding regions to amplify 60–120 bp regions for qPCR analyses. cDNA was synthesized from extracted total RNA preps using ABI TaqMan Gold Reverse Transcriptase reagents and random hexamers. The 10 uL reactions included 1.0 uL of RNA with 1 X TaqMan RT Buffer, 5.5 mM 25 mM MgCl2, 500 uM of each of the deoxyNTPs, 2.5 uM of the Random Hexamer primers, 0.4 U/uL of RNase Inhibitor and 1.5 U/uL of MultiScribe Reverse Transcriptase. Reactions were performed in triplicate for each sample at each time point. All reactions were run at 25°C for 10 minutes, 48°C for 30 minutes, 95°C for 5 minutes, and then stored at −20°C until quantitative PCR. For each cDNA replicate, gene expression was assayed on an ABI 7900 HT instrument using ABI Taqman Gold reagents with gene specific primers (<span class="info-text">Table S2</span>). The 25 uL qPCR reactions for each gene included 3.5 uL of template cDNA with 1 X TaqMan Buffer A, 5.5 mM 25 mM MgCl2, 200 uM of 10 mM deoxyATP, 200 uM of 10 mM deoxyCTP, 200 uM of 10 mM deoxyGTP, 400 uM of 20 mM deoxyUTP, 100 nM of probe, 200 nM of each primer, 0.01 U/uL of AmpErase UNG and 0.025 U/uL of AmpliTaq Gold DNA Polymerase. To standardize clock gene expression, EF1α was used as a control for each cDNA replicate. The 25 uL qPCR reactions for the control included 1.5 uL of template cDNA and the same reaction mixture described above. The following primers were used for the control gene: SiEF1α Forward: GGCTCTGAGGGAGGCTTT, SiEF1α Reverse: CGGAGATGTTCTTCACGTTGAA, SiEF1α Probe: CTCGCGATAACGTCG. Real-time PCR reactions for target genes and SiEF1α were performed under the following conditions: 2 min at 50°C for one cycle, 10 min at 95°C for one cycle, 15 sec at 95°C, 1 min at 60°C, for 45 cycles. Data were analyzed using SDS 2.1 software and quantification of relative mRNA levels and standard errors were calculated using the ΔΔCt method (ABI User Bulletin).

Gene expression patterns for each colony were analyzed separately. Significant differences in gene expression over time were tested using a two-way ANOVA [colony, timepoint] for each gene, including EF1α in STATA. Changes in gene expression over time were compared to a cosinor model to determine how closely the oscillations fit a generalized circadian model using MATLAB and the equation detailed in <span class="info-text">[12]</span>. Ten million simulations were run starting with random seeds. The top 400 values that best fit the cosinor model were averaged to obtain the R<sup>2</sup> and amplitude values. Significance values for the fit to the cosine model were calculated in MATLAB. R<sup>2</sup> adjusted was calculated from the estimates of R2 according to <span class="info-text">[12]</span>. Oscillations in genes were considered significant if both ANOVA p-values and R<sup>2</sup> were ≤0.05 and R<sup>2</sup> values explained a large portion of the variance (R<sup>2</sup>adj≥0.5).

<br />

## Supporting Information
<hr />

- [**Figure S1.**](/publications/2012-fire-ant-clock-genes-evolution-period-cycle-cryptochrome-vrille/pone.0045715.s001.ppt) <br />
Additional phylogenetic trees, (.PPTX)

- [**Figure S2.**](/publications/2012-fire-ant-clock-genes-evolution-period-cycle-cryptochrome-vrille/pone.0045715.s002.ppt) <br />
Additional gene models, (.PPTX)

- [**Table S1.**](/publications/2012-fire-ant-clock-genes-evolution-period-cycle-cryptochrome-vrille/pone.0045715.s003.docx) <br />
Sequences used for phylogenetic alignments and gene models, (.DOCX)

- [**Table S2.**](/publications/2012-fire-ant-clock-genes-evolution-period-cycle-cryptochrome-vrille/pone.0045715.s004.docx) <br />
Primers used in expression analyses, (.DOCX)

- [**Table S3.**](/publications/2012-fire-ant-clock-genes-evolution-period-cycle-cryptochrome-vrille/pone.0045715.s005.docx) <br />
Photo credits for gene models, (.DOCX)

<br />

## Acknowledgments
<hr />

We would like to thank E. Carroll for assistance with the collection of fire ant samples, S. Peteru & A.P. Hoadley for help with gene expression analyses, A. Filipowicz for help with the statistical analysis and A. Ay for assistance in analysis of cosinor models.

<br />

## Author Contributions
<hr />

Conceived and designed the experiments: KKI GB RM YW. Performed the experiments: KKI YW AK DS. Analyzed the data: KKI YW AK GB RM. Contributed reagents/materials/analysis tools: KKI DS YW AK. Wrote the paper: KKI AK GB.

<br />

## References
<hr />

1. Yuan Q, Metterville D, Briscoe AD, Reppert SM (2007) Insect cryptochromes: Gene duplication and loss define diverse ways to construct insect circadian clocks. Mol Biol Evol 24: 948–955.

2. Rubin EB, Shemesh Y, Cohen M, Elgavish S, Robertson HM, et al. (2006) Molecular and phylogenetic analyses reveal mammalian-like clockwork in the honey bee (Apis mellifera) and shed new light on the molecular evolution of the circadian clock. Genome Res 16: 1352–1365.

3. Reppert SM (2006) A colorful model of the circadian clock. Cell 124: 233–236.

4. Zhan S, Merlin C, Boore JL, Reppert SM (2011) The monarch butterfly genome yields insights into long-distance migration. Cell 147: 1171–1185.

5. Hardin PE (2005) The circadian timekeeping system of Drosophila.. Curr Biol 15: R714–22.

6. Allada R, Chung BY (2010) Circadian organization of behavior and physiology in Drosophila.. Annu Rev Physiol 72: 605–624.

7. Gallego M, Virshup DM (2007) Post-translational modifications regulate the ticking of the circadian clock. Nat Rev Mol Cell Biol 8: 139–148.

8. Hall JC (2003) Genetics and molecular biology of rhythms in Drosophila and other insects. Adv Genet 48: 1–280.

9. Majercak J, Chen WF, Edery I (2004) Splicing of the period gene 3′-terminal intron is regulated by light, circadian clock factors, and phospholipase C. Mol Cell Biol. 24: 3359–3372.

10. Majercak J, Sidote D, Hardin PE, Edery I (1999) How a circadian clock adapts to seasonal decreases in temperature and day length. Neuron 24: 219–230.

11. Bloch G, Grozinger CM (2011) Social molecular pathways and the evolution of bee societies. Philos Trans R Soc Lond B Biol Sci 366: 2155–2170.

12. Shemesh Y, Eban-Rothschild A, Cohen M, Bloch G (2010) Molecular dynamics and social regulation of context-dependent plasticity in the circadian clockwork of the honey bee. J Neurosci 30: 12517–12525.

13. Lone SR, Sharma VK (2011) Timekeeping through social contacts: Social synchronization of circadian locomotor activity rhythm in the carpenter ant Camponotus paria.. Chronobiol Int 28: 862–872.

14. Bloch G (2010) The social clock of the honeybee. J Biol Rhythms 25: 307–317.

15. Ingram KK, Krummey S, LeRoux M (2009) Expression patterns of a circadian clock gene are associated with age-related polyethism in harvester ants, Pogonomyrmex occidentalis.. BMC Ecol 9: 7.

16. Shemesh Y, Cohen M, Bloch G (2007) Natural plasticity in circadian rhythms is mediated by reorganization in the molecular clockwork in honeybees. FASEB J 21: 2304–2311.

17. Eban-Rothschild A, Belluci S, Bloch G (2011) Maternity-related plasticity in circadian rhythms of bumble-bee queens. Proc Biol Sci 278: 3510–3516.

18. Toma DP, Bloch G, Moore D, Robinson GE (2000) Changes in period mRNA levels in the brain and division of labor in honey bee colonies. Proc Natl Acad Sci USA 97: 6914–6919.

19. Bloch G, Toma DP, Robinson GE (2001) Behavioral rhythmicity, age, division of labor and period expression in the honey bee brain. J Biol Rhythms 16: 444–456.

20. Wilson EO (1971) The insect societies. Cambridge, MA: Belknap Press of Harvard University Press. 548 p.

21. Toth AL, Robinson GE (2007) Evo-devo and the evolution of social behavior. Trends Genet 23: 334–341.

22. Smith CR, Toth AL, Suarez AV, Robinson GE (2008) Genetic and genomic analyses of the division of labour in insect societies. Nat Rev Genet 9: 735–748.

23. Greene MJ, Gordon DM (2003) Social insects: Cuticular hydrocarbons inform task decisions. Nature 423: 32.

24. Greene MJ, Gordon DM (2007) How patrollers set foraging direction in harvester ants. Am Nat 170: 943–948.

25. Greene MJ, Gordon DM (2007) Interaction rate informs harvester ant task decisions. Behav Ecol 18: 451–455.

26. Honeybee Genome Sequencing Consortium (2006) Insights into social insects from the genome of the honeybee Apis mellifera.. Nature 443: 931–949.

27. Rodriguez-Zas SL, Southey BR, Shemesh Y, Rubin EB, Cohen M, et al. (2012) Microarray analysis of natural socially regulated plasticity in circadian rhythms of honey bees. J Biol Rhythms 27: 12–24.

28. Vinson SB (1986) Economic Impact and Control of Social Insects. New York: Praeger.

29. McDonald M (2006) Reds under your feet. New Scientist 189: 50–51.

30. Wurm Y, Wang J, Riba-Grognuz O, Corona M, Nygaard S, et al. (2011) The genome of the fire ant Solenopsis invicta.. Proc Natl Acad Sci USA 108: 5679–5684.

31. Matsumoto A, Ukai-Tadenuma M, Yamada RG, Houl J, Uno KD, et al. (2007) A functional genomics strategy reveals clockwork orange as a transcriptional regulator in the Drosophila circadian clock. Genes Dev 21: 1687–1700.

32. Lim C, Chung BY, Pitman JL, McGill JJ, Pradhan S, et al. (2007) Clockwork orange encodes a transcriptional repressor important for circadian-clock amplitude in Drosophila.. Curr Biol 17: 1082–1089.

33. Kadener S, Stoleru D, McDonald M, Nawathean P, Rosbash M (2007) Clockwork orange is a transcriptional repressor and a new Drosophila circadian pacemaker component. Genes Dev 21: 1675–1686.

34. Yamada K, Miyamoto K (2005) Basic helix-loop-helix transcription factors, BHLHB2 and BHLHB3; their gene expressions are regulated by multiple extracellular stimuli. Front Biosci 10: 3151–3171.

35. Sharkey MJ (2007) Phylogeny and classification of Hymenoptera. Zootaxa 1668: 521–548.

36. Bloch G, Rubinstein CD, Robinson GE (2004) Period expression in the honey bee brain is developmentally regulated and not affected by light, flight experience, or colony type. Insect Biochem Mol Biol 34: 879–891.

37. Bloch G, Robinson GE (2001) Reversal of honeybee behavioural rhythms. Nature 410: 1048.

38. Busza A, Emery-Le M, Rosbash M, Emery P (2004) Roles of the two Drosophila CRYPTOCHROME structural domains in circadian photoreception. Science 304: 1503–1506.

39. Cheeseman JF, Winnebeck EC, Millar CD, Kirkland LS, Sleigh J, et al. (2012) General anesthesia alters time perception by phase shifting the circadian clock. Proc Natl Acad Sci USA 109: 7061–7066.

40. Cortes T, Ortiz-Rivas B, Martinez-Torres D (2010) Identification and characterization of circadian clock genes in the pea aphid Acyrthosiphon pisum.. Insect Mol Biol 19 Suppl 2123–139.

41. He Y, Jones CR, Fujiki N, Xu Y, Guo B, et al. (2009) The transcriptional repressor DEC2 regulates sleep length in mammals. Science 325: 866–870.

42. Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res 22: 4673–4680.

43. Marchler-Bauer A, Anderson JB, Cherukuri PF, DeWeese-Scott C, Geer LY, et al. (2005) CDD: A conserved domain database for protein classification. Nucleic Acids Res 33: D192–6.

44. Letunic I, Copley RR, Schmidt S, Ciccarelli FD, Doerks T, et al. (2004) SMART 4.0: Towards genomic data integration. Nucleic Acids Res 32: D142–4.

45. Schultz J, Milpetz F, Bork P, Ponting CP (1998) SMART, a simple modular architecture research tool: Identification of signaling domains. Proc Natl Acad Sci USA 95: 5857–5864.

46. Bateman A, Coin L, Durbin R, Finn RD, Hollich V, et al. (2004) The pfam protein families database. Nucleic Acids Res 32: D138–41.

47. Hulo N, Bairoch A, Bulliard V, Cerutti L, De Castro E, et al. (2006) The PROSITE database. Nucleic Acids Res 34: D227–30.

48. Apweiler R, Attwood TK, Bairoch A, Bateman A, Birney E, et al. (2000) InterPro–an integrated documentation resource for protein families, domains and functional sites. Bioinformatics 16: 1145–1150.

49. Rice P, Longden I, Bleasby A (2000) EMBOSS: The European molecular biology open software suite. Trends Genet 16: 276–277.

50. Xu D, Xu Y, Uberbacher EC (2000) Computational tools for protein modeling. Curr Protein Pept Sci 1: 1–21.

51. Duncan EJ, Dearden PK (2010) Evolution of a genomic regulatory domain: The role of gene co-option and gene duplication in the enhancer of split complex. Genome Res 20: 917–928.

52. Wurm Y, Uva P, Ricci F, Wang J, Jemielity S, et al. (2009) Fourmidable: A database for ant genomics. BMC Genomics 10: 5.

53. Bonasio R, Zhang G, Ye C, Mutti NS, Fang X, et al. (2010) Genomic comparison of the ants Camponotus floridanus and Harpegnathos saltator.. Science 329: 1068–1071.

54. Smith CD, Zimin A, Holt C, Abouheif E, Benton R, et al. (2011) Draft genome of the globally widespread and invasive Argentine ant (Linepithema humile). Proc Natl Acad Sci USA 108: 5673–5678.

55. Smith CR, Smith CD, Robertson HM, Helmkampf M, Zimin A, et al. (2011) Draft genome of the red harvester ant Pogonomyrmex barbatus.. Proc Natl Acad Sci USA 108: 5667–5672.

56. Suen G, Teiling C, Li L, Holt C, Abouheif E, et al. (2011) The genome sequence of the leaf-cutter ant Atta cephalotes reveals insights into its obligate symbiotic lifestyle. PLoS Genet 7: e1002007.

57. Nygaard S, Zhang G, Schiott M, Li C, Wurm Y, et al. (2011) The genome of the leaf-cutting ant Acromyrmex echinatior suggests key adaptations to advanced social life and fungus farming. Genome Res 21: 1339–1348.

58. Hirayama J, Nakamura H, Ishikawa T, Kobayashi Y, Todo T (2003) Functional and structural analyses of cryptochrome. vertebrate CRY regions responsible for interaction with the CLOCK:BMAL1 heterodimer and its nuclear localization. J Biol Chem 278: 35620–35628.

<br />
