---
title: 'Genes and genomic processes underpinning the social lives of ants'
layout: publication
permalink: /publications/genes-and-processes-of-social-evolution-in-ants/
---

<br />

<p class="info-text">Published: 4 January 2018</p>

# Genes and genomic processes underpinning the social lives of ants

<h5 class="info-text my-4">EmelineFavreau, Carlos Martínez-Ruiz, Leandro Rodrigues Santiago, Robert L Hammond, YannickWurm</h5>

[*Current Opinion in Insect Science*](https://doi.org/10.1016/j.cois.2017.12.001), 2018, Volume 25, Pages 83-90

<br />

## Abstract
<hr />

The >15 000 ant species are all highly social and show great variation in colony organization, complexity and behavior. The mechanisms by which such sociality evolved, as well as those underpinning the elaboration of ant societies since their ∼140 million year old common ancestor, have long been pondered. Here, we review recent insights generated using various genomic approaches. This includes understanding the molecular mechanisms underlying caste differentiation and the diversity of social structures, studying the impact of eusociality on genomic evolutionary rates, and investigating gene expression changes associated with differences in lifespan between castes. Furthermore, functional studies involving RNAi and CRISPR have recently been successfully applied to ants, opening the door to exciting research that promises to revolutionize the understanding of the evolution and diversification of social living.

<br />

## Introduction
<hr />

Ants show remarkable division of labor whereby males and queens reproduce while other females, the workers, build and defend the nest, rear brood and forage for food <span class="info-text">[1••]</span>. Since their evolution from a solitary common ancestor ∼140 million years ago <span class="info-text">[2, 3••]</span>, the ants have radiated into >15 000 extant species. This ecologically dominant family exhibits an extraordinary diversity of social lifestyles, for example: obligatory fungus farming (leaf cutter ants), nomadic predatory lifestyles (army ants), slave-making and [social parasitism](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/social-parasitism){:target="_blank"} <span class="info-text">[4]</span>. Colonies vary greatly, with tens to several million individuals per colony, up to nine morphological worker castes within a species, as well as an array of mechanisms to resolve intra-colony conflicts <span class="info-text">[5]</span>.

While their behaviors, ecology, evolutionary contexts and morphologies have been extensively studied, it has only recently become possible to perform genome-scale analyses in ants <span class="info-text">[6]</span>, with the first seven ant genomes published in 2010 and 2011 <span class="info-text">[2]</span>. Here, we review publications from the last few years on the genes and genomic processes underlying caste differentiation, the evolution of social organization, [intraspecific variation](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/intraspecific-variation){:target="_blank"} in lifespan, symbiotic relationships, and chemical communication.

#### How the ants got their genomes

Publications of the first genome projects of humans and traditional laboratory organisms including *Drosophila* and *Caenorhabditis* alerted many biologists, including social insect researchers, to the power of genome knowledge. The development of post-Sanger sequencing technologies <span class="info-text">[7•]</span> (see <span class="info-text">Glossary</span>) made genome projects accessible to small teams interested in particular species. Within months of the 2009 ‘ant genomics’ meeting at Arizona State University <span class="info-text">[8]</span>, projects to sequence genomes of seven species were under way (<span class="info-text">Figure 1</span>).

<div class="publication-quote bg-light border p-2 mb-3">
  <h5><em>Box 1</em></h5>
  <h5>Glossary</h5>
  <ul>
    <li>Gamergate: Some species lack a distinct queen caste, or can have a hierarchy of reproductive workers. Such reproductive workers are called gamergates. Amongst others, this occurs in <em>Harpegnathos</em> and many other ponerine ants.</li>
    <li><a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/gene-regulatory-network" target="_blank">Gene regulatory networks</a> (GRNs): Modules of interacting, coexpressed genes. These networks are highly resilient to modifications of individual members of the network.</li>
    <li>Genetic accommodation: A process by which a phenotype initially determined by environmental cues becomes genetically determined and thus, heritable.</li>
    <li>Genomics: Generally taken to mean genome-scale analyses, that is, using a very large number of genetic markers. Can include transcriptomics and other large-scale analyses.</li>
    <li>
      RADseq: Method of <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/shotgun-sequencing" target="_blank">DNA sequencing</a> based on Restriction site Associated DNA markers, which retrieves a small fraction of the total genome (e.g. the same 0.1% of the genome in all samples) and enables low-cost studies of <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/genetic-divergence" target="_blank">genetic variation</a> at 1000s of markers across the genome.
    </li>
    <li>
      Sanger-sequencing: Low-throughput DNA sequencing method that is still used for some applications such as targeted sequencing of specific genes. However, the majority of sequencing efforts now use higher-throughput ‘post-Sanger’ sequencing methods including Illumina, Pacific Biosciences and Oxford <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/nanopore" target="_blank">Nanopore</a>. These technologies are more cost-effective for large projects.
    </li>
    <li>
      Social polymorphism: Different types of social organization within the same species. In social insects, a well-studied type of social polymorphism involves the number of the number of queens in the colony.
    </li>
    <li>Trophallaxis: Fluid passed from an individual to another, through the mouth.</li>
    <li>
      Ultraconserved elements (UCEs): non-coding regions of the genome that have high similarity among distant species. They have recently been used as a manner of increasing the power of phylogenetic studies.
    </li>
    <li>
      Taxonomically restricted genes: Genes identified exclusively in specific taxa. Such genes likely evolve through rapid divergence of existing genes, or through recruitment of previously non-coding sequence. They are thought to be associated with novel taxon-specific functions.
    </li>
    <li>Pleiotropic gene: Gene that affects more than one phenotypic trait. Most genes are likely pleiotropic.</li>
  </ul>
</div>

<div class="publication-image mb-3">
  <h5><em>Figure 1</em></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/solenopsis-invicta-fire-ants.webp" />
    <img src="/img/publications/solenopsis-invicta-fire-ants.jpeg" width="320" height="482" alt="Solenopsis invicta fire ants" style="display: block; margin: 15px auto; max-width: 320px;" />
  </picture>
  <p class="mt-2">
    <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/solenopsis-invicta" target="_blank"><em>Solenopsis invicta</em></a> fire ants (smaller) defending a region of the social chromosome supergene against a <em>Pogonomyrmex rugosus</em> <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/harvester-ant" target="_blank">harvester ant</a> worker (larger) symbolizes the amicable competition between early ant genome projects.
  </p>
</div>

These first seven ant genomes <span class="info-text">[2]</span> led to a broad range of studies on these species (some examples below), but also paved the way for the sequencing of genomes and ‘reference transcriptomes’ of additional species. As of writing, genomes of 23 ant species and <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/transcriptome" target="_blank">transcriptomes</a> of 26 additional species have been published (Table 1). Other studies have used reduced representation sequencing techniques such as restriction site associated <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/shotgun-sequencing" target="_blank">DNA sequencing</a> (RADseq, e.g. <span class="info-text">[6]</span>, see <span class="info-text">Glossary</span>) or the sequencing of ultraconserved elements (UCEs, e.g. <span class="info-text">[9•]</span>, see <span class="info-text">Glossary</span>) to identify genetic structures or to resolve phylogenies.

<div class="publication-image mb-3">
  <h5><em>Table 1 – </em><strong>Overview of ant species with published genomic and transcriptomic data.</strong></h5>
  <picture>
    <source type="image/webp" srcset="/img/publications/table-1-overview-of-ant-species.webp" />
    <img src="/img/publications/table-1-overview-of-ant-species.jpg" width="1092" height="1155" alt="Overview of ant species with published genomic and transcriptomic data" />
  </picture>
</div>

#### The coexistence of distinct female castes

Queen and [worker ants](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/worker-ants) are both female and do not differ genetically (with some exceptions <span class="info-text">[10]</span>). Instead, caste (worker or queen) is irreversibly determined by environmental factors that trigger larvae to develop down either the worker or queen pathway <span class="info-text">[11]</span>. Comparisons between queens and workers in multiple species demonstrate that many genes with strong caste-biased patterns of expression in one developmental stage show no such bias in other stages <span class="info-text">[12]</span>. The differences in whole-body gene expression between castes are highest during the last larval stages <span class="info-text">[12, 13]</span>. Interestingly, genes with high expression plasticity (i.e. with different expression levels in different castes) during [larval development](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/larval-development) in [*Cardiocondyla obscurior*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/cardiocondyla-obscurior) also showed increased evolutionary rates <span class="info-text">[14•]</span>. In adults, highly pleiotropic genes (i.e. under more phenotypic constraints, see <span class="info-text">Glossary</span>) are consistently less likely to become caste-biased across a wide taxonomic range of ant species <span class="info-text">[15•]</span>. Such findings support the idea that [phenotypic plasticity](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/phenotypic-plasticity) along with genetic accommodation (see <span class="info-text">Glossary</span>) are likely to play an essential role in caste differentiation <span class="info-text">[16, 17•]</span>.

Contrary to some expectations, genes expressed exclusively in a single caste are rare <span class="info-text">[18•, 19••]</span>. Instead, [transcriptome](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/transcriptome) comparisons suggest that groups of genes have correlated expression profiles (i.e. ‘coexpression modules’) with consistent differential expression between adult queens and workers across 15 ant species <span class="info-text">[15•, 19••]</span>. This suggests that caste-specific phenotypes can arise from modifications in [gene regulatory networks](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/gene-regulatory-network) (GRNs, see <span class="info-text">Glossary</span>) (e.g. <span class="info-text">[18•]</span>). However, other studies have found that taxonomically-restricted genes (see <span class="info-text">Glossary</span>) are overrepresented among worker-biased genes <span class="info-text">[20]</span>. These seemingly contradictory views are not mutually exclusive <span class="info-text">[21•]</span> and the divergence between castes can involve differential usage of existing modules of genes as well as differential usage of fast-evolving genes with low connectivity <span class="info-text">[22, 23]</span>.

Because workers generally do not reproduce, selection acts only indirectly on worker phenotypes. All else being equal, selection is therefore expected to be less efficient on genes expressed in workers than on genes expressed in queens <span class="info-text">[24•, 25]</span>. In agreement, expression profiling in [*Monomorium pharaonis*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/monomorium-pharaonis) reveals relaxed selection on worker-specific genes and faster evolution <span class="info-text">[26•]</span>. In contrast, a study of 15 species <span class="info-text">[19••]</span> shows no difference in evolutionary rates between castes, although this could be due to the use RNAseq from whole bodies. Indeed, allometric differences in tissues between castes are known to confound comparisons <span class="info-text">[27•]</span>. However, although the M. *pharaonis* study used more specific samples (heads, gasters, larvae), these still include multiple tissues <span class="info-text">[26•]</span>. Importantly, although the relaxed selection observed in M. *pharaonis* might be caused by indirect selection, an alternative is that relaxed selection facilitated the recruitment of genes to caste-biased roles <span class="info-text">[17•]</span>.

Many ant species have lost the queen caste but have both non-reproductive and reproductive workers (gamergates, see <span class="info-text">Glossary</span>), comparisons between which can illuminate important molecular mechanisms. For example, this has allowed the identification of differentially expressed genes underpinning reproductive hierarchies <span class="info-text">[28•]</span> and the role of specific proteins <span class="info-text">[29]</span> in individual behavior.

Other ant species also display within-caste phenotypic plasticity not associated with reproduction, offering the chance to study several levels of variability within the same genome. Such studies confirm the important role of epigenetic cues in generating diversity <span class="info-text">[30]</span>. For instance, experiments with [juvenile hormone](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/juvenile-hormone), known to regulate insect development, showed that the potential to produce super-soldiers is ancestral in many [*Pheidole*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/pheidole) species, and this facilitated the independent evolution of super-soldiers in two species <span class="info-text">[31]</span>. Methylation patterns have been proposed as a mechanism to explain both between-caste and within-caste diversity <span class="info-text">[30]</span>, but the evidence so far is unclear <span class="info-text">[32]</span>, and suggests that methylation patterns may not be as important as previously thought in controlling ant [polyphenism](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/polyphenism) <span class="info-text">[33]</span>.

#### Genes and genetic architectures underlying social organization

Many species of ants include both single-queen and multiple-queen colonies. It was long thought that such variation is determined by environmental factors <span class="info-text">[34]</span>. This idea was challenged in the late 1990s with the discovery that a genetic element, marked by alternate alleles of the odorant-binding protein (OBP) gene *Gp*-9, is responsible for different social organizations of the red [fire ant](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/fire-ants) [*Solenopsis invicta*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/solenopsis-invicta) <span class="info-text">[35]</span> (see <span class="info-text">Glossary</span> for social polymorphism). Recent genome-wide analyses showed that the two Gp-9 alleles in fact mark alternate variants of a ‘supergene’ carried by a pair of ‘social chromosomes’. Recombination is suppressed between the two variants of the >19 Mb long supergene <span class="info-text">[36]</span>. Intriguingly, the supergene variant responsible for multiple-queen colonies has much lower [genetic diversity](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/genetic-divergence) than the rest of the genome, consistent with the effect of a selective sweep or recent bottleneck <span class="info-text">[37•]</span>. The two variants of the supergene carry alternate alleles for many of >400 protein-coding genes, including nine additional *Gp*-9-like OBPs. Six of these OBPs have differences in coding sequence between supergene variants <span class="info-text">[38•]</span>. This work raised a question: is the situation in *S. invicta* an oddity? In short, no, because a pair of independently evolved social chromosomes, that bear almost no similarity to those in *S. invicta*, determine social form in the distantly related alpine silver ant, Formica selysi <span class="info-text">[39]</span>. It is currently unknown whether such genetic architecture is required for the coexistence of social forms in other species, or for transitions between species from obligate single-queen colonies to obligate multiple-queen colonies.

Genetic changes underlying social organization can also be identified by comparing populations or species. For example, young *Pogonomyrmex californicus* queens predisposed to found nests either alone or in groups show fundamental differences in gene expression profiles of heads <span class="info-text">[40]</span>. Similarly, in socially parasitic species, one can predict that the loss of the worker caste leads to a loss in the genome of potential ‘worker genes’. Instead, comparisons between the genomes of three social parasite species and genomes of non-parasitic host species found almost no differences in gene presence/absence <span class="info-text">[18•]</span>. This is consistent with the idea that genes used in workers also have roles in other castes.

#### Investigating intraspecific variation in lifespan

The intraspecific lifespan of ants is extremely variable and in the same colony queens can live for years while workers typically live a few months and males a few weeks, despite sharing the same genome <span class="info-text">[41, 42]</span>. Such disparity is fascinating yet challenges the application of classic theories of aging <span class="info-text">[43]</span> and we have limited understanding of the mechanisms involved. A [transcriptomic](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/transcriptomics) analysis of [*Lasius*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/lasius) ants at three time points found that [DNA repair](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/dna-repair) genes are more highly expressed in both brain and leg tissues of queens than age-matched workers <span class="info-text">[44•]</span>, pointing at an association between greater lifespan and somatic repair. Because queens are long lived but mate shortly after emergence from the [pupae](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/pupae), the long-term storage of viable sperm in the queen's specialized organ (spermatheca) is another conundrum that is only now being investigated. Recent comparative transcriptomic analysis of [spermatheca](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/spermatheca) in virgin and mated [*Crematogaster*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/crematogaster) queens identified a number of differentially expressed genes including some associated with antioxidizing function <span class="info-text">[45•]</span>.

#### Genes and mechanisms underlying the evolution of symbiotic relationships

The intricate symbiotic relationships (e.g. [mutualism](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/mutualism), parasitism) that many ants have with other taxa can now be scrutinized using genomics. For example, analysis of UCEs in 18 [*Pseudomyrmex*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/pseudomyrmex) species determined that after the ant-acacia plant mutualism evolved in one lineage, a second independent *Pseudomyrmex* lineage colonized plants that were involved in the first mutualism <span class="info-text">[9•]</span>. A different study identified protein-coding genes with signatures of adaptive evolution in three ant-plant mutualistic *Pseudomyrmex* in comparison with four generalist species <span class="info-text">[46•]</span>. In the attine ants, mutualistic fungus farming is associated with clear large-scale genomic changes including genomic rearrangements, loss of the arginine biosynthesis and positive selection on [chitinase](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/chitinase) pathways, with complementary changes in the fungus <span class="info-text">[47]</span>.

[Transcriptomics](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/transcriptomics) analyses are also beginning to illuminate the mechanisms by which parasites manipulate ant host behavior. For instance, hundreds of genes are differentially expressed in brains of [*Temnothorax*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/temnothorax) workers infected with the cestode *Anomotaenia* compared to controls <span class="info-text">[48•]</span>. Similarly, large-scale changes in gene expression occur in heads of [*Camponotus*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/camponotus) [carpenter ants](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/carpenter-ants) when their behavior is manipulated by the [*Ophiocordyceps*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/ophiocordyceps) fungus. Interestingly, many of the fungal genes up-regulated during manipulated biting behavior are absent from non-parasitic relatives <span class="info-text">[49]</span>.

Whole genome sequencing has also helped reveal new mutualisms. Sequencing of *C. obscurior* queens <span class="info-text">[50•]</span> revealed a previously unknown intracellular bacterium symbiont, *Westeberhardia*. Analysis of its genome and titers in the host suggest this bacterium may assist cuticle synthesis. Finally, there is also an interest to identify ant [microbiomes](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/microbiome) (e.g. in [*Formica*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/formica) <span class="info-text">[51]</span>) and, motivated by pest control efforts, transcriptomic analyses have identified viruses of invasive ants (e.g. *S. invicta* <span class="info-text">[52]</span>, [*Linepithema humile*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/linepithema-humile) <span class="info-text">[53]</span>, *Nylanderia* spp. <span class="info-text">[54, 55]</span>, [*Anoplolepis gracilipes*](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/anoplolepis-gracilipes) <span class="info-text">[56]</span>).

#### Roles of chemical communication genes in colony life <span class="info-text"></span>

Ants have a fascinating communication system based on the production and perception of pheromones. In line with this, the first ant genome projects discovered particularly high number of genes involved in chemical communication, with for example more than 400 putative [olfactory receptors](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/olfactory-receptor) in the [fire ant](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/fire-ants) <span class="info-text">[57]</span>. Some chemical communication genes likely have highly conserved functions as indicated by [conserved sequences](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/conserved-sequence) and antennal expression of core sets of single-ortholog odorant [binding proteins](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/binding-protein) (OBPs) and chemosensory proteins (CSPs) <span class="info-text">[58, 59]</span>. However, among the 330 ± 26 (mean ± standard deviation) odorant and the 80 ± 67 gustatory receptors identified in each of eight ant species, >30% were taxon-specific <span class="info-text">[60]</span>. Frequent [gene duplications](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/gene-duplication) and losses suggest that many chemical communication genes are involved in adaptive processes in response to changing environments or arms races. This is in line with analyses of a clade of CSPs with high turnover and extensive evidence of adaptive evolution <span class="info-text">[59]</span>. There is even variation in olfactory genes within species: the multiple-queen form of *S. invicta* includes an OBP absent from single-queen colonies <span class="info-text">[38•]</span>.

CRISPR transgenics recently confirmed that the highly conserved orco gene, required for proper functioning of other olfactory receptors in *Drosophila*, plays a similar role in ants: ants with broken orco lacked antennal glomeruli and their colonies were dysfunctional, highlighting the importance of such chemical communication genes for social organization <span class="info-text">[61••, 62••]</span>.

Recent analyses have shown that non-olfactory communication is also important: [trophallaxis](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/trophallaxis) fluid (see <span class="info-text">Glossary</span>) is used to transfer compounds including proteins, [microRNAs](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/microrna) and [juvenile hormone](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/juvenile-hormone) between colony members <span class="info-text">[63]</span>.

#### Concluding remarks

The continued improvements in low-cost sequencing technologies are making it possible to pursue analyses that would have been difficult to imagine only few years ago. Focusing on individual species, this includes in-depth tissue-specific profiling of RNAs or epigenetic marks <span class="info-text">[64•]</span>, as well as the sequencing of whole genomes of many individuals to enable population genomics approaches <span class="info-text">[65•]</span>. Large whole genome sequencing initiatives including large number of related species, or one species for many genera — such as in the Global Ant Genomics Alliance <span class="info-text">[3••]</span> — are already under way. Such datasets will deepen our understanding of the specific mechanisms and evolutionary forces involved in the evolution of complex phenotypes, symbiotic interactions and social behavior. Furthermore, analyses of such data will make it possible to pinpoint key candidate genes. Detailed analyses which involve artificially modifying the activity or sequence of genes have already begun, with several recent studies demonstrating the effectiveness of temporarily switching genes off using [RNA interference](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/rna-interference) <span class="info-text">[66, 67]</span> — which can propagate by [trophallaxis](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/trophallaxis) through the colony — of changing the reproductive status of an adult worker injecting specific [neuropeptides](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/neuropeptide) <span class="info-text">[29]</span> or of permanently modifying gene sequences using CRISPR transgenics <span class="info-text">[68]</span>. Such approaches pave the way for many new breakthroughs bridging our ultimate and proximate understanding of social evolution.

<br />

## Conflict of interest statement
<hr />

None declared.

<br />

## Acknowledgements
<hr />

The authors thank Rodrigo Pracana, Karina Zile and Gino Brignoli for comments on earlier drafts of this manuscript. This work was supported by the Biotechnology and Biological Sciences Research Council [grant BB/K004204/1] and the Natural Environment Research Council [grant NE/L00626X/1 and NE/L002485/1] and Conselho Nacional de Desenvolvimento Científico e Tecnológico [grant 248391/2013-5].

<br />

## References
<hr />

1. J.J. Boomsma, R. Gawne **Superorganismality and caste differentiation as points of no return: how the major evolutionary transitions were lost in translation**. Biol Rev Camb Philos Soc (2017), 10.1111/brv.12330
(*The authors present a historical and theoretical review of superorganismality and eusociality, arguing that precise terminology that aligns with major transitions should be preferred over muddled terms.*)

2. J. Gadau, M. Helmkampf, S. Nygaard, J. Roux, D.F. Simola, C.R. Smith, G. Suen, Y. Wurm, C.D. **Smith The genomic impact of 100 million years of social evolution in seven ant species**. Trends Genet, 28 (2012), pp. 14-21

3. ••J.J. Boomsma, S.G. Brady, R.R. Dunn, J. Gadau, J. Heinze, L. Keller, S.C. Moreau, N.J. Sanders, L. Schrader, T.R. Schultz, L. Sundström, P.S. Ward, W.T. Wcislo, G. Zhang
**The GAGA consortium: the global ant genomics alliance (GAGA)**. Myrmecol News, 25 (2017), pp. 61-66 (*Overview of this project to sequence the genomes of >200 ant species while collecting relevant ecological data.*)

4. B. Hölldobler, E.O. Wilson **The Ants** Harvard University Press (1990)

5. A.F.G. Bourke, N.R. Franks **Social Evolution in Ants** Princeton University Press (1995)

6. S. Nygaard, Y. Wurm **Ant genomics: challenges to overcome and opportunities to seize** Myrmecol News, 21 (2015), pp. 59-72

7. •S. Goodwin, J.D. McPherson, W.R. McCombie **Coming of age: ten years of next-generation sequencing technologies** Nat Rev Genet, 17 (2016), pp. 333-351 (*The authors review the sequencing methods (short and long reads) and applications that are currently available, and technologies that are promising for the near future.*)

8. C.D. Smith, C.R. Smith, U. Mueller, J. Gadau **Ant genomics: strength and diversity in numbers** Mol Ecol, 19 (2010), pp. 31-35

9. •P.S. Ward, M.G. Branstetter **The acacia ants revisited: convergent evolution and biogeographic context in an iconic ant/plant mutualism** Proc R Soc Lond B Biol Sci, 284 (2017), p. 20162569 (*This phylogenomic analysis of ultraconserved elements of 18 Pseudomyrmex demonstrated that a mutualism with acacia plants independently evolved twice in this group of ants.*)

10. K.E. Anderson, T.A. Linksvayer, C.R. Smith **The causes and consequences of genetic caste determination in ants (Hymenoptera: Formicidae)** Myrmecol News, 11 (2008), pp. 119-132

11. D.E. Wheeler **The developmental basis of worker caste polymorphism in ants** Am Nat, 138 (1991), pp. 1218-1238

12. L. Ometto, D. Shoemaker, K.G. Ross, L. Keller **Evolution of gene expression in fire ants: the effects of developmental stage, caste, and species** Mol Biol Evol, 28 (2011), pp. 1381-1392

13. C. Morandin, K. Dhaygude, J. Paviala, K. Trontti, C. Wheat, H. Helanterä
**Caste-biases in gene expression are specific to developmental stage in the ant Formica exsecta**
J Evol Biol, 28 (2015), pp. 1705-1718

14. •L. Schrader, H. Helanterä, J. Oettler
**Accelerated evolution of developmentally biased genes in the tetraphenic ant Cardiocondyla obscurior**
Mol Biol Evol, 34 (2017), pp. 535-544 (*This analysis of Cardiocondyla obscurior larvae compared expression levels of all castes (queen, worker, winged male and ergatoid male) and established a positive correlation between the plasticity seen in those transcriptomic data and the evolutionary rate of genes expressed in larval development.*)

15. •C. Morandin, A.S. Mikheyev, J.S. Pedersen, H. Helanterä
**Evolutionary constraints shape caste-specific gene expression across 15 ant species**
Evolution, 71 (2017), pp. 1273-1284 (*Transcriptomic data from 15 ant species show that pleiotropic genes are less likely to become caste-biased.*)

16. M. Nahmad, L. Glass, E. Abouheif
**The dynamics of developmental system drift in the gene network underlying wing polyphenism in ants: a mathematical model**
Evol Dev, 10 (2008), pp. 360-374

17. •B.G. Hunt, L. Ometto, Y. Wurm, D. Shoemaker, S.V. Yi, L. Keller, M.A.D. Goodisman
**Relaxed selection is a precursor to the evolution of phenotypic plasticity**
Proc Natl Acad Sci U S A, 108 (2011), pp. 15936-15941 (*Specific genes with different expression profiles in different ant castes are consistently under relaxed selection.*)

18. •C.R. Smith, S. Helms Cahan, C. Kemena, S.G. Brady, W. Yang, E. Bornberg-Bauer, T. Eriksson, J. Gadau, M. Helmkampf, D. Gotzek, et al.
**How do genomes create novel phenotypes? Insights from the loss of the worker caste in ant social parasites**
Mol Biol Evol, 32 (2015), pp. 2919-2931 (*The analysis of transcriptomic data from three different ant social parasites and their hosts show no clear gene loss or gain associated to caste loss.*)

19. ••C. Morandin, M.M.Y. Tin, S. Abril, C. Gómez, L. Pontieri, M. Schiøtt, L. Sundström, K. Tsuji, J.S. Pedersen, H. Helanterä, et al.
**Comparative transcriptomics reveals the conserved building blocks involved in parallel evolution of diverse phenotypic traits in ants**
Genome Biol, 17 (2016), p. 43 (*The transcriptomes of 15 ant species suggest that rather than specific caste-specific genes, entire coexpression modules associated to queen and worker caste are conserved across the ant phylogenetic tree.*)

20. B. Feldmeyer, D. Elsner, S. Foitzik
**Gene expression patterns associated with caste and reproductive status in ants: worker-specific genes are more derived than queen-specific ones**
Mol Ecol, 23 (2014), pp. 151-161

21. •A.S. Mikheyev, T.A. Linksvayer
**Genes associated with ant social behavior show distinct transcriptional and evolutionary patterns**
Elife, 4 (2015), p. e04775 (*This genomic and transcriptomic analysis in Monomorium pharaonis identified modules of coexpressed genes associated with age polyphenism that are evolutionarily conserved in other ant species.*)

22. E. Akçay, T.A. Linksvayer, J. Van Cleve
**Bridging social evolution theory and emerging empirical approaches to social behavior**
Curr Opin Behav Sci, 6 (2015), pp. 59-64

23. T.A. Linksvayer
**Chapter eight — the molecular and evolutionary genetic implications of being truly social for the social insects**
Adv In Insect Phys, 48 (2015), pp. 271-292

24. •T.A. Linksvayer, M.J. Wade
**Theoretical predictions for sociogenomic data: the effects of kin selection and sex-limited expression on the evolution of social insect genomes**
Front Ecol Evol, 4 (2016), p. E1 (*Using a theoretical model, the authors hypothesize that due to the effects of indirect selection in workers and under similar selection pressures, selection should be relaxed in worker-biased genes if compared to queen-biased genes.*)

25. D.W. Hall, M.A.D. Goodisman
**The effects of kin selection on rates of molecular evolution in social insects**
Evolution, 66 (2012), pp. 2080-2093

26. •M.R. Warner, A.S. Mikheyev, T.A. Linksvayer
**Genomic signature of kin selection in an ant with obligately sterile workers**
Mol Biol Evol, 34 (2017), pp. 1780-1787 (*Genomic and transcriptomic data from the ant Monomorium pharaonis reveal that worker-biased genes are under relaxed selection if compared to queen-biased genes.*)

27. •B.R. Johnson, J. Atallah, D.C. Plachetzki
**The importance of tissue specificity for RNA-seq: highlighting the errors of composite structure extractions**
BMC Genomics, 14 (2013), p. 586 (*Empirical gene expression profiling of gasters and individual tissues within gasters of honeybee queens and workers. This study demonstrates that studies from heterogeneous body parts (e.g. whole bodies or gasters) provide misleading results.*)

28. •Y. Okada, Y. Watanabe, M.M.Y. Tin, K. Tsuji, A.S. Mikheyev
**Social dominance alters nutrition-related gene expression immediately: transcriptomic evidence from a monomorphic queenless ant**
Mol Ecol, 26 (2017), pp. 2922-2938 (*This caste differentiation analysis of Diacamma ant species is based on transcriptomics (brain and gaster) and correlates nutrient-processing gene expression levels to the formation of social hierarchy.*)

29. J. Gospocic, E.J. Shields, K.M. Glastad, Y. Lin, C.A. Penick, H. Yan, A.S. Mikheyev, T.A. Linksvayer, B.A. Garcia, S.L. Berger, et al.
**The neuropeptide corazonin controls social behavior and caste identity in ants**
Cell, 170 (2017) 748–759.e12

30. A. Chittka, Y. Wurm, L. Chittka
**Epigenetics: the making of ant castes**
Curr Biol, 22 (2012), pp. R835-R838

31. R. Rajakumar, D. San Mauro, M.B. Dijkstra, M.H. Huang, D.E. Wheeler, F. Hiou-Tim, A. Khila, M. Cournoyea, E. Abouheif
**Ancestral developmental potential facilitates parallel evolution in ants**
Science, 335 (2012), pp. 79-82

32. R. Libbrecht, P.R. Oxley, L. Keller, D.J.C. Kronauer
**Robust DNA methylation in the clonal raider ant brain**
Curr Biol, 26 (2016), pp. 391-395

33. S. Patalano, A. Vlasova, C. Wyatt, P. Ewels, F. Camara, P.G. Ferreira, C.L. Asher, T.P. Jurkowski, A. Segonds-Pichon, M. Bachman, et al.
**Molecular signatures of plastic phenotypes in two eusocial insect species with simple societies**
Proc Natl Acad Sci USA, 112 (2015), pp. 13970-13975

34. L. Keller, K. Ross
**Phenotypic plasticity and “cultural transmission” of alternative social organizations in the fire ant Solenopsis invicta**
Behav Ecol Sociobiol, 33 (1993)

35. K.G. Ross, L. Keller
**Ecology and evolution of social organization: insights from fire ants and other highly eusocial insects**
Annu Rev Ecol Syst, 26 (1995), pp. 631-656

36. J. Wang, Y. Wurm, M. Nipitwattanaphon, O. Riba-Grognuz, Y.-C. Huang, D. Shoemaker, L. Keller
**A Y-like social chromosome causes alternative colony organization in fire ants**
Nature, 493 (2013), pp. 664-668

37. •R. Pracana, A. Priyam, I. Levantis, R.A. Nichols, Y. Wurm
**The fire ant social chromosome supergene variant Sb shows low diversity but high divergence from SB**
Mol Ecol, 26 (2017), pp. 2864-2879 (*In the introduced North American population of Solenopsis invicta, the social supergene variant carrying Gp-9b show signatures of a recent selective sweep.*)

38. •R. Pracana, I. Levantis, C. Martínez-Ruiz, E. Stolle, A. Priyam, Y. Wurm
**Fire ant social chromosomes: differences in number, sequence and expression of odorant binding proteins**
Evol Lett, 1 (2017), pp. 199-210 (*The Gp-9 odorant binding protein (OBP) has been associated with social form in Solenopsis invicta since the late 1990s. An exhaustive search of the recently discovered social chromosome region identified nine additional Gp-9-like OBPs in complete linkage to Gp-9.*)

39. J. Purcell, A. Brelsford, Y. Wurm, N. Perrin, M. Chapuisat
**Convergent genetic architecture underlies social organization in ants**
Curr Biol, 24 (2014), pp. 2728-2732

40. M. Helmkampf, A.S. Mikheyev, Y. Kang, J. Fewell, J. Gadau
**Gene expression and variation in social aggression by queens of the harvester ant Pogonomyrmex californicus**
Mol Ecol, 25 (2016), pp. 3716-3730

41. L. Keller
**Queen lifespan and colony characteristics in ants and termites**
Insectes Soc, 45 (1998), pp. 235-246

42. J.D. Parker, K.M. Parker, B.H. Sohal, R.S. Sohal, L. Keller
**Decreased expression of Cu–Zn superoxide dismutase 1 in ants with extreme lifespan**
Proc Natl Acad Sci U S A, 101 (2004), pp. 3486-3489

43. B.H. Kramer, G.S. van Doorn, F.J. Weissing, I. Pen
**Lifespan divergence between social insect castes: challenges and opportunities for evolutionary theories of aging**
Curr Opin Insect Sci, 16 (2016), pp. 76-80

44. •E.R. Lucas, E. Privman, L. Keller
**Higher expression of somatic repair genes in long-lived ant queens than workers**
Aging, 8 (2016), pp. 1940-1951 (*In Lasius niger, expression of genes related to DNA and protein repair is higher in older queens than in other colony individuals.*)

45. •A. Gotoh, S. Shigenobu, K. Yamaguchi, S. Kobayashi, F. Ito, K. Tsuji
**Transcriptome profiling of the spermatheca identifies genes potentially involved in the long-term sperm storage of ant queens**
Sci Rep, 7 (2017), p. 5972 (*Gene expression in the spermatheca of Crematogaster osakensis queens correlate with age and mating status.*)

46. •B.E.R. Rubin, C.S. Moreau
**Comparative genomics reveals convergent rates of evolution in ant–plant mutualisms**
Nat Commun, 7 (2016), p. 12679 (*Seven Pseudomyrmex genomes were compared, finding that rates of molecular evolution are higher in the species that are mutualistic with Acaciathan in non-mutualists.*)

47. S. Nygaard, H. Hu, C. Li, M. Schiøtt, Z. Chen, Z. Yang, Q. Xie, C. Ma, Y. Deng, R.B. Dikow, et al.
**Reciprocal genomic evolution in the ant–fungus agricultural symbiosis**
Nat Commun, 7 (2016), p. 12233

48. •B. Feldmeyer, J. Mazur, S. Beros, H. Lerp, H. Binder, S. Foitzik
**Gene expression patterns underlying parasite-induced alterations in host behaviour and life history**
Mol Ecol, 25 (2016), pp. 648-660 (*Infection of Temnothorax nylanderi workers with the cestdoe Anomotaeniaaffects gene expression profiles in the brain.*)

49. C. de Bekker, R.A. Ohm, R.G. Loreto, A. Sebastian, I. Albert, M. Merrow, A. Brachmann, D.P. Hughes
**Gene expression during zombie ant biting behavior reflects the complexity underlying fungal parasitic behavioral manipulation**
BMC Genomics, 16 (2015), p. 620

50. •A. Klein, L. Schrader, R. Gil, A. Manzano-Marín, L. Flórez, D. Wheeler, J.H. Werren, A. Latorre, J. Heinze, M. Kaltenpoth, et al.
**A novel intracellular mutualistic bacterium in the invasive ant Cardiocondyla obscurior**
ISME J, 10 (2016), pp. 376-388 (*Genome sequencing of Cardiocondyla revealed the presence of a previously unknown bacterial endosymbiont, here described and named Westeberhardia.*)

51. J.A. Russell, J.G. Sanders, C.S. Moreau
**Hotspots for symbiosis: function, evolution, and specificity of ant-microbe associations from trunk to tips of the ant phylogeny (Hymenoptera: Formicidae)**
Myrmecol News, 24 (2017), pp. 43-69

52. S.M. Valles, D. Shoemaker, Y. Wurm, C.A. Strong, L. Varone, J.J. Becnel, P.D. Shirk
**Discovery and molecular characterization of an ambisense densovirus from South American populations of Solenopsis invicta**
Biol Control, 67 (2013), pp. 431-439

53. M.A.M. Gruber, M. Cooling, J.W. Baty, K. Buckley, A. Friedlander, O. Quinn, J.F.E.J. Russell, A. Sébastien, P.J. Lester
**Single-stranded RNA viruses infecting the invasive Argentine ant, Linepithema humile**
Sci Rep, 7 (2017), p. 3304

54. S.M. Valles, D.H. Oi, J.J. Becnel, J.K. Wetterer, J.S. LaPolla, A.E. Firth
**Isolation and characterization of Nylanderia fulva virus 1, a positive-sense, single-stranded RNA virus infecting the tawny crazy ant, Nylanderia fulva**
Virology, 496 (2016), pp. 244-254

55. S.M. Valles, D.H. Oi, F. Yu, X.-X. Tan, E.A. Buss
**Metatranscriptomics and pyrosequencing facilitate discovery of potential viral natural enemies of the invasive Caribbean crazy ant, Nylanderia pubens**
PLoS ONE, 7 (2012), p. e31828

56. M. Cooling, M.A.M. Gruber, B.D. Hoffmann, A. Sébastien, P.J. Lester
**A metatranscriptomic survey of the invasive yellow crazy ant, Anoplolepis gracilipes, identifies several potential viral and bacterial pathogens and mutualists**
Insectes Soc, 64 (2016), pp. 197-207

57. Y. Wurm, J. Wang, O. Riba-Grognuz, M. Corona, S. Nygaard, B.G. Hunt, K.K. Ingram, L. Falquet, M. Nipitwattanaphon, D. Gotzek, et al.
**The genome of the fire ant Solenopsis invicta**
Proc Natl Acad Sci U S A, 108 (2011), pp. 5679-5684

58. S.K. McKenzie, P.R. Oxley, D.J.C. Kronauer
**Comparative genomics and transcriptomics in ants provide new insights into the evolution and function of odorant binding and chemosensory proteins**
BMC Genomics, 15 (2014), p. 718

59. J. Kulmuni, Y. Wurm, P. Pamilo
**Comparative genomics of chemosensory protein genes reveals rapid evolution and positive selection in ant-specific duplicates**
Heredity, 110 (2013), pp. 538-547

60. X. Zhou, A. Rokas, S.L. Berger, J. Liebig, A. Ray, L.J. Zwiebel
**Chemoreceptor evolution in hymenoptera and its implications for the evolution of eusociality**
Genome Biol Evol, 7 (2015), pp. 2407-2416

61. ••W. Trible, L. Olivos-Cisneros, S.K. McKenzie, J. Saragosti, N.-C. Chang, B.J. Matthews, P.R. Oxley, D.J.C. Kronauer
**orco mutagenesis causes loss of antennal lobe glomeruli and impaired social behavior in ants**
Cell, 170 (2017) 727–735.e10 (*CRISPR was used on Ooceraea biroi ants to disable orco, a gene needed for odorant receptors to work. The mutant ants lacked antennal glomeruli.*)

62. ••H. Yan, C. Opachaloemphan, G. Mancini, H. Yang, M. Gallitto, J. Mlejnek, A. Leibholz, K. Haight, M. Ghaninia, L. Huo, et al.
**An engineered orco mutation produces aberrant social behavior and defective neural development in ants**
Cell, 170 (2017) 736–747.e9 (*CRISPR was used on Harpegnathos saltator ants to disable the orco gene required for odorant receptors. This similarly had strong developmental and behavioral and effects.*)

63. A.C. LeBoeuf, P. Waridel, C.S. Brent, A.N. Gonçalves, L. Menin, D. Ortiz, O. Riba-Grognuz, A. Koto, Z.G. Soares, E. Privman, et al.
**Oral transfer of chemical cues, growth proteins and hormones in social insects**
Elife, 5 (2016)

64. •K.M. Glastad, S.V. Arsenault, K.L. Vertacnik, S.M. Geib, S. Kay, B.N. Danforth, S.M. Rehan, C.R. Linnen, S.D. Kocher, B.G. Hunt
**Variation in DNA methylation is not consistently reflected by sociality in hymenoptera**
Genome Biol Evol, 9 (2017), pp. 1687-1698 (*Analysis of bisulfite sequencing data from nine hymenopteran species finds no clear correlation between methylation and sociality, and highlights that the correlation between presence of CpG dinucleotides in the genome and actual methylation varies between taxa.*)

65. •M. Hasselmann, L. Ferretti, A. Zayed
**Beyond fruit-flies: population genomic advances in non-Drosophila arthropods**
Brief Funct Genomics, 14 (2015), pp. 424-431 (*Review of non-Drosophila genomics and transcriptomics projects, with emphasis on population genomic analysis of social insects, butterflies and water flea.*)

66. C. Ratzka, R. Gross, H. Feldhaar
**Systemic gene knockdown in Camponotus floridanus workers by feeding of dsRNA**
Insectes Soc, 60 (2013), pp. 475-484

67. M.-Y. Choi, R.K. Vander Meer, M. Coy, M.E. Scharf
**Phenotypic impacts of PBAN RNA interference in an ant, Solenopsis invicta, and a moth, Helicoverpa zea**
J Insect Physiol, 58 (2012), pp. 1159-1165

68. D.A. Friedman, D.M. Gordon, L. Luo
**The MutAnts are here**
Cell, 170 (2017), pp. 601-602

69. C.D. Smith, A. Zimin, C. Holt, E. Abouheif, R. Benton, E. Cash, V. Croset, C.R. Currie, E. Elhaik, C.G. Elsik, et al.
**Draft genome of the globally widespread and invasive Argentine ant (Linepithema humile)**
Proc Natl Acad Sci USA, 108 (2011), pp. 5673-5678

70. P.R. Oxley, L. Ji, I. Fetter-Pruneda, S.K. McKenzie, C. Li, H. Hu, G. Zhang, D.J.C. Kronauer
**The genome of the clonal raider ant Cerapachys biroi**
Curr Biol, 24 (2014), pp. 451-458

71. R. Bonasio, G. Zhang, C. Ye, N.S. Mutti, X. Fang, N. Qin, G. Donahue, P. Yang, Q. Li, C. Li, et al.
**Genomic comparison of the ants Camponotus floridanus and Harpegnathos saltator**
Science, 329 (2010), pp. 1068-1071

72. S.K. Gupta, M. Kupper, C. Ratzka, H. Feldhaar, A. Vilcinskas, R. Gross, T. Dandekar, F. Förster
**Scrutinizing the immune defence inventory of Camponotus floridanus applying total transcriptome sequencing**
BMC Genomics, 16 (2015), p. 540

73. M.K. Hojo, K. Ishii, M. Sakura, K. Yamaguchi, S. Shigenobu, M. Ozaki
**Antennal RNA-sequencing analysis reveals evolutionary aspects of chemosensory proteins in the carpenter ant, Camponotus japonicus**
Sci Rep, 5 (2015), p. 13541

74. H. Johansson, K. Dhaygude, S. Lindström, H. Helanterä, L. Sundström, K. Trontti
**A metatranscriptomic approach to the identification of microbiota associated with the ant Formica exsecta**
PLoS ONE, 8 (2013), p. e79777

75. E.A. Konorov, M.A. Nikitin, K.V. Mikhailov, S.N. Lysenkov, M. Belenky, P.L. Chang, S.V. Nuzhdin, V.A. Scobeyeva
**Genomic exaptation enables Lasius niger adaptation to urban environments**
BMC Evol Biol, 17 (2017), p. 39

76. E.R. Lucas, J. Romiguier, L. Keller
**Gene expression is more strongly influenced by age than caste in the ant Lasius niger**
Mol Ecol, 26 (2017), pp. 5058-5073

77. S. Nygaard, G. Zhang, M. Schiøtt, C. Li, Y. Wurm, H. Hu, J. Zhou, L. Ji, F. Qiu, M. Rasmussen, et al.
**The genome of the leaf-cutting ant Acromyrmex echinatior suggests key adaptations to advanced social life and fungus farming**
Genome Res, 21 (2011), pp. 1339-1348

78. J. Stanton-Geddes, A. Nguyen, L. Chick, J. Vincent, M. Vangala, R.R. Dunn, A.M. Ellison, N.J. Sanders, N.J. Gotelli, S.H. Cahan
**Thermal reactionomes reveal divergent responses to thermal extremes in warm and cool-climate ant species**
BMC Genomics, 17 (2016), p. 171

79. G. Suen, C. Teiling, L. Li, C. Holt, E. Abouheif, E. Bornberg-Bauer, P. Bouffard, E.J. Caldera, E. Cash, A. Cavanaugh, et al.
**The genome sequence of the leaf-cutter ant Atta cephalotes reveals insights into its obligate symbiotic lifestyle**
PLoS Genet, 7 (2011), p. e1002007

80. L. Schrader, J.W. Kim, D. Ence, A. Zimin, A. Klein, K. Wyschetzki, T. Weichselgartner, C. Kemena, J. Stökl, E. Schultner, et al.
**Transposable element islands facilitate adaptation to novel environments in an invasive species**
Nat Commun, 5 (2014), p. 5495

81. K. von Wyschetzki, H. Lowack, J. Heinze
**Transcriptomic response to injury sheds light on the physiological costs of reproduction in ant queens**
Mol Ecol, 25 (2016), pp. 1972-1985

82. C.R. Smith, C.D. Smith, H.M. Robertson, M. Helmkampf, A. Zimin, M. Yandell, C. Holt, H. Hu, E. Abouheif, R. Benton, et al.
**Draft genome of the red harvester ant Pogonomyrmex barbatus**
Proc Natl Acad Sci U S A, 108 (2011), pp. 5667-5672

83. P.M. Rydzak, B.R. Bextine
**Manipulation of viral titers of Solenopsis invicta virus-1 by RNA interference in laboratory colonies of red imported fire ant**
Southwest Entomol, 41 (2016), pp. 379-388

84. W. Bouzid, M. Verdenaud, C. Klopp, F. Ducancel, C. Noirot, A. Vétillard
**De novo sequencing and transcriptome analysis for Tetramorium bicarinatum: a comprehensive venom gland transcriptome analysis from an ant species**
BMC Genomics, 15 (2014), p. 987

85. A.F.C. Torres, C. Huang, C.-M. Chong, S.W. Leung, A.R.B. Prieto-da-Silva, A. Havt, Y.P. Quinet, A.M.C. Martins, S.M.Y. Lee, G. Rádis-Baptista
**Transcriptome analysis in venom gland of the predatory giant ant Dinoponera quadriceps: insights into the polypeptide toxin arsenal of hymenopterans**
PLoS ONE, 9 (2014), p. e87556

<br />
